<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
  <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
</head>
<body>
<h1 id="genetic-drift-and-neutral-diversity">Genetic Drift and Neutral Diversity</h1>
<p>Various sources of randomness are inherent in evolution. One major source of stochasticity in population genetics is genetic drift. Genetic drift occurs because more or less copies of an allele by chance can be transmitted to the next generation. This can occur because by chance the individuals carrying a particular allele can leave more or less offspring in the next generation. In a sexual population genetic drift also occurs because mendelian transmission means that only one of the two alleles in an individual, chosen at random at a locus, is transmitted to the offspring.</p>
<p>Genetic drift can play a role in the dynamics of all alleles and populations, but it will play the biggest role for neutral alleles. A neutral polymorphism occurs when the segregating alleles at a polymorphic site have no discernable effect on the fitness (we’ll make clear what we mean by discernable later, for the moment think of this as “no effect” on fitness).</p>
<h2 id="LossofHet">Loss of heterozygosity due to drift.</h2>
<p>Genetic drift will, in the absence of new mutations, slowly purge our population of neutral genetic diversity as alleles slowly drift to high or low frequencies and are lost or fixed over time.<br />
Imagine a population of a constant size <span class="math inline">\(N\)</span> diploid individuals, and that we are examining a locus segregating for two alleles that are neutral with respect to each other. This population is randomly mating with respect to the alleles at this locus.<br />
In generation <span class="math inline">\(t\)</span> our current level of heterozygosity is <span class="math inline">\(H_t\)</span>, i.e. the probability that two randomly sampled alleles in generation <span class="math inline">\(t\)</span> are non-identical is <span class="math inline">\(H_t\)</span>. Assuming that the mutation rate is zero (or vanishing small), what is our level of heterozygosity in generation <span class="math inline">\(t+1\)</span>?<br />
In the next generation (<span class="math inline">\(t+1\)</span>) we are looking at the alleles in the offspring of generation <span class="math inline">\(t\)</span>. If we randomly sample two alleles in generation <span class="math inline">\(t+1\)</span> which had different parental alleles in generation <span class="math inline">\(t\)</span> then it is just like drawing two random alleles from generation <span class="math inline">\(t\)</span>. So the probability that these two alleles in generation <span class="math inline">\(t+1\)</span>, that have different parental alleles in generation <span class="math inline">\(t\)</span>, are non-identical is <span class="math inline">\(H_t\)</span>.<br />
Conversely, if our pair of alleles have the same parental allele in the proceeding generation (i.e. the alleles are identical by descent one generation back) then these two alleles must be identical (as we are not allowing for any mutation).<br />
In a diploid population of size <span class="math inline">\(N\)</span> individuals there are <span class="math inline">\(2N\)</span> alleles. The probability that our two alleles have the same parental allele in the proceeding generation is <span class="math inline">\(1/(2N)\)</span>, the probability that they have different parental alleles is is <span class="math inline">\(1-1/(2N)\)</span>. So by the above argument the expected heterozygosity in generation <span class="math inline">\(t+1\)</span> is <span class="math display">\[H_{t+1} = \frac{1}{2N} \times 0 + \left(1-\frac{1}{2N} \right)H_t\]</span> By this argument if the heterzygosity in generation <span class="math inline">\(0\)</span> is <span class="math inline">\(H_0\)</span> our expected heterozygosity in generation <span class="math inline">\(t\)</span> is <span class="math display">\[H_t = \left(1-\frac{1}{2N} \right)^tH_0\]</span> i.e. the expected heterozygosity with our population is decaying geometrically with each passing generation. If we assume that <span class="math inline">\(1/(2N)
\ll 1\)</span> then we can approximate this geometric decay by an exponential decay, such that <span class="math display">\[H_t =H_0 \exp \left(-\frac{t}{2N} \right)\]</span> i.e. heterozygosity decays exponentially at a rate <span class="math inline">\(1/(2N)\)</span>.</p>
<h2 id="DriftMutationBalance">Levels of diversity maintained by a balance between mutation and drift</h2>
<p>Looking backwards in time from one generation to the next, we are going to say that two alleles which have the same parental allele (i.e. find their common ancestor) in the preceding generation have <span><em>c</em>oalesced</span>, and refer to this event as a <span><em>c</em>oalescent event</span>.</p>
<p>The probability that our pair of randomly sampled alleles have coalesced in the preceding generation is <span class="math inline">\(1/(2N)\)</span>, the probability that our pair of alleles fail to coalesce is <span class="math inline">\(1-1/(2N)\)</span>.</p>
<p>The probability that a mutation changes the identity of the transmitted allele is <span class="math inline">\(\mu\)</span> per generation. So the probability of no mutation occurring is <span class="math inline">\((1-\mu)\)</span>. We’ll assume that when a mutation occurs it creates some new allelic type which is not present in the population. This assumption (commonly called the infinitely-many-alleles model) makes the math slightly cleaner, and also is not too bad an assumption biologically.</p>
<p>This model lets us calculate when our two alleles last shared a common ancestor and whether these alleles are identical as a result of failing to mutate since this shared ancestor. For example we can work out the probability that our two randomly sampled alleles coalesced <span class="math inline">\(2\)</span> generations in the past (i.e. they fail to coalesce in generation <span class="math inline">\(1\)</span> and then coalescent in generation <span class="math inline">\(2\)</span>), and that they are identical as <span class="math display">\[\left(1- \frac{1}{2N} \right) \frac{1}{2N} (1-\mu)^4\]</span> note the power of <span class="math inline">\(4\)</span> is because our two alleles have to have failed to mutate through <span class="math inline">\(2\)</span> meioses each.</p>
<p>More generally the probability that our alleles coalesce in generation <span class="math inline">\(t+1\)</span> and are identical due to no mutation to either allele in the subsequent generations is <span class="math display">\[P(\textrm{coal. in t+1 \&amp; no mutations}) =  \frac{1}{2N} \left(1- \frac{1}{2N} \right)^t \left(1-\mu \right)^{2(t+1)}\]</span> assuming that <span class="math inline">\(\frac{1}{2N} \ll\)</span> and <span class="math inline">\(\mu \ll 1\)</span> then we can approximate this as <span class="math display">\[P(\textrm{coal. in t+1 \&amp; no mutations}) \approx  \frac{1}{2N}
e^{-t/(2N)} e^{-2\mu (t+1)}\]</span> to make this slightly easier on ourselves lets further assume that <span class="math inline">\(t
\approx t+1\)</span> and so rewrite this as <span class="math display">\[\approx \frac{1}{2N} e^{-t(2\mu+1/(2N))}\]</span></p>
<p>If we sample two alleles at random from the population we will not in general know when they share a common ancestor. In which case we will need to integrate out over when this coalescent event occured. Doing this we find the probability that our two alleles are identical due to no mutation on either ancestral lineage since the pair shared a common ancestor to be <span class="math display">\[\frac{1}{2N} \int_0^{\infty} e^{-t(2\mu+1/(2N))} dt =
\frac{1/(2N)}{1/(2N)+2\mu} = \frac{1}{1+4N\mu}\]</span> The probability that our our pair of alleles are non-identical is simply one minus this, i.e. <span class="math display">\[\frac{4N\mu}{1+4N\mu} \label{eqn:hetero}\]</span> This compound parameter <span class="math inline">\(4N\mu\)</span>, the population-scaled mutation rate, will come up a number of times so we’ll give it its own name <span class="math display">\[\theta = 4N\mu\]</span></p>
<p>So all else being equal, species with larger population sizes should have proportionally higher levels of neutral polymorphism.</p>
<h2 id="the-effective-population-size.">The effective population size.</h2>
<p>In practice populations rarely conform to our assumptions of being constant in size with low variance in reproduction success. Real populations experience dramatic fluctuations in size, and there is often high variance in reproductive success. Thus rates of drift in natural populations are often a lot higher than the census population size would imply.<br />
To cope with this population geneticists often invoke the concept of an effective population size (<span class="math inline">\(N_e\)</span>). In many situations (but not all), departures from model assumptions can be captured by substituting <span class="math inline">\(N_e\)</span> for <span class="math inline">\(N\)</span>.</p>
<p>Specifically the effective population size (<span class="math inline">\(N_e\)</span>) is the population size that would result in the same rate of drift in an idealized constant population size, obeying our modeling assumptions, as that observed in our true population.<br />
If population sizes vary rapidly in size, we can (if certain conditions are met) replace our population size by the harmonic mean population size. Consider a diploid population of variable size, whose size is <span class="math inline">\(N_t\)</span> <span class="math inline">\(t\)</span> generations into the past. The probability our pairs of alleles have not coalesced by the generation <span class="math inline">\(t^{th}\)</span> is given by <span class="math display">\[\prod_{i=1}^{t} \left(1-\frac{1}{2N_t} \right)\]</span> note that this is simply collapses to our original expression <span class="math inline">\(\left(1-\frac{1}{2N } \right)^t \)</span> if <span class="math inline">\(N_i\)</span> is constant. If <span class="math inline">\(1/(N_i)\)</span> is small, then we can approximate <span class="math inline">\(1-\frac{1}{2N_i}\)</span> by <span class="math inline">\(\exp(-\frac{1}{2N_i})\)</span>. Such that if <span class="math inline">\(N_i\)</span> is never too small <span class="math display">\[\prod_{i=1}^{t} \left(1-\frac{1}{2N_i} \right)
\approx \prod_{i=1}^{t} \exp \left( -\frac{1}{2N_i} \right)   =
\exp \left(- \sum_{i=1}^{t} \frac{1}{2N_i} \right) .\]</span> In our constant population size case the probability of failing to coalesce is <span class="math inline">\(\exp(-t/(2N))\)</span>. So the variable population coalescent probabilities are still of the same form but the exponent has changed. Comparing the exponent in the two cases we see <span class="math display">\[\frac{t}{2N} = \sum_{i=1}^{t} \frac{1}{2N_i}\]</span> so that if we want a constant effective population size (<span class="math inline">\(N_e\)</span>) that has the same coalescent probability as our variable population we need to set <span class="math inline">\(N=N_e\)</span> and rearrange this to see <span class="math display">\[N_e =\frac{1}{\frac{1}{t} \sum_{i=1}^{t} \frac{1}{N_i} }.\]</span> this is the harmonic mean of the varying population size. Thus our effective population size, the size of an idealized constant population which matches the rate of genetic drift, is the harmonic mean true population size over time. The harmonic mean is very storngly affected by small values, such that if our population size is one million <span class="math inline">\(99\%\)</span> of the time but drops to a <span class="math inline">\(1000\)</span> every hundred or so generations, <span class="math inline">\(N_e\)</span> will be much closer to <span class="math inline">\(1000\)</span> than a million.<br />
Variance in reproductive success will also affect our effective population size. Even if our population has a large constant size of <span class="math inline">\(N\)</span> individuals, if only small proportion of them get to reproduce then the rate of drift will reflect this much small number of reproducing individuals. If only <span class="math inline">\(N_M\)</span> males get to contribute to the next generation and <span class="math inline">\(N_F\)</span> females get to contribute to the next generation. When our two alleles pick an ancestor, <span class="math inline">\(25\%\)</span> of the time our alleles were both in a female ancestor in which case they coalesce with probability <span class="math inline">\(1/(2N_F)\)</span>, and <span class="math inline">\(25\%\)</span> of the time they are both in a male ancestor in which case they coalesce with probability <span class="math inline">\(1/(2N_M)\)</span>. The remaining <span class="math inline">\(50\%\)</span> of the time our ancestral lineages are in two individuals are different sexes in a generation so cannot coalescence. Therefore, our probability of coalescence in the preceding generation is <span class="math display">\[\frac{1}{4}\frac{1}{2N_M}+\frac{1}{4}\frac{1}{2N_F} =
\frac{1}{8}\frac{N_F+N_M}{N_FN_M}\]</span> i.e. the rate of coalescence is the harmonic mean of the two sexes population sizes, equating this to <span class="math inline">\(\frac{1}{2N_e}\)</span> we find <span class="math display">\[N_e = \frac{4N_FN_M}{N_F+N_M}\]</span> Thus if reproductive success is very skewed in one sex (e.g. <span class="math inline">\(N_M \ll
N/2\)</span>) our effective population size will be much reduced as a result.<br />
</p>
<h2 id="neutral-diversity-and-population-structure">Neutral diversity and population structure</h2>
<p>Upto now we have assumed that our alleles that we have modelled in the coalescent setting are drawn from a randomly mating population such that any pair of lineages is equally likely to coalesce with each other. However, when there is population structure this assumption is violated.<br />
We have previously written the measure of population structure <span class="math inline">\({F_{\mathrm{ST}}}\)</span> as <span class="math display">\[{F_{\mathrm{ST}}}= \frac{H_T-H_S}{H_T}\]</span> where <span class="math inline">\(H_S\)</span> is the probability that two alleles sampled at random from a subpopulation differ, and <span class="math inline">\(H_T\)</span> is the probability that two alleles sampled at random from the total population differ.</p>
<h4 id="a-simple-population-split-model">A simple population split model</h4>
<p>Imagine a population of constant size of <span class="math inline">\(N_e\)</span> diploid individuals that <span class="math inline">\(\tau\)</span> generations in the past split into two daughter populations (sub-populations) each of size <span class="math inline">\(N_e\)</span> individuals, who do not subsequently exchange migrants. In the current day we sample an equal number of alleles from both subpopulations.</p>
<p>Consider a pair of alleles sampled within one of our sub-populations, they have experienced a population of size <span class="math inline">\(N_e\)</span> and so the probability that they differ is <span class="math inline">\(H_S = \theta/(1+\theta)\)</span> (where <span class="math inline">\(\theta=4N_e\mu\)</span>). The heterozygosity in our total population is a little more tricky to calculate. Assuming that we equally sample both sub-populations, when we draw two alleles from our total sample, <span class="math inline">\(50\%\)</span> of the time they are drawn from the same subpopulation and <span class="math inline">\(50\%\)</span> of the time they are drawn from different subpopulations. Therefore, our total heterozygosity is given by <span class="math display">\[H_T = {\tfrac{1}{2}}H_S + {\tfrac{1}{2}}H_B\]</span> where <span class="math inline">\(H_B\)</span> is the probability that a pair of alleles drawn from our two different sub-populations differ from each other. Our pair of alleles can not find a common ancestor with each other for at least <span class="math inline">\(\tau\)</span> generations into he past as they are in distinct populations (not connected by migration). The probability that one or other of them mutates in this time is <span class="math inline">\(1-(1-\mu)^{2T}\)</span>. With probability <span class="math inline">\((1-\mu)^{2T} \)</span> neither of our alleles mutate in the <span class="math inline">\(T\)</span> generations back in time before they find themselves back in the combined ancestral population. Conditional on failing to mutating before the combined ancestral population, the probability that they do manage to mutate before coalescing in that population of size <span class="math inline">\(N_e\)</span> is <span class="math inline">\(\theta/(\theta+1)\)</span>. Putting these components together <span class="math display">\[H_B = \left( 1-(1-\mu)^{2T} \right) + (1-\mu)^{2T}
  \frac{\theta}{\theta+1}\]</span> We can plug this into our expression for <span class="math inline">\(H_T\)</span>, and then that in turn into <span class="math inline">\({F_{\mathrm{ST}}}\)</span>.</p>
<p>To understand this better we can make a simple approximation based on our mutation rate being very low, such that <span class="math inline">\(N_e \mu \ll 1\)</span> ao <span class="math inline">\(H_S \approx
4N_e\mu\)</span>, and that <span class="math inline">\(\mu \ll 1\)</span> and <span class="math inline">\(\mu T \ll 1\)</span>. Assuming this, then <span class="math display">\[H_B \approx 2 \mu T + 4N_e\mu.\]</span> So that <span class="math display">\[{F_{\mathrm{ST}}}\approx \frac{ \mu T}{\mu T +  4N_e\mu }  \]</span> note that <span class="math inline">\(\mu\)</span> cancels out of this. In this simple toy model <span class="math inline">\({F_{\mathrm{ST}}}\)</span> is increasing because the amount of between population diversity increases with the divergence time of the two populations (initially linearly with <span class="math inline">\(T\)</span>). It does so at a rate give by <span class="math inline">\(T/(4N_e)\)</span> so that differentiation will be higher between populations seperated by long divergence times or with small effective population sizes.</p>
<h4 id="a-simple-model-of-migration-between-an-island-and-the-mainland.">A simple model of migration between an island and the mainland.</h4>
<p>We can also use the coalescent to think about patterns of differentiation under a simple model of migration drift equilibrium. Lets consider a small island population that is relatively isolated from a large mainland population, and that both of these populations are constant in size. We’ll assume that the expected heterozygosity for a pair of alleles sampled on the mainland is <span class="math inline">\(H_M\)</span>.</p>
<p>Our island has a population size <span class="math inline">\(N_{I}\)</span> that is very small compared to our mainland population. Each generation some low fraction <span class="math inline">\(m\)</span> of our individuals on the island have migrant parents from the mainland the generation before. Our island may also send migrants back to the mainland, but these are a drop in the ocean compared to the large population size on the mainland and their effect can be ignored.</p>
<p>If we sample an allele on the island back and trace its ancestral lineage backward in time, each generation our ancestral allele have a low probability <span class="math inline">\(m\)</span> of being descended from the mainland in the proceeding generation (if we go far enough the allele eventually has to be descended from an allele on the mainland). The probability that a pair of alleles sampled on the island are descended from a shared recent common ancestral allele on the island, is the probability that our pair of alleles coalesce before either lineage migrates. For example, the probability that our pair of alleles coalesce <span class="math inline">\(t+1\)</span> generations back is <span class="math display">\[\frac{1}{2N_I}(1-m)^{2(t+1)} \left(1-\frac{1}{2N_I} \right)^{t} \approx
\frac{1}{2N_I} \exp\left( -t\left (\frac{1}{2N_I} + 2m\right) \right),\]</span> with the approximation following from assuming that <span class="math inline">\(m \ll 1\)</span> &amp; <span class="math inline">\(1/(2N_I)
\ll 1\)</span> (note that this is very similar to our derivation of heterozygosity above). The probability that our alleles coalescence before either one of them migrates off the island, irrespective of the time, is <span class="math display">\[\int_0^{\infty} \frac{1}{2N_I} \exp\left( -t\left (\frac{1}{2N_I} +
    2m\right) \right) dt = \frac{1/(2N_I) }{1/(2N_I) +
    2m}.\]</span></p>
<p>Lets assume that the mutation rate is very low such as it is very unlikely that the pair of alleles mutate before they coalesce on the island. Therefore, the only way that the alleles can be different from each other is if one or other of them migrates to the mainland, which happens with probability <span class="math display">\[1 - \frac{1/(2N_I) }{1/(2N_I) + 2m}\]</span> Conditional on one or other of our alleles migrating to the mainland, both of our alleles represent independent draws from the mainland and so differ from each other with probability <span class="math inline">\(H_M\)</span>. Therefore, the level of heterozygosity on the island is given by <span class="math display">\[H_I = (1 - \frac{1/(2N_I) }{1/(2N_I) + 2m})H_M\]</span> So the reduction of heterozygosity on the island compared to the mainland is <span class="math display">\[F_{IM} = 1- \frac{H_I}{H_M} = \frac{ 1/(2N_I) }{1/(2N_I) + 2m} = \frac{ 1 }{1 + 4N_Im}.\]</span> The level of inbreeding on the island compared to the mainland will be high in the migration rate is low and the effective population size of the island is low, as allele frequencies on the island are drifting and diversity is not being replenished on the island by migration. The key parameter here is the number individuals on the island replaced by immigrants from the mainland each generation (<span class="math inline">\(N_I m\)</span>).</p>
<p>We have framed this as being about the reduction in genetic diversity on the island compared to the mainland. However, if we consider collecting a individuals on the island and mainland in proportion to population sizes the total level of heterozygosity would be <span class="math inline">\(H_T=H_M\)</span>, as samples from our mainland would greatly outnumber those from our island. Therefore, considering our island our sub-population we have derived another simple model of <span class="math inline">\(F_{ST}\)</span> .</p>
<h2 id="other-approaches-to-population-structure">Other approaches to population structure</h2>
<p>There is a broad spectrum of methods to describe patterns of population structure in populaion genetic datasets. We’ll briefly discuss two broad-classes of methods, assigment methods and principal components analysis,that appear often in the literature.</p>
<h3 id="assignment-methods">Assignment Methods</h3>
<p>Here we’ll describe a simple probabilistic assignment to find the probability that an individual of unknown population comes from one of <span class="math inline">\(K\)</span> predefined populations. We’ll then briefly explain how to extend this to cluster individuals into <span class="math inline">\(K\)</span> initially unknown populations. This method is a simplified version of what Bayesian population genetics clustering algorithms such as STRUCTURE and ADMIXTURE do (Pritchard et al. Genetics 2000).</p>
<h4 id="a-simple-assignment-method">A simple assignment method</h4>
<p>We have genotype data from unlinked S bi-allelic loci for <span class="math inline">\(K\)</span> populations. The allele frequency of allele <span class="math inline">\(A_1\)</span> at locus <span class="math inline">\(l\)</span> in population <span class="math inline">\(k\)</span> is denoted by <span class="math inline">\(p_{k,l}\)</span>, so that the allele frequencies in population 1 are <span class="math inline">\(p_{1,1},\cdots p_{1,L}\)</span> and population 2 are <span class="math inline">\(p_{2,1},\cdots p_{2,L}\)</span> and so on.</p>
<p>You type a new individual from an unknown population at these <span class="math inline">\(L\)</span> loci. This individual’s genotype at locus <span class="math inline">\(l\)</span> is <span class="math inline">\(g_l\)</span>, where <span class="math inline">\(g_l\)</span> denotes the number of copies of allele <span class="math inline">\(A_1\)</span> this individual carries at this locus (<span class="math inline">\(g_l=0,1,2\)</span>).</p>
<p>The probability of this individual’s genotype at locus <span class="math inline">\(l\)</span> conditional on coming from population <span class="math inline">\(k\)</span> (i.e. their alleles being a random HW draw from population <span class="math inline">\(k\)</span>) is <span class="math display">\[P(g_l | \textrm{pop k}) = I(g_l=0) (1-p_{k,l})^2 +  I(g_l=1) 2 p_{k,l} (1-p_{k,l}) + I(g_l=2) p_{k,l}^2\]</span> where <span class="math inline">\(I(g_l=0)\)</span> is an indicator function which is <span class="math inline">\(1\)</span> if <span class="math inline">\(g_l=0\)</span> and zero otherwise, and likewise for the other indicator functions. This follows simply from HWE.</p>
<p>Assuming that the loci are independent, the probability of individual’s genotypes conditional on them coming from population <span class="math inline">\(k\)</span> is <span class="math display">\[P(\textrm{ind.} | \textrm{pop k})  = \prod_{l=1}^S P(g_l | \textrm{pop k}) \label{eqn_assignment}\]</span></p>
<p>We wish to know the probability that this new individual comes from population <span class="math inline">\(k\)</span>, i.e. <span class="math inline">\(P(\textrm{pop k} | \textrm{new ind.})\)</span>. We can obtain this through Bayes rule <span class="math display">\[P(\textrm{pop k} | \textrm{ind.})  = \frac{P(\textrm{ind.} | \textrm{pop k}) P(\textrm{pop k})}{P(\textrm{ind.})}\]</span> where <span class="math display">\[P(\textrm{ind.}) = \sum_{k=1}^K  P(\textrm{ind.} | \textrm{pop k}) P(\textrm{pop k})\]</span> is the normalizing constant. We interpret <span class="math inline">\(P(\textrm{pop k})\)</span> as the prior probability of the individual coming from population <span class="math inline">\(k\)</span>, unless we have some other prior knowledge we will assume that the new individual has a equal probability of coming from each population <span class="math inline">\(P(\textrm{pop k})=1/K\)</span>.</p>
<p>We intepret <span class="math display">\[P(\textrm{pop k} | \textrm{ind.})\]</span> as the posterior probability that our new individual comes from each of our <span class="math inline">\(1,\cdots, K\)</span> populations.</p>
<p>More sophisticated versions of this are now used to allow for hybrids, e.g, we can have a proportion <span class="math inline">\(q_k\)</span> of our individual’s genome come from population <span class="math inline">\(k\)</span> and estimate the set of <span class="math inline">\(q_k\)</span>’s.</p>
<p><span><strong>Question.</strong></span> We have two populations where the frequency of allele <span class="math inline">\(A_1\)</span> at two SNPs (<span class="math inline">\(A_1/A_2\)</span>) is given by</p>
<table>
<thead>
<tr class="header">
<th style="text-align: center;">Population</th>
<th style="text-align: center;">locus 1</th>
<th style="text-align: center;">locus 2</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">A</td>
<td style="text-align: center;"><span class="math inline">\(0.1\)</span></td>
<td style="text-align: center;"><span class="math inline">\(0.85\)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;">B</td>
<td style="text-align: center;"><span class="math inline">\(0.95\)</span></td>
<td style="text-align: center;"><span class="math inline">\(0.2\)</span></td>
</tr>
</tbody>
</table>
<p>We sample an individual whose genotype is <span class="math inline">\(A_1A_1\)</span> at the first locus and <span class="math inline">\(A_2A_2\)</span> at the second. What is the probability that our indvidual comes from population 1 vs population 2? Lets assume that with probability <span class="math inline">\(q_1\)</span> our individual draws an allele from population <span class="math inline">\(1\)</span> and that with probability <span class="math inline">\(q_2=1-q_1\)</span> they draw an allele from population <span class="math inline">\(2\)</span>. What is the probability of our individual’s genotype given <span class="math inline">\(q_1\)</span>? Plot this probability as a function of <span class="math inline">\(q_1\)</span>. How does your plot change if our individual is heterozygote at both loci?</p>
<h4 id="clustering-based-on-assignment-methods">Clustering based on assignment methods</h4>
<p>While it is great to be able to assign our individuals to particular population, these ideas can be pushed to learn about how best to describe our genotype data in terms of discrete populations without assigning any of our individuals to populations <span><em>a priori</em></span>. We wish to cluster our individuals into <span class="math inline">\(K\)</span> unknown populations. We begin by assigning our individuals at random to these <span class="math inline">\(K\)</span> populations.</p>
<ul>
<li><p>Given these assignments we estimate the allele frequencies at all of our loci in each population.</p></li>
<li><p>Given these allele frequencies we chose to reassign each individual to a population <span class="math inline">\(k\)</span> with a probability given by eqn. (<span class="math inline">\(\ref{eqn_assignment}\)</span>).</p></li>
</ul>
<p>We iterate steps 1 and 2 for many iterations. If the data is sufficiently informative the assignments and allele frequencies will quickly converge.</p>
<p>To do this in a full bayesian scheme we need to place priors on the allele frequencies (e.g. a beta distribution).Technically we are using this is the joint posterior of our allele frequencies and assignments.</p>
<h3 id="principal-components-analysis">Principal components analysis</h3>
<p>The use of principal component analysis in population genetics was pioneered by Cavalli-Sforza. With large genotyping datasets PCA has made a come back. See Patterson et al 2006, PLoS Genetics and McVean, G. 2010 PLoS Genetics and for recent discussion.</p>
<p>Consider a dataset consisting of N individuals at S bi-allelic SNPs. The <span class="math inline">\(i^{th}\)</span> individual’s genotype data at locus <span class="math inline">\(\ell\)</span> takes value <span class="math inline">\(g_{i,\ell}\)</span>=0,1, or 2 (corresponding to the number of copies of allele <span class="math inline">\(A_1\)</span> an individual carrys at this SNP). We can think of this as a N x S matrix (where usually <span class="math inline">\(N \ll S\)</span>).</p>
<p>Denoting the sample mean allele freq at SNP <span class="math inline">\(\ell\)</span> by <span class="math inline">\(p_{\ell}\)</span> we usually standardize the genotype in the following way <span class="math display">\[\frac{g_{i,\ell} - 2 p_{\ell}}{\sqrt{p_{\ell}(1-p_{\ell})}}\]</span> i.e. at each SNP we center the genotypes by minusing of the mean genotype (<span class="math inline">\(2\epsilon_{\ell}\)</span>) and divide through by the expected variance assuming that alleles are sampled binomially from the mean frequency (<span class="math inline">\(\sqrt{p_{\ell} (1-p_{\ell})}\)</span>). Doing this to all of our genotypes we form a data matrix (of dimension N x S). We can then perform principal components analysis of this data matrix to cover the major axes of genotype variance in our sample.</p>
<p>It is worth taking a moment to delve further into what we are doing here. There’s a number of equivalent ways to thinking about what PCA is doing, one of these is to think that when we do PCA we are building the individual by individual covariance matrix and performing eigen-value decomposition of this matrix (with the eigen-vectors giving the PC). This individual by individual covariance matrix has entries the <span class="math inline">\((i,~j)^{th}\)</span> entry given by <span class="math display">\[\sum_{\ell=1}^S \frac{(g_{i,\ell} - 2p_{\ell})(g_{j,\ell} - 2p_{\ell})}{p_{\ell}(1-p_{\ell})}\]</span> note that this is the covariance, is very similar to those we encountered in discussing <span class="math inline">\(F\)</span>-statistics as correlations (equation ), expect now we are asking about the allelic covariance between two individuals above that expected if they were both drawn from the total sample at random (rather than the covariance of alleles within a single individual). So by performing PCA on the data we are learning about the major (orthogonal) axes of the kinship matrix.</p>
</body>
</html>
