<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
  <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
</head>
<body>
<h1 id="the-phenotypic-resemblance-between-relatives">The phenotypic resemblance between relatives</h1>
<p>We can use our understanding of the sharing of alleles between relatives to understand the phenotypic resemblance between relatives in quantitative phenotypes. We can then use this to understand the evolutionary change in quantitative phenotypes in response to selection.<br />
Let’s imagine that the genetic component of the variation in our trait is controlled by <span class="math inline">\(L\)</span> autosomal loci that act in an additive manner. The frequency of allele <span class="math inline">\(1\)</span> at locus <span class="math inline">\(l\)</span> is <span class="math inline">\(p_l\)</span>, with each copy of allele <span class="math inline">\(1\)</span> at this locus increasing your trait value by <span class="math inline">\(a_l\)</span> above the population mean. The phenotype of an individual, let’s call her <span class="math inline">\(i\)</span>, is <span class="math inline">\(X_i\)</span>. Her genotype at SNP <span class="math inline">\(l\)</span>, is <span class="math inline">\(G_{i,l}\)</span>. Here <span class="math inline">\(G_{i,l}=0,~1,\)</span> or <span class="math inline">\(2\)</span> represents the number of copies of allele <span class="math inline">\(1\)</span> she has at this SNP. Her expected phenotype, given her genotype, is then <span class="math display">\[X_{A,i} = \mu + {\mathbb{E}}(X_i | G_{i,1},\cdots,G_{i,L}) = \mu+ \sum_{l=1}^L G_{i,l} a_{l}\]</span> where <span class="math inline">\(\mu\)</span> is the mean phenotype in our population. Now in reality the genetic phenotype is a function of the expression of those alleles in a particular environment. Therefore, we can think of this expected phenotype as being an average across a set of environments that occur in the population.<br />
When we measure our individual’s phenotype we see <span class="math display">\[X_i =   \mu+X_{A,i} + X_{E,i} \label{pheno_geno_environ}\]</span> where <span class="math inline">\(X_E\)</span> is the deviation from the mean phenotype due to the environment. This <span class="math inline">\(X_E\)</span> included the systematic effects of the environment our individual finds herself in and all of the noise during development, growth, and the various random insults that life throws at our individual. If a reasonable number of loci contribute to variation in our trait then we can approximate the distribution of <span class="math inline">\(X_{A,i}\)</span> by a normal distribution due to the central limit theorem (see R exercise). Thus if we can approximate the distribution of the effect of environmental variation on our trait (<span class="math inline">\(X_{E,i}\)</span>) also by a normal distribution, which is reasonable as there are many small environmental effects, then the distribution of phenotypes within the population (<span class="math inline">\(X_i\)</span>) will be normally distributed (see Figure [fig:QT1]).<br />
</p>
<figure>
<embed src="figures/QT1.eps" /><figcaption>The convergence of the phenotypic distribution to a normal distribution.<span data-label="fig:QT1"></span></figcaption>
</figure>
<p>Note that as this is an additive model we can decompose eqn. [pheno_geno_environ] into the effects of the two alleles at each locus, in particular we can rewrite it as <span class="math display">\[X_i = \mu + X_{iM}+X_{iP} +X_{iE}\]</span> where <span class="math inline">\(X_{iM}\)</span> and <span class="math inline">\(X_{iP}\)</span> are the contribution to the phenotype of the allele that our individual received from her mother (maternal alleles) and father (paternal alleles) respectively. This will come in handy in just a moment when we start thinking about the phenotype covariance of relatives.<br />
Now obviously this model seems silly at first sight as alleles don’t only act in an additive manner, as they interact with alleles at the same loci (dominance) and at different loci (epistasis). Later we’ll relax this assumption, however, we’ll find that if we are interested in evolutionary change over short time-scales it is actually only the “additive component” of genetic variation that will (usually) concern us. We will define this more formally later on, but for the moment we can offer the intuition that parents only get to pass on a single allele at each locus on to the next generation. As such, it is the effect of these transmitted alleles, averaged over possible matings, that is an individual’s average contribution to the next generation (i.e. the additive effect of the alleles that their genotype consists of).</p>
<h3 id="additive-genetic-variance-and-heritability">Additive genetic variance and heritability</h3>
<p>As we are talking about an additive genetic model we’ll talk about the additive genetic variance (<span class="math inline">\(V_A\)</span>), the variance due to the additive effects of segregating genetic variation. This is a subset of the total genetic variance if we allow for non-additive effects.<br />
The variance of our phenotype across individuals (<span class="math inline">\(V\)</span>) can write this as <span class="math display">\[V = Var(X_A) + Var(X_E) = V_A+V_E\]</span> in writing this we are assuming that there is no covariance between <span class="math inline">\(X_{G,i}\)</span> and <span class="math inline">\(X_{E,i}\)</span> i.e. there is no covariance between genotype and environment.<br />
Our additive genetic variance can be written as <span class="math display">\[V_A = \sum_{l=1}^L Var(G_{i,l} a_{l})\]</span> where <span class="math inline">\(Var(G_{i,l} a_{l})\)</span> is the contribution to the additive variance among individuals of the <span class="math inline">\(l\)</span> locus. Assuming random mating we can write our additive genetic variance as <span class="math display">\[V_A = \sum_{l=1}^L a_{l}^2 2 p_l(1-p_l)\]</span> where the <span class="math inline">\( 2 p_l(1-p_l)\)</span> term follows the binomial sampling of two alleles per individual at each locus.<br />
</p>
<h4 id="the-narrow-sense-heritability">The narrow sense heritability</h4>
<p>We would like a way to think about what proportion of the variation in our phenotype across individuals is due to genetic differences as opposed to environmental differences. Such a quantity will be key in helping us think about the evolution of phenotypes. For example, if variation in our phenotype had no genetic basis then no matter how much selection changes the mean phenotype within a generation the trait will not change over generations.<br />
We’ll call the proportion of the variance that is genetic the heritability, and denote it by <span class="math inline">\(h^2\)</span>. We can then write this as <span class="math display">\[h^2 = \frac{Var(X_A)}{V} = \frac{V_A}{V}\]</span> remember that we thinking about a trait where all of the alleles act in a perfectly additive manner. In this case our heritability <span class="math inline">\(h^2\)</span> is referred to as the narrow sense heritability, the proportion of the variance explained by the additive effect of our loci. When we allow dominance and epistasis into our model we’ll also have to define the broad sense heritability (the total proportion of the phenotypic variance attributable to genetic variation).<br />
The narrow sense heritability of a trait is a useful quantity, indeed we’ll see shortly that it is exactly what we need to understand the evolutionary response to selection on a quantitative phenotype. We can calculate the narrow sense heritability by using the resemblance between relatives. For example, if our phenotype was totally environmental we should not expect relatives to resemble each other any more than random individuals drawn from the population. Now the obvious caveat here is that relatives also share an environment, so may resemble each other due to shared environmental effects.<br />
</p>
<h3 id="the-covariance-between-relatives">The covariance between relatives</h3>
<p>So we’ll go ahead and calculate the covariance in phenotype between two individuals (<span class="math inline">\(1\)</span> and <span class="math inline">\(2\)</span>) who have a phenotype <span class="math inline">\(X_1\)</span> and <span class="math inline">\(X_2\)</span> respectively. <span class="math display">\[Cov(X_1,X_2) =
Cov\left((X_{1M}+X_{1P}+X_{1E}),((X_{2M}+X_{2P}+X_{2E}) \right)\]</span> We can expand this out in terms of the covariance between the various components in these sums.<br />
To make our task easier we (and most analyses) will assume two things</p>
<ol>
<li><p>that we can ignore the covariance of the environments between individuals (i.e. <span class="math inline">\(Cov(X_{1E},X_{2E})=0\)</span>)</p></li>
<li><p>that we can ignore the covariance between the environment variation experience by an individual and the genetic variation in another individual (i.e. <span class="math inline">\(Cov(X_{1E},(X_{2M}+X_{2P}))=0\)</span>).</p></li>
</ol>
<p>The failure of these assumptions to hold can severely undermine our estimates of heritability, but we’ll return to that later. Moving forward with these assumptions, we can write our phenotypic covariance between our pair of individuals as <span class="math display">\[Cov(X_1,X_2) =
Cov((X_{1M},X_{2M})+Cov(X_{1M},X_{2P})+Cov(X_{1P},X_{2M})
+Cov(X_{1P},X_{2P}) \label{cov_rels_1}\]</span> This is saying that under our simple additive model we can see the covariance in phenotypes between individuals as the covariance between the allelic effects in our individuals. We can use our results about the sharing of alleles between relatives to obtain these terms. But before we write down the general case lets quickly work through some examples.<br />
</p>
<h4 id="the-covariance-between-identical-twins">The covariance between Identical Twins</h4>
<p>Lets first consider the case of a pair of identical twins from two unrelated parents. Our pair of twins share their maternal and paternal allele identical by descent (<span class="math inline">\(X_{1M}=X_{2M}\)</span> and <span class="math inline">\(X_{1P}=X_{2P}\)</span>). As their maternal and paternal alleles are not correlated draws from the population, i.e. have no probability of being <span class="math inline">\(IBD\)</span> as we’ve said the parents are unrelated, the covariance between their effects on the phenotype is zero (i.e. <span class="math inline">\(Cov(X_{1P},X_{2M})=Cov(X_{1M},X_{2P})=0\)</span>). In that case eqn. [cov_rels_1] is <span class="math display">\[Cov(X_1,X_2) = Cov((X_{1M},X_{2M})+Cov(X_{1P},X_{2P}) = 2Var(X_{1M})
= V_A\]</span> Now in general identical twins are not going to be super helpful for us in estimating <span class="math inline">\(h^2\)</span> as under models with non-additive effects identical twins have higher covariance than we’d expect as they resemble each other also because of the dominance effects as they don’t just share alleles they share their entire genotype.<br />
</p>
<h4 id="the-covariance-in-phenotype-between-mother-and-child">The covariance in phenotype between mother and child</h4>
<p>. If the mother and father are unrelated individuals (i.e. are two random draws from the population) then the mother and a child share one allele IBD at each locus (i.e. <span class="math inline">\(r_1=1\)</span> and <span class="math inline">\(r_0=r_2=0\)</span>). Half the time our mother transmits her paternal allele to the child, in which case <span class="math inline">\(X_{P1}=X_{M2}\)</span> and so <span class="math inline">\(Cov(X_{P1},X_{M2})=Var(X_{P1})\)</span> and all the other covariances in eqn. [cov_rels_1] zero, and half the time she transmits her maternal allele to the child <span class="math inline">\(Cov(X_{M1},X_{M2})=Var(X_{M1})\)</span> and all the other terms zero. By this argument <span class="math inline">\(Cov(X_1,X_2) = {\tfrac{1}{2}}Var(X_{M1}) + {\tfrac{1}{2}}Var(X_{P1}) = {\tfrac{1}{2}}V_A\)</span>.<br />
</p>
<h4 id="the-covariance-between-general-pairs-of-relatives-under-an-additive-model">The covariance between general pairs of relatives under an additive model</h4>
<p>The two examples make clear that to understand the covariance between phenotypes of relatives we simply need to think about the alleles they share IBD. Consider a pair of relatives (<span class="math inline">\(1\)</span> and <span class="math inline">\(1\)</span>) with a probability <span class="math inline">\(r_0\)</span>, <span class="math inline">\(r_1\)</span>, and <span class="math inline">\(r_2\)</span> of sharing zero, one, or two alleles IBD respectively. When they share zero alleles <span class="math inline">\(Cov((X_{1M}+X_{1P}),(X_{2M}+X_{2P}))=0\)</span>, when they share one allele <span class="math inline">\(Cov((X_{1M}+X_{1P}),(X_{2M}+X_{2P}))=
Var(X_{1M})=\frac{1}{2}V_A\)</span>, and when they share two alleles <span class="math inline">\(Cov((X_{1M}+X_{1P}),(X_{2M}+X_{2P}))=
V_A\)</span>. Therefore, the general covariance between two relatives is <span class="math display">\[Cov(X_1,X_2) = r_0 \times 0 + r_1 \frac{1}{2}V_A + r_2  V_A =
2 F_{1,2} V_A\]</span><br />
So under a simple additive model of the genetic basis of a phenotype to measure the narrow sense heritability we need to measure the covariance between a set of pairs of relatives (assuming that we can remove the effect of shared environmental noise). From the covariance between relatives we can calculate <span class="math inline">\(V_A\)</span>, we can then divide this by the total phenotypic variance to get <span class="math inline">\(h^2\)</span>.<br />
Another way that we can estimate the narrow sense heritability is through the regression of child’s phenotype on the parental mid-point phenotype. The parental mid-point phenotype is simple the average of the mum and dad’s phenotype. Denoting the child’s phenotype by <span class="math inline">\(X_{kid}\)</span> and mid-point phenotype by <span class="math inline">\(X_{mid}\)</span> so that if we take the regression <span class="math inline">\(X_{kid} \sim X_{mid}\)</span> this regression has slope <span class="math inline">\(\beta = Cov(X_{kid},X_{mid})/Var(X_{mid})\)</span>. The covariance of <span class="math inline">\(Cov(X_{kid},X_{mid})={\tfrac{1}{2}}V_A\)</span>, and <span class="math inline">\(Var(X_{mid}) = {\tfrac{1}{2}}V\)</span> as by taking the average of the parents we have halved the variance, such that the slope of the regression is <span class="math display">\[\beta= \frac{Cov(X_{kid},X_{mid})}{Var(X_{mid})} = \frac{V_A}{V} = h^2\]</span> i.e. the regression of the child’s phenotype on the parental midpoint phenotype is an estimate of the narrow sense heritability. This is a common way to estimate heritability, although it doesn’t bypass the need to control for environmental correlations between relatives.<br />
Our regression allows us to attempt to predict the phenotype of the child given the child; how well we can do this depends on the slope. If the slope is close to zero then the parental phenotypes hold no information about the phenotype of the child, while if the slope is close to one then the parental mid-point is a good guess at the child’s phenotype.<br />
</p>
<figure>
<embed src="figures/QT2.eps" /><figcaption>Regression of parental mid-point phenotype on child’s phenotype.</figcaption>
</figure>
<p>More formally the expected phenotype of the child given the parental phenotypes is <span class="math display">\[{\mathbb{E}}(X_{kid} | X_{mum},X_{dad}) = \mu +
\beta(X_{mid} - \mu) =\mu + h^2(X_{mid} - \mu)  \label{predict_kid}\]</span> this follows from the definition of linear regression. So to find the child’s predicted phenotype we simply take the mean phenotype and add on the difference between our parental mid-point multiplied by our narrow sense heritability.<br />
</p>
<h4 id="estimating-additive-genetic-variance-across-a-variety-of-different-relationships.">Estimating additive genetic variance across a variety of different relationships.</h4>
<p>In many natural populations we may have access to individuals of a range of different relationships to each other (through monitoring of the paternity of individuals), but relatively few individuals of a given relationship (e.g. sibs). We can try and use this information as fully as possible in a mixed model framework. Considering equation [pheno_geno_environ] we can write an individual’s phenotype <span class="math inline">\(X_i\)</span> as <span class="math display">\[X_i =  \mu  + X_{A,i} + e_i\]</span> where <span class="math inline">\(e_i \sim N(0,V_E)\)</span> and <span class="math inline">\(X_{A,i}\)</span> is normally distributed across individuals with covariance matrix <span class="math inline">\(V_A A\)</span> where the the entries for an pair of individuals i and i are <span class="math inline">\(A_{ij}= 2 F_{i,j}\)</span> and <span class="math inline">\(A_{ii} 1\)</span>. Given the matrix <span class="math inline">\(A\)</span> we can estimate <span class="math inline">\(V_A\)</span>. We can also add fixed effects into this model to account for generation effects, additional mixed effects could also be included to account for shared environments between particular individuals (e.g. a shared nest). This is sometimes called the “animal model”, and also goes by the name of variance components analysis.</p>
<h4 id="multiple-traits.">Multiple traits.</h4>
<p>Traits often covary with each other, due to both environmentally induced effects (e.g. due to the effects of diet on multiple traits) and due to the expression of underlying genetic covariance between traits. In turn this genetic covariance can reflect pleiotropy, a mechanistic effect of an allele on multiple traits (e.g. variants that effect skin pigmentation often effect hair color) or the genetic linkage of loci independently affecting multiple traits. If we are interested in evolution over short time-scales we can (often) ignore the genetic basis of this correlation.</p>
<p>Consider two traits <span class="math inline">\(X_{1,i}\)</span> and <span class="math inline">\(X_{2,i}\)</span> in an indivdual <span class="math inline">\(i\)</span>, these could be say the indivdual’s leg length and nose length. As before we can write these as <span class="math display">\[\begin{aligned}
X_{1,i} &amp;= \mu_1+ X_{1,A,i} + X_{1,E,i}  \nonumber \\
X_{2,i} &amp;= \mu_2 +X_{2,A,i} + X_{2,E,i} \nonumber \\\end{aligned}\]</span> As before we can talk about the total phenotypic variance (<span class="math inline">\(V_1,V_2\)</span>), environmental variance (<span class="math inline">\(V_{1,E}\)</span> and <span class="math inline">\(V_{2,E}\)</span>), and the additive variance and in trait one and two and (<span class="math inline">\(V_{1,A}\)</span>, <span class="math inline">\(V_{2,A}\)</span>). But now we also have to consider the total covariance <span class="math inline">\(V_{1,2}=Cov(X_{1},X_{2})\)</span>, the environmentally induced covariance between the traits (<span class="math inline">\(V_{E,1,2}=Cov(X_{1,E}
,X_{2,E} )\)</span>) and the additive genetic covariance (<span class="math inline">\(V_{A,1,2}
=Cov(X_{1,A} ,X_{2,A} )\)</span>) between trait one and two.</p>
<p>We can store these values in a matrices <span class="math display">\[\bf{V}= \left( \begin{array}{cc} 
V_{1} &amp; V_{1,2} \\
V_{1,2} &amp; V_{2} \\
\end{array} \right) \label{P_matrix}\]</span> and <span class="math display">\[\bf{G}= \left( \begin{array}{cc} 
V_{1,A} &amp; V_{A,1,2} \\
V_{A,1,2} &amp; V_{2,A} \\
\end{array} \right)  \label{G_matrix}\]</span> we can generalize this to an abitrary number of traits.</p>
<p>We can estimate these quantities, in a similar way to before, by studying the covariance in different traits between relatives:</p>
<p><span class="math display">\[Cov(X_{1,i},X_{2,j}) = 2 F_{i,j} V_{A,1,2}\]</span></p>
<h3 id="the-response-to-selection">The response to selection</h3>
<p>Evolution by natural selection requires:</p>
<ol>
<li><p>Variation in a phenotype</p></li>
<li><p>That survival is non-random with respect to this phenotypic variation.</p></li>
<li><p>That this variation is heritable.</p></li>
</ol>
<p>Points 1 and 2 encapsulate our idea of Natural Selection, but evolution by natural selection will only occur if the 3rd condition is met. It is the heritable nature of variation that couples change within a generation due to natural selection, to change across generations (evolutionary change).<br />
Lets start by thinking about the change within a generation due to directional selection, where selection acts to change the mean phenotype within a generation. For example, a decrease in mean height within a generation, due to taller organisms having a lower chance of surviving to reproduction than shorter organisms. Specifically, we’ll denote our mean phenotype at reproduction by <span class="math inline">\(\mu_S\)</span>, i.e. after selection has acted, and our mean phenotype before selection acts by <span class="math inline">\(\mu_{BS}\)</span>. This second quantity may be hard to measure, as obviously selection acts throughout the life-cycle, so it might be easier to think of this as the mean phenotype if selection hadn’t acted. So the mean phenotype changes within a generation is <span class="math inline">\(\mu_{S} - \mu_{BS}= S\)</span>.<br />
We are interested in predicting the distribution of phenotypes in next generation, in particular we are interested in the mean phenotype in the next generation to understand how directional selection has contributed to evolutionary change. We’ll denote the mean phenotype in offspring, i.e. the mean phenotype in the next generation before selection acts, as <span class="math inline">\(\mu_{NG}\)</span>. The change across generations we’ll call the response to selection <span class="math inline">\(R\)</span> and put this equal to <span class="math inline">\(\mu_{NG}- \mu_{BS}\)</span>.<br />
The mean phenotype in the next generation is <span class="math display">\[\mu_{NG} = {\mathbb{E}}\left( {\mathbb{E}}(X_{kid} | X_{mum},X_{dad}) \right)\]</span> where the outer expectation is over the randomly mating of individuals who survive to reproduce. We can use eqn. [predict_kid] to obtain an expression for this <span class="math display">\[\mu_{NG} = \mu_{BS} +
\beta ( {\mathbb{E}}(X_{mid}) - \mu_{BS})\]</span> so to obtain <span class="math inline">\(\mu_{NG}\)</span> we need to compute <span class="math inline">\({\mathbb{E}}(X_{mid})\)</span> the expected mid-point phenotype of pairs of individuals who survive to reproduce. Well this is just the expected phenotype in the individuals who survived to reproduce (<span class="math inline">\(\mu_{S}\)</span>), so <span class="math display">\[\mu_{NG} = \mu_{BS} +
h^2 (\mu_S - \mu_{BS})\]</span> So we can write our response to selection as <span class="math display">\[R = \mu_{NG} -\mu_{BS}  =
h^2 (\mu_S - \mu_{BS}) = h^2 S \label{breeders_eqn}\]</span> So our response to selection is proportional to our selection differential, and the constant of proportionality is the narrow sense heritability. This equation is sometimes termed the Breeders equation. It is a statement that the evolutionary change across generations (<span class="math inline">\(R\)</span>) is proportional to the change caused by directional selection within a generation, and the strength of this relationship is determined by the narrow sense heritability.<br />
Using the fact that <span class="math inline">\(h^2=V_A/V\)</span> we can rewrite this in a different form as <span class="math display">\[R= V_A \frac{S}{V}\]</span> i.e. our response to selection is the additive genetic variance of our trait (<span class="math inline">\(V_A\)</span>) multiplied by the change within a generation as a fraction of the total phenotypic variance (<span class="math inline">\(S/V\)</span>, sometimes called the the selection gradient <span class="math inline">\(\beta\)</span>).<br />
</p>
<p><embed src="figures/QT3.eps" /></p>
<p>A change in mean phenotype within a generation occurs because of the differential fitness of our organisms. To think more carefully about this change within a generation lets think about a simple fitness model where our phenotype affects the viability of our organisms (i.e. the probability they survive to reproduce). The probability that an individual has a phenotype <span class="math inline">\(X\)</span> before selection is <span class="math inline">\(p(X)\)</span>, so that the mean phenotype before selection is <span class="math display">\[\mu_{BS} = {\mathbb{E}}[X] =  \int_{-\infty}^{\infty} x p(x) dx\]</span> The probability that an organism with a phenotype <span class="math inline">\(X\)</span> survives to reproduce is <span class="math inline">\(w(X)\)</span>, and we’ll think about this as the fitness of our organism. The probability distribution of phenotypes in those who do reproduce is <span class="math display">\[{\mathbb{P}}(X | \textrm{survive}) =  \frac{p(x) w(x)}{
\int_{-\infty}^{\infty} p(x) w(x) dx}.\]</span> where the denominator is a normalization constant which ensures that our phenotypic distribution integrates to one. The denominator also has the interpretation of being the mean fitness of the population, which we’ll call <span class="math inline">\({\overline{w}}\)</span>, i.e. <span class="math display">\[{\overline{w}}=  
\int_{-\infty}^{\infty} p(x) w(x) dx.\]</span></p>
<p>Therefore, we can write the mean phenotype in those who survive to reproduce as <span class="math display">\[\mu_S = \frac{1}{{\overline{w}}}\int_{-\infty}^{\infty} x p(x) w(x) dx\]</span></p>
<p>If we mean center our population, i.e. set the phenotype before selection to zero, then <span class="math display">\[S= \frac{1}{{\overline{w}}}\int_{-\infty}^{\infty} x p(x) w(x) dx\]</span> if <span class="math inline">\(\mu_S=0\)</span>. Inspecting this more closely we can see that <span class="math inline">\(S\)</span> has the form of a covariance between our phenotype <span class="math inline">\(X\)</span> and our fitness <span class="math inline">\(w(X)\)</span> (<span class="math inline">\(Cov(X,w(X))\)</span>). Thus our change in mean phenotype is directly a measure of the covariance of our phenotype and our fitness. Rewriting our breeder’s equation using this observation we see <span class="math display">\[R = \frac{V_A}{V}  Cov(X,w(X))\]</span> we see that the response to selection is due to the fact that our fitness (viability) of our organisms/parents covaries with our phenotype, and that our child’s phenotype is correlated with the parent phenotype.</p>
<h4 id="the-response-of-multiple-traits-to-selection-the-multivariate-breeders-equation.">The response of multiple traits to selection, the multivariate breeder’s equation.</h4>
<p>We can generalize these results for multiple traits, to ask how selection on multiple phenotypes plays out over short time intervals. We’ll write our change in the mean our multiple phenotypes within a generation as the vector <span class="math inline">\(\bf{S}\)</span> and our response across multiple generations as the vector <span class="math inline">\(\bf{R}\)</span>. These two quantities are related by <span class="math display">\[\bf{R} = \bf{G} \bf{V}^{-1} \bf{S} = \bf{G} \bf{\beta}\]</span> where <span class="math inline">\(\bf{V}\)</span> and <span class="math inline">\(\bf{G}\)</span> are our matrices of the variance-covariance of phenotypes and additive genetic values (eqn. ) and <span class="math inline">\(\bf{\beta}\)</span> is a vector of selection gradients (i.e. the change within a generation as a fraction of the total phenotypic variance). To make this a bit more intuitive, consider two traits we are writing <span class="math display">\[\begin{aligned}
R_1 &amp; = V_{A,1} \beta_1 + V_{A,1,2} \beta_2 \nonumber \\
R_2 &amp; = V_{A,2} \beta_2 + V_{A,1,2} \beta_1  \nonumber \\\end{aligned}\]</span> where the <span class="math inline">\(1\)</span> and <span class="math inline">\(2\)</span> index our two different traits. This is a statement that our response in any one phenotype is modified by selection on other traits that covary with that trait. This offers a good way to think about how genetic trade offs play out over evolution over short time-scales.</p>
</body>
</html>
