<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
  <link rel="stylesheet" href="css/style.css">
  <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
</head>
<body>
<nav id="TOC">
<ul>
<li><a href="#the-effect-of-linked-selection-on-patterns-of-neutral-diversity">The effect of linked selection on patterns of neutral diversity</a><ul>
<li><a href="#a-simple-recurrent-model-of-selective-sweeps">A simple recurrent model of selective sweeps</a></li>
</ul></li>
</ul>
</nav>
<h1 id="the-effect-of-linked-selection-on-patterns-of-neutral-diversity">The effect of linked selection on patterns of neutral diversity</h1>
<p>A newly derived allele with an additive selection coefficient <span class="math inline">\(s\)</span> will take a time <span class="math inline">\(\tau \approx 2\log(2N)/s\)</span> generations to reach to fixation within our population. This short time window offers very little time for recombination between the selected site and linked neutral sites.<br />
First lets imagine examining variation at a locus fully linked to our selected locus, just after our sweep reached fixation. A pair of neutral alleles sampled at this locus must both trace their ancestral lineages back through to the neutral allele on whose background the selected allele initially arose. As that neutral allele, which existed <span class="math inline">\(\tau\)</span> generations ago is the ancestor of the entire population at this locus. Our individuals who carry the beneficial allele are, from the perspective of these two alleles, exactly like a rapidly expanding population. Therefore, our pair of neutral alleles sampled at our locus will be forced to coalesce <span class="math inline">\(\approx \tau\)</span> generations ago. This is a very short-time scale compared to the average neutral coalescent tie of <span class="math inline">\(2N\)</span> generations of a pair of alleles.<br />
If we now allow recombination into our model we can think about a pair of alleles sampled at a neutral locus a recombination distance <span class="math inline">\(r\)</span> away from our selected site. Our pair of alleles will be forced to coalesce <span class="math inline">\(\approx \tau\)</span> generations if neither of them reside on haplotypes that the selected allele recombined onto during the sweep. This is equivalent to saying that neither of our neutral alleles recombine off of the beneficial allele’s background moving backward in time.<br />
The probability that our lineage fail recombines off our beneficial allele’s background and onto the ancestral background in the <span class="math inline">\(j^{th}\)</span> generation back is <span class="math display">\[r (1-X(j))\]</span> so the probability (<span class="math inline">\(p_{NR}\)</span>) that our lineage fails to recombine off in the <span class="math inline">\(\tau\)</span> generations it takes our selected allele to move through the population is <span class="math display">\[p_{NR}=\prod_{j=1}^{\tau} \big(1- r(1-X(j))\big)\]</span> assuming that <span class="math inline">\(r\)</span> is small then <span class="math inline">\( \left(1- r(1-X(j))\right) \approx
e^{-r(1-X(j))}\)</span>, such that <span class="math display">\[p_{NR}=\prod_{j=1}^{\tau} \left(1- r(1-X(j))\right) \approx \exp
\left( -r\sum_{j=1}^{\tau}
1- X(j) \right) =\exp
\left( -r \tau (1-\widehat{X}) \right)\]</span> where <span class="math inline">\(\widehat{X}\)</span> is the average frequency of the derived allele across the trajectory <span class="math inline">\(\widehat{X} = \frac{1}{\tau}  \sum_{j=1}^{\tau}
 X(j)\)</span>. As our allele is additive its trajectory for frequencies <span class="math inline">\(&lt;0.5\)</span> is the mirror image of its trajectory for frequency <span class="math inline">\(&gt;0.5\)</span>, therefore it average frequency <span class="math inline">\(\widehat{X} =0.5\)</span>. So <span class="math display">\[p_{NR} = e^{-r \tau/2 }.\]</span> The probability that both of our lineages fail to recombine off the sweep and hence are forced to coalesce is <span class="math inline">\(p_{NR}^2\)</span>, assuming that they coalesce at a time close to <span class="math inline">\(\tau\)</span> so that they recombine independently of each other for times <span class="math inline">\(&lt; \tau\)</span>.<br />
If one or other of our lineages recombine off the sweep it will take them on average <span class="math inline">\(\approx 2N\)</span> generations to find a common ancestor as we are back our neutral coalescent. Thus the expected time till our pair of lineages find a common ancestor is <span class="math display">\[{\mathbb{E}}(T_2)  = \tau \times p_{NR}^2 +(1-p_{NR}^2) (\tau +2N) \approx
\left(1-p_{NR}^2 \right) 2N\]</span> where this last approximation assumes that <span class="math inline">\(\tau \ll 2N\)</span>. So the expected pairwise diversity for neutral alleles at a recombination distance <span class="math inline">\(r\)</span> away from the selected sweep (<span class="math inline">\(\pi_r\)</span>) is <span class="math display">\[{\mathbb{E}}(\pi_r) = 2\mu {\mathbb{E}}(T_2)  \approx \theta \left(1-e^{-r\tau} \right) \label{eqn:pi_HH}\]</span> so diversity increases as we move away from the selected site, slowly exponentially plauteuing to its neutral expectation <span class="math inline">\(\theta=4N\mu\)</span>.<br />
To get a sense of the physical scale over which diversity is reduced consider a region where recombination occurs at a rate <span class="math inline">\(r_{BP}\)</span> per base pair per generation, and our locus is <span class="math inline">\( \ell \)</span> base pairs away from the selected site <span class="math inline">\(r=r_{BP } \ell \)</span> (where <span class="math inline">\(r_{BP}  \ell  \ll 1\)</span> so we don’t need to worry about more than one recombination event occurring per generation). Typical recombination rates are on the order of <span class="math inline">\(r_{BP} = 10^{-8}\)</span>, in Figure [fig:hitchhiking_reduction] we show the reduction in diversity, given by eqn. , for two different selection coefficients.<br />
For our expected diversity levels to recover to <span class="math inline">\(50\%\)</span> of its neutral expectation <span class="math inline">\({\mathbb{E}}(\pi_r)/\theta=0.5\)</span>, requires a physical distance <span class="math inline">\(\ell^{*}\)</span> such that <span class="math inline">\(\log(0.5) = -r_{BP} \ell ^*\tau\)</span> as using our expression for <span class="math inline">\(\tau\)</span> then <span class="math inline">\( \ell^* = \frac{s}{r_{BP} \log (2N) }\)</span>. The width of our trough of reduced diversity depends on <span class="math inline">\(s/r_{BP}\)</span>, so else being equal we expect stronger sweeps or sweeps in regions of low recombination to have a larger hitchhiking effect. So that a selection coefficient of <span class="math inline">\(s=0.1\%\)</span> would reduce diversity over 10’s of kb, while a sweep of <span class="math inline">\(s=1\%\)</span> would affect <span class="math inline">\(\sim\)</span>100kb.<br />
</p>
<figure>
<embed src="figures/hitchhiking_reduction.eps" /><figcaption>Reduction in diversity compared to its neutral expectation as a function of the distance away from a site where a selected allele has just gone to fixation. The recombination rate is <span class="math inline">\(r_{BP}= 1\times
10^{-8}\)</span>.<span data-label="fig:hitchhiking_reduction"></span></figcaption>
</figure>
<h2 id="a-simple-recurrent-model-of-selective-sweeps">A simple recurrent model of selective sweeps</h2>
<p>We sample a pair of neutral alleles at a locus a genetic distance <span class="math inline">\(r\)</span> away from a locus where sweeps are initiated within the population at some very low rate <span class="math inline">\(\nu\)</span> per generation. The waiting time between sweeps at our locus is exponential <span class="math inline">\(\sim Exp(\nu)\)</span>. Each sweep rapidly transits through the population in <span class="math inline">\(\tau\)</span> generations, such that each sweep is finished long before the next sweep (<span class="math inline">\(\tau \ll 1/\nu\)</span>).<br />
As before our chance that our neutral lineage fails to recombine off the sweep is <span class="math inline">\(p_{NR}\)</span>, such that the probability that our pair of lineages are forced to coalesce by a sweep <span class="math inline">\(e^{-r \tau}\)</span>. Our lineages therefore have a very low probability <span class="math display">\[\nu e^{-r \tau}\]</span> of being forced to coalesce by a sweep per generation. In addition of lineages can coalesce at a neutral rate of <span class="math inline">\(1/(2N)\)</span>. Thus the average waiting time till a coalescent event between our neutral pair of lineages due to either a sweep or a neutral coalescent event is <span class="math display">\[{\mathbb{E}}(T_2) = \frac{1}{\nu e^{-r \tau} + 1/(2N)}\]</span></p>
<p>Now imagine that the sweeps don’t occur at a fixed location with respect to our locus of interest, but now occur uniformly at random across our sequence. The sweeps are initiated at a very low rate of <span class="math inline">\(\nu_{BP}\)</span> per basepair per generation. The rate of coalescent due to sweeps at a locus <span class="math inline">\(\ell\)</span> basepairs away from our neutral loci is <span class="math inline">\(\nu_{BP} e^{-r_{BP} \ell \tau}\)</span>. If our neutral locus is in the middle of a chromosome that stretches <span class="math inline">\(L\)</span> basepairs in either direction the total rate of sweeps per generation that force our pair of lineages to coalesce is <span class="math display">\[2\int_0^{L} \nu_{BP} e^{-r_{BP} \ell \tau} d \ell =
\frac{2\nu_{BP}}{r_{BP} \tau} \left(1-e^{-r_{BP} \tau L} \right)\]</span> so that if <span class="math inline">\(L\)</span> is very large (<span class="math inline">\(r_{BP} \tau L \gg 1\)</span>) the rate of coalesce per generation due to sweeps is <span class="math inline">\(\frac{2\nu_{BP}}{r_{BP} \tau}\)</span>. The total rate of coalescence for a pair of lineages per generation is then <span class="math display">\[\frac{2\nu_{BP}}{r_{BP} \tau}+\frac{1}{2N}\]</span> So our average time till a pair of lineages coalesce is <span class="math display">\[{\mathbb{E}}(T_2) = \frac{1}{\frac{2\nu_{BP}}{r_{BP} \tau}+\frac{1}{2N}} = \frac{r_{BP}2N}{\frac{4N\nu_{BP}}{ \tau}+r_{BP}}\]</span> such that our expected pairwise diversity (<span class="math inline">\(\pi=2\mu{\mathbb{E}}(T_2)\)</span>) in a region of recombination rate <span class="math inline">\(r_{BP}\)</span> that experiences sweeps at rate <span class="math inline">\(\nu_{BP}\)</span> is <span class="math display">\[{\mathbb{E}}(\pi) = \theta \frac{r_{BP}}{\frac{4N\nu_{BP}}{ \tau}+r_{BP}} \label{eqn:pi_GW_HH}\]</span></p>
<figure>
<embed src="figures/Genomewide_HH.eps" /><figcaption>The relationship between (sex-averaged) recombination rate and synonymous site pairwise diversity (<span class="math inline">\(\pi\)</span>) in <span><em>Drosophila melanogaster</em></span> using the data of Shapiro et al. 07 (kindly provided by Peter Andolfatto, see Sella et al. 09 for details). The curve is the predicted relationship between <span class="math inline">\(\pi\)</span> and recombination rate obtained by fitting equation to this data using non-linear least squares via the <span>nls()</span> function in <span>R</span>.<span data-label="fig:GW_hitchhiking_reduction"></span></figcaption>
</figure>
</body>
</html>
