<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
  <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
</head>
<body>
<h1 id="genetic-drift-and-neutral-alleles">Genetic drift and Neutral alleles</h1>
<h2 id="the-fixation-of-neutral-alleles">The fixation of neutral alleles</h2>
<p>It is very unlikely that a rare neutral allele accidentally drifts up to fixation, it is much more likely that such an allele is eventually lost from the population. However, there is a large and constant influx of rare alleles into the population due to mutation, so even if it is very unlikely that an individual allele fixes within the population, some neutral alleles will fix.<br />
</p>
<h4 id="probability-of-the-eventual-fixation-of-a-neutral-allele.">Probability of the eventual fixation of a neutral allele.</h4>
<p>An allele which reaches fixation within a population, is an ancestor to the entire population. In a particular generation there can be only single allele that all other alleles at the locus in later generation can claim as an ancestor. As at a neutral locus all of our alleles are exchangeable, as they have no effect on the number of descendents an individual leaves, so any allele is equally likely to be the ancestor of the entire population. In a diploid population size of size <span class="math inline">\(N\)</span>, there are <span class="math inline">\(2N\)</span> alleles all of which are equally likely to be the ancestor of the entire population at some later time point. So if our allele is present in a single copy, the chance that is the ancestor to the entire population in some future generation is <span class="math inline">\(1/(2N)\)</span>, i.e. the chance our neutral allele is eventually fixed is <span class="math inline">\(1/(2N)\)</span>.<br />
More generally if our neutral allele is present in <span class="math inline">\(i\)</span> copies in the population, of <span class="math inline">\(2N\)</span> alleles, the probability that this allele is fixed is <span class="math inline">\(i/(2N)\)</span>. I.e. the probability that a neutral allele is eventually fixed is simply given by its frequency (<span class="math inline">\(p\)</span>) in the population. We can also derive this result by letting <span class="math inline">\(Ns \rightarrow
0\)</span> in eqn. .</p>
<h4 id="rate-of-substitution-of-neutral-alleles.">Rate of substitution of neutral alleles.</h4>
<p>A substitution between populations that do not exchange gene flow is simply a fixation event within one population. The rate of substitution is therefore the rate at which new alleles fix in the population, so that the long-term substitution rate is the rate at which mutations arise that will eventually become fixed within our population.<br />
Assume that there are two classes of mutational changes that can occur with a region, highly deleterious mutations and neutral mutations. A fraction <span class="math inline">\(C\)</span> of all mutational changes are highly deleterious, and can not possibly contribute to substitution nor polymorphism (i.e. <span class="math inline">\(Ns \gg 1\)</span>). While a fraction <span class="math inline">\(1-C\)</span> are neutral. If our mutation rate is <span class="math inline">\(\mu\)</span> per transmitted allele per generation, then a total of <span class="math inline">\(2N \mu (1-C)\)</span> neutral mutations enter our population each generation.<br />
Each of these neutral mutations has a <span class="math inline">\(1/(2N)\)</span> probability chance of eventually becoming fixed in the population. Therefore, the rate at which neutral mutations arise that eventually become fixed within our population is <span class="math display">\[2N\mu(1-C)\frac{1}{2N} = \mu(1-C)\]</span> thus the rate of substitution under a model where newly arising alleles are either highly deleterious or neutral, is simply given by the mutation rate towards neutral alleles, i.e. <span class="math inline">\(\mu(1-C)\)</span>.<br />
Consider a pair of species have diverged for <span class="math inline">\(T\)</span> generations, i.e. orthologous sequences shared between the species last shared a common ancestor <span class="math inline">\(T\)</span> generations ago. If they have maintained a constant <span class="math inline">\(\mu\)</span> over that time, will have accumulated an average of <span class="math display">\[2\mu(1-C)T\]</span> neutral substitutions. This assumes that <span class="math inline">\(T\)</span> is a lot longer than the time it takes to fix a neutral allele, such that the total number of alleles introduced into the population that will eventually fix is the total number of substitutions. We’ll see below that a neutral allele takes on average <span class="math inline">\(4N\)</span> generations to fix from its introduction into the population.<br />
This is a really pretty result as the population size has completely canceled out of the neutral substitution rate. However, there is another way to see this in a more straightward way. If I look at a sequence in me compared to say a particular chimp, I’m looking at the mutations that have occurred in both of our germlines since they parted ways <span class="math inline">\(T\)</span> generations ago. Since neutral alleles do not alter the probability of their transmission to the next generation, we are simply looking at the mutations that have occurred in <span class="math inline">\(2T\)</span> generations worth of transmissions. Thus the average number of neutral mutational differences separating our pair of species is simply <span class="math inline">\(2\mu (1-C) T\)</span>.<br />
</p>
<h2 id="the-coalescent-and-patterns-of-neutral-diversity">The Coalescent and patterns of neutral diversity</h2>
<h4 id="pairwise-coalescent-time-distribution-and-the-number-of-pairwise-differences.">Pairwise Coalescent time distribution and the number of pairwise differences.</h4>
<p>Thinking back to our calculations we made about the loss of neutral heterozygosity and equilibrium levels of diversity (in Sections [LossofHet] and [DriftMutationBalance]), you’ll note that we could first specify what generation a pair of sequences coalesce in, and then calculate some properties of heterozygosity based on that. That’s because neutral mutations do not affect the probability that an individual transmits that allele, so don’t affect the way in which we can trace ancestral lineages back.<br />
As such it will often be helpful to consider the time to the common ancestor of a pair of sequences, and then think of the impact of that on patterns of diversity. See Figure [fig:Coalescent_simulation] for an example of this. The probability that a pair of alleles have failed to coalesce in <span class="math inline">\(t\)</span> generations and then coalesce in the <span class="math inline">\(t+1\)</span> generation back is <span class="math display">\[\frac{1}{2N} (1- \frac{1}{2N} )^{t} \approx \frac{1}{2N} e^{-t/(2N)} \label{eqn:coal_time_dist}\]</span> thus the coalescent time of a pair of sequences (<span class="math inline">\(T_2\)</span>) is approximately exponentially distributed with a rate <span class="math inline">\(1/(2N)\)</span>. We’ll denote that by saying that <span class="math inline">\(T_2 \sim Exp \left( 1/(2N) \right)\)</span>. The mean coalescent time of a pair of a pair of alleles is <span class="math inline">\(2N\)</span> generations<br />
</p>
<figure>
<embed src="figures/Coalescent.eps" /><figcaption>A simple simulation of the coalescent process. The simulation consists of a diploid population of 10 individuals (20 alleles). In each generation, each individual is equally likely to be the parent of an offspring (and the allele transmitted is indicated by a light grey line). We track a pair of alleles, chosen in the present day, back 14 generations untill they find a common ancestor.<span data-label="fig:Coalescent_simulation"></span></figcaption>
</figure>
<p>Conditional on a pair of alleles coalescing <span class="math inline">\(t\)</span> generations ago there are <span class="math inline">\(2t\)</span> generations in which a mutation could occur. Thus the probability of our pair of alleles are separated by <span class="math inline">\(j\)</span> mutations since they last shared a common ancestor is <span class="math display">\[P(j | T_2 = t ) = {2t \choose j} \mu^{j} (1-\mu)^{2t-j}\]</span> i.e. mutations happen in <span class="math inline">\(j\)</span> generations, and do not happen in <span class="math inline">\(2t-j\)</span> generations (with <span class="math inline">\({2t \choose j}\)</span> ways this can possibly happen). Assuming that <span class="math inline">\(\mu \ll 1\)</span>, and that <span class="math inline">\(2t-j \approx 2t\)</span> then we can approximate the probability that we have <span class="math inline">\(j\)</span> mutations as a Poisson distribution <span class="math display">\[P(j | T_2 = t ) = \frac{ (2 \mu t )^{j} e^{-2\mu t}}{j!}\]</span> i.e. a Poisson with mean <span class="math inline">\(2\mu t \)</span>.<br />
As our expected coalescent time is <span class="math inline">\(2N\)</span> generations, the expected number of mutations separating two alleles drawn at random from the population is <span class="math display">\[{\mathbb{E}}(j) = 4N\mu = \theta\]</span> We’ll assume that mutations never happen at the same site twice, i.e. no multiple hits, such that we get to see all of the mutation events that separate our pair of sequences (we’ll call this the infinitely-many-sites assumption, which should be fine if <span class="math inline">\(N\mu_{BP} \ll 1\)</span>). Thus the number of mutations between a pair of sites is the observed number of differences between a pair of sequences.<br />
We’ll denote the observed number of pairwise differences at putatively neutral sites separating a pair of sequences as <span class="math inline">\(\pi\)</span> (we usually average this over a number of pairs of sequences for a region). So we can estimate of <span class="math inline">\(\theta\)</span> from <span class="math inline">\(\pi\)</span>, <span class="math inline">\(\widehat{\theta}_{\pi}\)</span> by setting <span class="math inline">\(\widehat{\theta}_{\pi}=\pi\)</span>. If we have an independent estimate of <span class="math inline">\(\mu\)</span>, then from setting <span class="math inline">\(\pi = \widehat{\theta}_{\pi} = 4N\mu\)</span> we can obtain an estimate of the population size <span class="math inline">\(N\)</span> that is consistent with our levels of neutral polymorphism.</p>
<h2 id="the-coalescent-process-of-a-sample-of-alleles.">The coalescent process of a sample of alleles.</h2>
<p>Usually we are not just interested pairs of alleles, or the average pairwise diversity, we are interested in the properties of diversity in samples of a number of alleles drawn from the population. To allow for this instead of just following a pair of lineages back until they coalesce, we can follow the history of a sample of alleles back through the population.</p>
<p>Consider first sampling three alleles at random from the population. The probability that all three alleles choose exactly the same ancestral allele one generation back is <span class="math inline">\(1/(2N)^2\)</span>. If <span class="math inline">\(N\)</span> is reasonably large then this is a very small probability. As such it is very unlikely that our three alleles coalesce at once, a in a moment we’ll see that it is safe to ingnore such unlikely events.<br />
The probability that a specific pair of alleles find a common ancestor in the preceding generation is still <span class="math inline">\(1/(2N)\)</span>. There are three possible pairs of alleles so the probability that no pair finds a common ancestor is <span class="math display">\[\left(1-\frac{1}{(2N)} \right)^3 \approx \left( 1- \frac{3}{2N} \right)\]</span> in making this approximation we are multiplying out the right hand-side and ignoring terms of <span class="math inline">\(1/N^2\)</span> and higher. See Figure [fig:Coalescent_simulation_3] for a random realization of this process.<br />
</p>
<figure>
<embed src="figures/Coalescent_3.eps" /><figcaption>A simple simulation of the coalescent process for three lineages. We track the ancestry of three modern-day alleles, the first pair (blue and purple) coalesce four generations back their are then two independent lineages we are tracking, this pair then coalesces twelve generations in the past. Note that different random realizations of this process will differ from each other a lot.<span data-label="fig:Coalescent_simulation_3"></span></figcaption>
</figure>
<p>More generally when we sample <span class="math inline">\(i\)</span> alleles there are <span class="math inline">\({i \choose
 2}\)</span> pairs, i.e. <span class="math inline">\(i(i-1)/2\)</span> pairs, thus the probability that no pair of alleles coalesces in the preceding generation is <span class="math display">\[\left(1-\frac{1}{(2N)} \right)^{{i \choose
 2}} \approx \left( 1- \frac{{i \choose
 2}}{2N}\right)\]</span> while the probability of any pair coalescing is <span class="math inline">\(\approx \frac{{i \choose
 2}}{2N}\)</span>.<br />
We can ignore the possibility of more than pairs of alleles (e.g. tripletons) simultaneously coalescing at once as terms of <span class="math inline">\(1/N^2\)</span> and higher can be ignored as they are vanishingly rare. Obviously there are in reasonable sample sizes there are many more triples (<span class="math inline">\({i \choose 3}\)</span>), and higher order combinations, than pairs (<span class="math inline">\({i \choose 2}\)</span>) but if <span class="math inline">\(i \ll N\)</span> then we are safe to ignore these terms.</p>
<p>When there are <span class="math inline">\(i\)</span> alleles the probability that we wait until the <span class="math inline">\(t+1\)</span> generation before any pair of alleles coalesce is <span class="math display">\[\frac{{i \choose
 2}}{2N}\left( 1- \frac{{i \choose
 2}}{2N}\right)^{t} \approx  \frac{{i \choose
 2}}{2N} \exp \left( - \frac{{i \choose
 2}}{2N} t \right)\]</span> thus the waiting time <span class="math inline">\(T_i\)</span> to the first coalescent event in a sample of <span class="math inline">\(i\)</span> alleles is exponentially distributed with rate <span class="math inline">\(\frac{{i \choose
 2}}{2N}\)</span>, i.e. <span class="math inline">\(T_i \sim Exp\left(\frac{{i \choose
 2}}{2N} \right)\)</span>. The mean waiting time till any of pair within our sample to coalesce is <span class="math inline">\(2N/{i \choose
 2}\)</span>.<br />
When a pair of alleles first find a common ancestral allele some number of generations back further into the past we only have to keep track of that common ancestral allele for the pair. Thus when a pair of alleles in our sample of <span class="math inline">\(i\)</span> alleles coalesce, we then switch to having to follow <span class="math inline">\(i-1\)</span> alleles back. Then when a pair of these <span class="math inline">\(i-1\)</span> alleles coalesce, we then have to follow <span class="math inline">\(i-2\)</span> alleles back. This process continues until we coalesce back to a sample of two, and from there to a single most recent common ancestor (MRCA).<br />
To simulate a coalescent genealogy at a locus for a sample of <span class="math inline">\(n\)</span> alleles we therefore simply follow this algorithm</p>
<ol>
<li><p>set <span class="math inline">\(i=n\)</span>.</p></li>
<li><p>We simulate a random variable to be the time <span class="math inline">\(t_i\)</span> to the next coalescent event from <span class="math inline">\(t_i \sim
 Exp\left(\frac{{i \choose
 2}}{2N} \right)\)</span></p></li>
<li><p>choose a pair of alleles to coalesce at random from all possible pairs.</p></li>
<li><p>set <span class="math inline">\(i=i-1\)</span></p></li>
<li><p>continue looping of steps 1-3 until <span class="math inline">\(i=1\)</span> i.e. the most recent common ancestor of the sample is found.</p></li>
</ol>
<p>by following this algorithm we are generating realizations of the genealogy of our sample.<br />
We will first consider the time to the most recent common ancestor of the entire sample (<span class="math inline">\(T_{MRCA}\)</span>). This is <span class="math display">\[T_{MRCA} = \sum_{i=n}^2 T_i\]</span> generations back. As our coalescent times for different <span class="math inline">\(i\)</span> are independent, the expected time to the most recent common ancestor is <span class="math display">\[{\mathbb{E}}(T_{MRCA}) = \sum_{i=n}^2 {\mathbb{E}}(T_i) = \sum_{i=n}^2  2N/{i \choose
 2}\]</span> using the fact that <span class="math inline">\(\frac{1}{i(i-1)}=\frac{1}{i-1} - \frac{1}{i}\)</span> with a bit of rearrangement we can rewrite this is <span class="math display">\[{\mathbb{E}}(T_{MRCA}) = 4N\left(1- \frac{1}{n} \right)\]</span> so the average <span class="math inline">\(T_{MRCA}\)</span> scales linearly with population size. Interestingly, as we move to larger and larger samples (i.e. <span class="math inline">\(n \gg 1\)</span>) the average time to the most recent common ancestor is converging on <span class="math inline">\(4N\)</span>. What’s happening here is that in large samples our lineages typically coalesce rapidly at the start and very soon coalesce down to a much smaller number of lineages.<br />
Above we argued that a mutation is only becomes a fixed difference if it is lucky enough to be the ancestor of the entire population. As we saw above this occurs with probability <span class="math inline">\(1/(2N)\)</span>. How long does is take on average for such an allele to fix within our population. We’ve just seen that it takes <span class="math inline">\(4N\)</span> generations for a large sample of alleles to all trace their ancestry back to a single most recent common ancestor. Thus it must take roughly <span class="math inline">\(4N\)</span> generations for a neutral allele present in a single copy within the the population to the ancestor of all alleles within our population. This argument can be made more precise, but in general we would still find that it takes <span class="math inline">\(\approx 4N\)</span> generations for a neutral allele to go from its introduction to fixation with the population.<br />
The total amount of time in the genealogy (<span class="math inline">\(T_{tot}\)</span>) <span class="math display">\[T_{tot} = \sum_{i=n}^2 iT_i\]</span> as when there are <span class="math inline">\(i\)</span> lineages each contributes a time <span class="math inline">\(T_i\)</span> to the total time. Taking the expectation of the total time in the genealogy <span class="math display">\[{\mathbb{E}}(T_{tot}) = \sum_{i=n}^2 i \frac{2N}{{i \choose
 2} } = \sum_{i=n}^2 \frac{4N}{i -1} =\sum_{i=n-1}^1 \frac{4N}{i}\]</span> so our expected total amount of time in the genealogy scales linearly with our population size. Our expected total amount of time is also increasing with sample size but is doing so very slowly. To see this more carefully we can see that for large <span class="math inline">\(n\)</span> <span class="math display">\[{\mathbb{E}}(T_{tot}) = \sum_{i=n-1}^1 \frac{4N}{i} \approx 4N \int_1^n \frac{1}{i} di
= 4N \log(n-1)\]</span> here we are approximating our sum by an integral, which will work for large <span class="math inline">\(n\)</span>. So our expected total amount of time in the genealogy is growing with <span class="math inline">\(n\)</span> but it is doing so very slowly. This again follows from the fact that in large samples the initial coalescence usually happens very rapidly, so that extra samples adds little to the total amount of time in the tree.<br />
We saw above that the number of mutational differences between a pair of alleles that coalescence <span class="math inline">\(T_2\)</span> generations ago was Poisson with a mean of <span class="math inline">\(2 \mu T_2\)</span>. A mutation that occurs on any branch of our genealogy will cause a segregating polymorphism in the sample (making our infinitely-many-sites assumption). Thus if the total time in the genealogy is <span class="math inline">\(T_{tot}\)</span> there is <span class="math inline">\(T_{tot}\)</span> generations for mutations. So the total number of mutations segregating in our sample (<span class="math inline">\(S\)</span>) is Poisson with mean <span class="math inline">\(\mu T_{tot}\)</span>. Thus the expected number of segregting in history a sample of size <span class="math inline">\(n\)</span> is <span class="math display">\[{\mathbb{E}}(S) = \mu {\mathbb{E}}(T_{tot}) = \sum_{i=n-1}^1 \frac{4N\mu }{i} = \theta
\sum_{i=n-1}^1 \frac{1}{i}\]</span> Thus we can use this formula to derive another estimate of the population scaled mutation rate, by setting our observed number of segregating sites in a sample (<span class="math inline">\(S\)</span>) equal to this expectation. We’ll call this estimator <span class="math inline">\(\widehat{\theta}_W\)</span> <span class="math display">\[\widehat{\theta}_W =\frac{ S}{\sum_{i=n-1}^1 \frac{1}{i}}\]</span> this estimator was devised by Watterson, hence the <span class="math inline">\(W\)</span>.</p>
<h2 id="comparing-polymorphism-and-divergence">Comparing polymorphism and divergence</h2>
<h2 id="deviations-from-the-constant-population-model.">Deviations from the constant population model.</h2>
<p>We’ve seen previously that changes in our population size can be captured by an effective population size. However, this will only be a useful measure if population sizes vary rapidly enough, that the harmonic mean effective population size over short time periods (<span class="math inline">\(\ll
N_e\)</span> generations) is representative of the effective population size averaged over longer time periods. If this is not the case there is no one effective population size, as we can not approximate our rate of drift by a single constant population. Furthermore, we’ve ignored the effect of population structure and selection which will violate our modeling assumptions.<br />
We can hope to detect violations from our constant population size neutral model, by comparing aspects of our dataset to their expectations and distributions under our neutral model.<br />
For example we have devised two estimates of <span class="math inline">\(\theta\)</span>, <span class="math inline">\(\widehat{\theta_{\pi}}\)</span> and <span class="math inline">\(\widehat{\theta_{W}}\)</span>, using expectations of different aspects of our data (pairwise diversity and number of segregating sites respectively). Under our constant neutral model if we have sufficient data those two estimates should be equal to each other on average. But if there’s some violation of our model they might not be. So one test statistic might be to take <span class="math display">\[D = \widehat{\theta_{\pi}} - \widehat{\theta_{W}}\]</span> which will be zero in expectation if our data was generated by a neutral constant population model.</p>
</body>
</html>
