<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <title>Notes on Population Genetics</title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
  <script src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
</head>
<body>
<header>
<h1 class="title">Notes on Population Genetics</h1>
</header>
<h1 id="allele-and-genotype-frequencies">Allele and genotype frequencies</h1>
<h2 id="allele-frequencies">Allele frequencies</h2>
<p>Consider a diploid autosomal locus segregating at two alleles (<span class="math">\(A_1\)</span> and <span class="math">\(A_2\)</span>). Let <span class="math">\(N_{11}\)</span> and <span class="math">\(N_{12}\)</span> be the number of <span class="math">\(A_1A_1\)</span> homozygotes and <span class="math">\(A_1A_2\)</span> heterozygotes, respectively. Moreover, let <span class="math">\(N\)</span> be the total number of diploid individuals in the population. We can then define the relative frequencies of <span class="math">\(A_1A_1\)</span> and <span class="math">\(A_1A_2\)</span> genotypes as <span class="math">\(f_{11} = N_{11}/N\)</span> and <span class="math">\(f_{12} = N_{12}/N\)</span>, respectively. The frequency of allele <span class="math">\(A_1\)</span> in the population is then given by</p>
<p><span class="math">\[p = \frac{2 N_{11} + N_{12}}{2N} = f_{11} + \frac{1}{2} f_{12}.\]</span></p>
<p>Note that this holds independently of Hardy–Weinberg proportions and equilibrium [see below]. The frequency of the alternate allele (<span class="math">\(A_2\)</span>) is then just <span class="math">\(q=1-p\)</span>.</p>
<h2 id="hardyweinberg-proportions">Hardy–Weinberg proportions</h2>
<p>Imagine a population mating at random with respect to genotypes, i.e.no inbreeding, no population structure, no sex differences in allele frequencies. The frequency of allele <span class="math">\(A_1\)</span> in the population at the time of reproduction is <span class="math">\(p\)</span>. An <span class="math">\(A_1A_1\)</span> genotype is made by reaching out into our population and independently drawing two <span class="math">\(A_1\)</span> alelle gametes to form a zygote. Therefore, the probability that an individual is an <span class="math">\(A_1A_1\)</span> homozygote is <span class="math">\(p^2\)</span>. This probability is also the expected frequency of the <span class="math">\(A_1A_1\)</span> homozygote in the population. The expected frequency of the three possible genotypes is</p>
<table>
<thead>
<tr class="header">
<th style="text-align: center;"><span class="math">\(f_{11}\)</span></th>
<th style="text-align: center;"><span class="math">\(f_{12}\)</span></th>
<th style="text-align: center;"><span class="math">\(f_{22}\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"><span class="math">\(p^2\)</span></td>
<td style="text-align: center;"><span class="math">\(2pq\)</span></td>
<td style="text-align: center;"><span class="math">\(q^2\)</span></td>
</tr>
</tbody>
</table>
<p>.</p>
<p>Note that we only need to assume random mating with respect to our alleles in order for these expected frequencies to hold, as long at <span class="math">\(p\)</span> is the frequency of the <span class="math">\(A_1\)</span> allele in the population at the time when gametes fuse.</p>
<p><embed src="Figs/CEU_YRI_separately_HWE.eps" /></p>
<p><embed src="Figs/CEU_YRI_together_HWE.eps" /></p>
<h2 id="coefficient-of-kinship">Coefficient of kinship</h2>
<p>We will define two alleles to be identical by descent if they are identical due to a common ancestor in the past few generations. For the moment, we ignore mutation, and we will be more precise about what we mean by ‘past few generations’ later on. For example, parent and child share exactly one allele identical by descent at a locus, assuming that the two parents of the child are randomly mated individuals from the population.<br /></p>
<p>A key quantity is the probability that our pair of individuals share 0, 1, or 2 alleles identical by descent. We denote these probabilities by <span class="math">\(r_0\)</span>, <span class="math">\(r_1\)</span>, and <span class="math">\(r_2\)</span> respectively. See Table [table:IBDprobs] for some examples.<br /></p>
<p>One summary of relatedness that will be important is the probability that two alleles picked at random, one from each of the two different individuals <span class="math">\(i\)</span> and <span class="math">\(j\)</span>, are identical by descent. We call this quantity the coefficient of kinship of individuals <span class="math">\(i\)</span> and <span class="math">\(j\)</span>, and denote it by <span class="math">\(F_{ij}\)</span>. It is calculated as</p>
<p><span class="math">\[F_{ij}= 0 \times r_0 + \frac{1}{4} r_1  + \frac{1}{2} r_2. 
\label{eqn:coeffkinship}\]</span></p>
<p>The coefficient of kinship will appear multiple times, in both our discussion of inbreeding and in the context of phenotypic resemblance between relatives.<br /></p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Relationship (i,j)<span class="math">\(^{*}\)</span></th>
<th style="text-align: center;"><span class="math">\(r_0\)</span></th>
<th style="text-align: center;"><span class="math">\(r_1\)</span></th>
<th style="text-align: center;"><span class="math">\(r_2\)</span></th>
<th style="text-align: center;"><span class="math">\(F_{ij}\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">parent–child</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">1/4</td>
</tr>
<tr class="even">
<td style="text-align: left;">full siblings</td>
<td style="text-align: center;">1/4</td>
<td style="text-align: center;">1/2</td>
<td style="text-align: center;">1/4</td>
<td style="text-align: center;">1/4</td>
</tr>
<tr class="odd">
<td style="text-align: left;">identical (monzygotic) twins</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">1/2</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math">\(1^{st}\)</span> cousins</td>
<td style="text-align: center;">3/4</td>
<td style="text-align: center;">1/4</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">1/16</td>
</tr>
</tbody>
</table>
<p>[table:IBDprobs]</p>
<h2 id="inbreeding">Inbreeding</h2>
<p>We can define an inbred individual as an individual whose parents are more closely related to each other than two random individuals drawn from some reference population.<br /></p>
<p>When two related individuals produce an offspring, that individual can receive two alleles that are identical by descent, i.e. they can be homozygous by descent (sometimes termed autozygous), due to the fact that they have two copies of an allele through different paths through the pedigree. This increased likelihood of being homozygous relative to an outbred individual is the most obvious effect of inbreeding. It is also the one that will be of most interest to us, as it underlies a lot of our ideas about inbreeding depression and population structure.<br /></p>
<p>As the offspring receives a random allele from each parent (<span class="math">\(i\)</span> and <span class="math">\(j\)</span>), the probability that those two alleles are identical by descent is equal to the kinship coefficient <span class="math">\(F_{ij}\)</span> of the two parents (Eqn. [eqn:coeffkinship]). This follows from the fact that the genotype of the offspring is made by sampling an allele at random from each of our parents. We will use IBD for identical by descent.<br /></p>
<p>The only way the offspring can be heterozygous (<span class="math">\(A_1 A_2\)</span>) is if their two alleles at a locus are not IBD (otherwise they would necessarily be homozygous). Therefore, the probability that they are heterozygous is</p>
<p><span class="math">\[(1-F) 2p q,
\label{eq:hetGenHW}\]</span></p>
<p>where we have dropped the indices <span class="math">\(i\)</span> and <span class="math">\(j\)</span> for simplicity. The offspring can be homozygous for the <span class="math">\(A_1\)</span> allele in two different ways. They can have two non-IBD alleles that are not IBD but happen to be of the allelic type <span class="math">\(A_1\)</span>, or their two alleles can be IBD, such that they inherited allele <span class="math">\(A_1\)</span> by two different routes from the same ancestor. Thus, the probability that an offspring is homozygous is</p>
<p><span class="math">\[(1-F) p^2 + F p.\]</span></p>
<p>Therefore, the frequencies of the three possible genotypes can be written as given in Table [table:GeneralizedHWE], which provides a generalization of the Hardy–Weinberg proportions.<br /></p>
<table>
<thead>
<tr class="header">
<th style="text-align: center;"><span class="math">\(f_{11}\)</span></th>
<th style="text-align: center;"><span class="math">\(f_{12}\)</span></th>
<th style="text-align: center;"><span class="math">\(f_{22}\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"><span class="math">\((1-F) p^2 + F p\)</span></td>
<td style="text-align: center;"><span class="math">\((1-F) 2pq\)</span></td>
<td style="text-align: center;"><span class="math">\((1-F) q^2 + F q\)</span></td>
</tr>
</tbody>
</table>
<p>[table:GeneralizedHWE]</p>
<p>Note that the generalized Hardy–Weinberg proportions completely specify the genotype probabilities, as there are two parameters (<span class="math">\(p\)</span> and <span class="math">\(F\)</span>) and two degrees of freedom (as <span class="math">\(p\)</span> and <span class="math">\(q\)</span> have to sum to one). Therefore, any combination of genotype frequencies at a biallelic site can be specified by a combination of <span class="math">\(p\)</span> and <span class="math">\(F\)</span>.<br /></p>
<h2 id="calculating-inbreeding-coefficients-from-data">Calculating inbreeding coefficients from data</h2>
<p>If the observed heterozygosity in a population is <span class="math">\(H_O\)</span>, and we assume that the generalized Hardy–Weinberg proportions hold, we can set <span class="math">\(H_O\)</span> equal to <span class="math">\(f_{12}\)</span>, and solve Eq.  for <span class="math">\(F\)</span> to obtain an estimate of the inbreeding coefficient as</p>
<p><span class="math">\[\hat{F} = 1-\frac{f_{12}}{2pq} = \frac{2pq - f_{12}}{2pq}.
\label{eqn:Fhat}\]</span></p>
<p>As before, <span class="math">\(p\)</span> is the frequency of allele <span class="math">\(A_{1}\)</span> in the population. This can be rewritten in terms of the observed heterozygosity (<span class="math">\(H_O\)</span>) and the heterozygosity expected in the absence of inbreeding, <span class="math">\(H_E=2pq\)</span>, as</p>
<p><span class="math">\[\hat{F} = \frac{H_E-H_O}{H_E} = 1 - \frac{H_O}{H_E}.
\label{eqn:FhatHO}\]</span></p>
<p>Hence, <span class="math">\(F\)</span> quantifies the deviation due to inbreeding of the observed heterozygosity from the one expected under random mating, relative to the latter. If we have multiple loci, we can replace <span class="math">\(H_O\)</span> and <span class="math">\(H_E\)</span> by their means over loci, <span class="math">\(\bar{H}_O\)</span> and <span class="math">\(\bar{H}_E\)</span>, respectively. Note that, in principle, we could also calculate <span class="math">\(F\)</span> for each individual locus first, and then take the average across loci. However, this procedure is more prone to introducing a bias if sample sizes vary across loci, which is not unlikely when we are dealing with real data.<br /></p>
<h2 id="summarizing-population-structure">Summarizing population structure</h2>
<p>We defined inbreeding as having parents that are more closely related to each other than two individuals drawn at random from some reference population. The question that naturally arises is: Which reference population should we use? While I might not look inbred in comparison to allele frequencies in the United Kingdom (UK), where I am from, my parents certainly are not two individuals drawn at random from the world-wide population. If we estimated my inbreeding coefficient <span class="math">\(F\)</span> using allele frequencies within the UK, it would be close to zero, but would likely be larger if we used world-wide frequencies. This is because there is a somewhat lower level of expected heterozygosity within the UK than in the human population across the world as a whole.<br /></p>
<p>Wright (1943, 1951) developed a set of ‘F-statistics’ (also called ‘fixation indices’) that formalize the idea of inbreeding with respect to different levels of population structure. He defined <span class="math">\(F_{\mathrm{XY}}\)</span> as the correlation between random gametes, drawn from the same level <span class="math">\(X\)</span>, relative to level <span class="math">\(Y\)</span>. We will return to why <span class="math">\(F\)</span>-statistics are statements about correlations between alleles in just a moment. One commonly uses <span class="math">\({F_{\mathrm{IS}}}\)</span> for the inbreeding coefficient between an individual (<span class="math">\(I\)</span>) and the subpopulation (<span class="math">\(S\)</span>). Consider a single locus, where in a subpopulation (<span class="math">\(S\)</span>) a fraction <span class="math">\(H_I=f_{12}\)</span> of individuals are heterozygous. In this subpopulation, let the frequency of allele <span class="math">\(A_1\)</span> be <span class="math">\(p_S\)</span>, such that the expected heterozygosity under random mating is <span class="math">\(H_S = 2 p_S (1 - p_S)\)</span>. We will write <span class="math">\({F_{\mathrm{IS}}}\)</span> as</p>
<p><span class="math">\[{F_{\mathrm{IS}}}= 1-\frac{H_I}{H_S}= 1-\frac{f_{12}}{2p_Sq_S},
\label{eqn:FIS}\]</span></p>
<p>a direct analog of eqn. [eqn:Fhat]. Hence, <span class="math">\({F_{\mathrm{IS}}}\)</span> is the relative difference between observed and expected heterozygosity due to a deviation from random mating within the subpopulation. We could also compare the observed heterozygosity in individuals (<span class="math">\(H_I\)</span>) to that expected in the total population, <span class="math">\(H_T\)</span>. If the frequency of allele <span class="math">\(A_1\)</span> in the total population is <span class="math">\(p_T\)</span>, then we can write <span class="math">\({F_{\mathrm{IT}}}\)</span> as</p>
<p><span class="math">\[{F_{\mathrm{IT}}}=1-\frac{H_I}{H_T}= 1-\frac{f_{12}}{2p_Tq_T},
\label{eqn:FIT}\]</span></p>
<p>which compares heterozygosity in individuals to that expected in the total population. As a simple extension of this, we could imagine comparing the expected heterozygosity in the subpopulation (<span class="math">\(H_S\)</span>) to that expected in the total population <span class="math">\(H_T\)</span>, via <span class="math">\({F_{\mathrm{ST}}}\)</span>:</p>
<p><span class="math">\[{F_{\mathrm{ST}}}= 1-\frac{H_S}{H_T}=1-\frac{2p_Sq_S}{2p_Tq_T} \label{eqn:FST}.\]</span></p>
<p>If the total population contains the subpopulation then, as we will see below, due to the Wahlund effect (to be added) <span class="math">\(2p_Sq_S \leq
2p_Tq_T\)</span>, and so <span class="math">\({F_{\mathrm{IS}}}\leq {F_{\mathrm{IT}}}\)</span> and <span class="math">\({F_{\mathrm{ST}}}\geq 0\)</span>. We can relate the three <span class="math">\(F\)</span>-statistics to each other as</p>
<p><span class="math">\[(1-\hat{{F_{\mathrm{IT}}}}) =\frac{H_I}{H_S} \frac{H_S}{H_T}=(1-{F_{\mathrm{IS}}})(1-{F_{\mathrm{ST}}}).
\label{eqn:F_relationships}\]</span></p>
<p>Hence, the reduction in heterozygosity within individuals compared to that expected in the total population can be decomposed to the reduction in heterozygosity of individuals compared to the subpopulation, and the reduction in heterozygosity from the total population to that in the subpopulation.<br /></p>
<p>If we want a summary of population structure across multiple subpopulations, we can average <span class="math">\(H_I\)</span> and/or <span class="math">\(H_S\)</span> across populations, and use a <span class="math">\(p_T\)</span> calculated by averaging <span class="math">\(p_S\)</span> across subpopulations (or our samples from sub-populations). For example, the average <span class="math">\({F_{\mathrm{ST}}}\)</span> across <span class="math">\(K\)</span> subpopulations (sampled with equal effort) is</p>
<p><span class="math">\[{F_{\mathrm{ST}}}= 1 - \frac{\bar{H}_{S}}{H_T},\]</span></p>
<p>where <span class="math">\(\bar{H}_S = 1/K \sum_{i = 1}^{K} H_{S}^{(i)}\)</span>, and <span class="math">\(H_{S}^{(i)} = 2 p_{i} q_{i}\)</span> is the expected heterozygosity in subpopulation <span class="math">\(i\)</span>. Furthermore, if we have multiple sites, we can replace <span class="math">\(H_I\)</span>, <span class="math">\(H_S\)</span>, and <span class="math">\(H_T\)</span> with their averages across loci (as above).<br /></p>
<p>Let us now return to Wright’s definition of the <span class="math">\(F\)</span>-statistics as correlations between random gametes, drawn from the same level <span class="math">\(X\)</span>, relative to level <span class="math">\(Y\)</span>. Without loss of generality, we may think about <span class="math">\(X\)</span> as individuals and <span class="math">\(S\)</span> as the subpopulation. Rewriting <span class="math">\({F_{\mathrm{IS}}}\)</span> in terms of the observed homozygote frequencies (<span class="math">\(f_{11}\)</span>, <span class="math">\(f_{22}\)</span>) and expected homozygosities (<span class="math">\(p_{S}^2\)</span>, <span class="math">\(q_{S}^2\)</span>) we find</p>
<p><span class="math">\[{F_{\mathrm{IS}}}= \frac{2p_Sq_S - f_{12}}{2p_Sq_S} = \frac{f_{11}+f_{22} -
p_S^2 - q_S^2}{2p_Sq_S},
\label{eqn:Fascorr}\]</span></p>
<p>using the fact that <span class="math">\(p^2+2pq+q^2=1\)</span>, and <span class="math">\(f_{12} = 1 - f_{11} - f_{12}\)</span>. The form of eqn. ([eqn:Fascorr]) reveals that <span class="math">\({F_{\mathrm{IS}}}\)</span> is the covariance between pairs of alleles found in an individual, divided by the expected variance under binomial sampling. Thus, <span class="math">\(F\)</span>-statistics can be understood as the correlation between alleles drawn from a population (or an individual) above that expected by chance (i.e. drawing alleles sampled at random from some broader population).<br /></p>
<p>We can also interpret <span class="math">\(F\)</span>-statistics as proportions of variance explained by different levels of population structure. To see this, let us think about <span class="math">\({F_{\mathrm{ST}}}\)</span> averaged over <span class="math">\(K\)</span> subpopulations, whose frequencies are <span class="math">\(p_1,\dots,p_K\)</span>. The frequency in the total population is <span class="math">\(p_T=\bar{p} = 1/K \sum_{i=1}^K p_i\)</span>. Then, we can write</p>
<p><span class="math">\[{F_{\mathrm{ST}}}= \frac{2 \bar{p}\bar{q} - \frac{1}{K}\sum_{i=1}^K 2p_iq_i }{2
\bar{p}\bar{q}} = \frac{ \left(\frac{1}{K} \sum_{i=1}^K p_i^2 +
\frac{1}{K} \sum_{i=1}^K q_i^2 \right) -  \bar{p}^2-\bar{q}^2 }{2
\bar{p}\bar{q}} = \frac{\mathrm{Var}(p_i)}{\mathrm{Var}(\bar{p})},
\label{eqn:F_as_propvar}\]</span></p>
<p>which shows that <span class="math">\({F_{\mathrm{ST}}}\)</span> is the proportion of the variance explained by the subpopulation labels.</p>
<h1 id="genetic-drift-and-neutral-diversity">Genetic Drift and Neutral Diversity</h1>
<p>Various sources of randomness are inherent in evolution. One major source of stochasticity in population genetics is genetic drift. Genetic drift occurs because more or less copies of an allele by chance can be transmitted to the next generation. This can occur because by chance the individuals carrying a particular allele can leave more or less offspring in the next generation. In a sexual population genetic drift also occurs because mendelian transmission means that only one of the two alleles in an individual, chosen at random at a locus, is transmitted to the offspring.</p>
<p>Genetic drift can play a role in the dynamics of all alleles and populations, but it will play the biggest role for neutral alleles. A neutral polymorphism occurs when the segregating alleles at a polymorphic site have no discernable effect on the fitness (we’ll make clear what we mean by discernable later, for the moment think of this as “no effect” on fitness).</p>
<h2 id="LossofHet">Loss of heterozygosity due to drift.</h2>
<p>Genetic drift will, in the absence of new mutations, slowly purge our population of neutral genetic diversity as alleles slowly drift to high or low frequencies and are lost or fixed over time.<br /></p>
<p>Imagine a population of a constant size <span class="math">\(N\)</span> diploid individuals, and that we are examining a locus segregating for two alleles that are neutral with respect to each other. This population is randomly mating with respect to the alleles at this locus.<br /></p>
<p>In generation <span class="math">\(t\)</span> our current level of heterozygosity is <span class="math">\(H_t\)</span>, i.e. the probability that two randomly sampled alleles in generation <span class="math">\(t\)</span> are non-identical is <span class="math">\(H_t\)</span>. Assuming that the mutation rate is zero (or vanishing small), what is our level of heterozygosity in generation <span class="math">\(t+1\)</span>?<br /></p>
<p>In the next generation (<span class="math">\(t+1\)</span>) we are looking at the alleles in the offspring of generation <span class="math">\(t\)</span>. If we randomly sample two alleles in generation <span class="math">\(t+1\)</span> which had different parental alleles in generation <span class="math">\(t\)</span> then it is just like drawing two random alleles from generation <span class="math">\(t\)</span>. So the probability that these two alleles in generation <span class="math">\(t+1\)</span>, that have different parental alleles in generation <span class="math">\(t\)</span>, are non-identical is <span class="math">\(H_t\)</span>.<br /></p>
<p>Conversely, if our pair of alleles have the same parental allele in the proceeding generation (i.e. the alleles are identical by descent one generation back) then these two alleles must be identical (as we are not allowing for any mutation).<br /></p>
<p>In a diploid population of size <span class="math">\(N\)</span> individuals there are <span class="math">\(2N\)</span> alleles. The probability that our two alleles have the same parental allele in the proceeding generation is <span class="math">\(1/(2N)\)</span>, the probability that they have different parental alleles is is <span class="math">\(1-1/(2N)\)</span>. So by the above argument the expected heterozygosity in generation <span class="math">\(t+1\)</span> is</p>
<p><span class="math">\[H_{t+1} = \frac{1}{2N} \times 0 + \left(1-\frac{1}{2N} \right)H_t\]</span></p>
<p>By this argument if the heterzygosity in generation <span class="math">\(0\)</span> is <span class="math">\(H_0\)</span> our expected heterozygosity in generation <span class="math">\(t\)</span> is</p>
<p><span class="math">\[H_t = \left(1-\frac{1}{2N} \right)^tH_0\]</span></p>
<p>i.e. the expected heterozygosity with our population is decaying geometrically with each passing generation. If we assume that <span class="math">\(1/(2N)
\ll 1\)</span> then we can approximate this geometric decay by an exponential decay, such that</p>
<p><span class="math">\[H_t =H_0 \exp \left(-\frac{t}{2N} \right)\]</span></p>
<p>i.e. heterozygosity decays exponentially at a rate <span class="math">\(1/(2N)\)</span>.</p>
<h2 id="DriftMutationBalance">Levels of diversity maintained by a balance between mutation and drift</h2>
<p>Looking backwards in time from one generation to the next, we are going to say that two alleles which have the same parental allele (i.e. find their common ancestor) in the preceding generation have <span><em>c</em>oalesced</span>, and refer to this event as a <span><em>c</em>oalescent event</span>.</p>
<p>The probability that our pair of randomly sampled alleles have coalesced in the preceding generation is <span class="math">\(1/(2N)\)</span>, the probability that our pair of alleles fail to coalesce is <span class="math">\(1-1/(2N)\)</span>.</p>
<p>The probability that a mutation changes the identity of the transmitted allele is <span class="math">\(\mu\)</span> per generation. So the probability of no mutation occurring is <span class="math">\((1-\mu)\)</span>. We’ll assume that when a mutation occurs it creates some new allelic type which is not present in the population. This assumption (commonly called the infinitely-many-alleles model) makes the math slightly cleaner, and also is not too bad an assumption biologically.</p>
<p>This model lets us calculate when our two alleles last shared a common ancestor and whether these alleles are identical as a result of failing to mutate since this shared ancestor. For example we can work out the probability that our two randomly sampled alleles coalesced <span class="math">\(2\)</span> generations in the past (i.e. they fail to coalesce in generation <span class="math">\(1\)</span> and then coalescent in generation <span class="math">\(2\)</span>), and that they are identical as</p>
<p><span class="math">\[\left(1- \frac{1}{2N} \right) \frac{1}{2N} (1-\mu)^4\]</span></p>
<p>note the power of <span class="math">\(4\)</span> is because our two alleles have to have failed to mutate through <span class="math">\(2\)</span> meioses each.</p>
<p>More generally the probability that our alleles coalesce in generation <span class="math">\(t+1\)</span> and are identical due to no mutation to either allele in the subsequent generations is</p>
<p><span class="math">\[P(\textrm{coal. in t+1 \&amp; no mutations}) =  \frac{1}{2N} \left(1- \frac{1}{2N} \right)^t \left(1-\mu \right)^{2(t+1)}\]</span></p>
<p>assuming that <span class="math">\(\frac{1}{2N} \ll\)</span> and <span class="math">\(\mu \ll 1\)</span> then we can approximate this as</p>
<p><span class="math">\[P(\textrm{coal. in t+1 \&amp; no mutations}) \approx  \frac{1}{2N}
e^{-t/(2N)} e^{-2\mu (t+1)}\]</span></p>
<p>to make this slightly easier on ourselves lets further assume that <span class="math">\(t
\approx t+1\)</span> and so rewrite this as</p>
<p><span class="math">\[\approx \frac{1}{2N} e^{-t(2\mu+1/(2N))}\]</span></p>
<p>If we sample two alleles at random from the population we will not in general know when they share a common ancestor. In which case we will need to integrate out over when this coalescent event occured. Doing this we find the probability that our two alleles are identical due to no mutation on either ancestral lineage since the pair shared a common ancestor to be</p>
<p><span class="math">\[\frac{1}{2N} \int_0^{\infty} e^{-t(2\mu+1/(2N))} dt =
\frac{1/(2N)}{1/(2N)+2\mu} = \frac{1}{1+4N\mu}\]</span></p>
<p>The probability that our our pair of alleles are non-identical is simply one minus this, i.e.</p>
<p><span class="math">\[\frac{4N\mu}{1+4N\mu} \label{eqn:hetero}\]</span></p>
<p>This compound parameter <span class="math">\(4N\mu\)</span>, the population-scaled mutation rate, will come up a number of times so we’ll give it its own name</p>
<p><span class="math">\[\theta = 4N\mu\]</span></p>
<p>So all else being equal, species with larger population sizes should have proportionally higher levels of neutral polymorphism.</p>
<h2 id="the-effective-population-size.">The effective population size.</h2>
<p>In practice populations rarely conform to our assumptions of being constant in size with low variance in reproduction success. Real populations experience dramatic fluctuations in size, and there is often high variance in reproductive success. Thus rates of drift in natural populations are often a lot higher than the census population size would imply.<br /></p>
<p>To cope with this population geneticists often invoke the concept of an effective population size (<span class="math">\(N_e\)</span>). In many situations (but not all), departures from model assumptions can be captured by substituting <span class="math">\(N_e\)</span> for <span class="math">\(N\)</span>.</p>
<p>Specifically the effective population size (<span class="math">\(N_e\)</span>) is the population size that would result in the same rate of drift in an idealized constant population size, obeying our modeling assumptions, as that observed in our true population.<br /></p>
<p>If population sizes vary rapidly in size, we can (if certain conditions are met) replace our population size by the harmonic mean population size. Consider a diploid population of variable size, whose size is <span class="math">\(N_t\)</span> <span class="math">\(t\)</span> generations into the past. The probability our pairs of alleles have not coalesced by the generation <span class="math">\(t^{th}\)</span> is given by</p>
<p><span class="math">\[\prod_{i=1}^{t} \left(1-\frac{1}{2N_t} \right)\]</span></p>
<p>note that this is simply collapses to our original expression <span class="math">\(\left(1-\frac{1}{2N } \right)^t \)</span> if <span class="math">\(N_i\)</span> is constant. If <span class="math">\(1/(N_i)\)</span> is small, then we can approximate <span class="math">\(1-\frac{1}{2N_i}\)</span> by <span class="math">\(\exp(-\frac{1}{2N_i})\)</span>. Such that if <span class="math">\(N_i\)</span> is never too small</p>
<p><span class="math">\[\prod_{i=1}^{t} \left(1-\frac{1}{2N_i} \right)
\approx \prod_{i=1}^{t} \exp \left( -\frac{1}{2N_i} \right)   =
\exp \left(- \sum_{i=1}^{t} \frac{1}{2N_i} \right) .\]</span></p>
<p>In our constant population size case the probability of failing to coalesce is <span class="math">\(\exp(-t/(2N))\)</span>. So the variable population coalescent probabilities are still of the same form but the exponent has changed. Comparing the exponent in the two cases we see</p>
<p><span class="math">\[\frac{t}{2N} = \sum_{i=1}^{t} \frac{1}{2N_i}\]</span></p>
<p>so that if we want a constant effective population size (<span class="math">\(N_e\)</span>) that has the same coalescent probability as our variable population we need to set <span class="math">\(N=N_e\)</span> and rearrange this to see</p>
<p><span class="math">\[N_e =\frac{1}{\frac{1}{t} \sum_{i=1}^{t} \frac{1}{N_i} }.\]</span></p>
<p>this is the harmonic mean of the varying population size. Thus our effective population size, the size of an idealized constant population which matches the rate of genetic drift, is the harmonic mean true population size over time. The harmonic mean is very storngly affected by small values, such that if our population size is one million <span class="math">\(99\%\)</span> of the time but drops to a <span class="math">\(1000\)</span> every hundred or so generations, <span class="math">\(N_e\)</span> will be much closer to <span class="math">\(1000\)</span> than a million.<br /></p>
<p>Variance in reproductive success will also affect our effective population size. Even if our population has a large constant size of <span class="math">\(N\)</span> individuals, if only small proportion of them get to reproduce then the rate of drift will reflect this much small number of reproducing individuals. If only <span class="math">\(N_M\)</span> males get to contribute to the next generation and <span class="math">\(N_F\)</span> females get to contribute to the next generation. When our two alleles pick an ancestor, <span class="math">\(25\%\)</span> of the time our alleles were both in a female ancestor in which case they coalesce with probability <span class="math">\(1/(2N_F)\)</span>, and <span class="math">\(25\%\)</span> of the time they are both in a male ancestor in which case they coalesce with probability <span class="math">\(1/(2N_M)\)</span>. The remaining <span class="math">\(50\%\)</span> of the time our ancestral lineages are in two individuals are different sexes in a generation so cannot coalescence. Therefore, our probability of coalescence in the preceding generation is</p>
<p><span class="math">\[\frac{1}{4}\frac{1}{2N_M}+\frac{1}{4}\frac{1}{2N_F} =
\frac{1}{8}\frac{N_F+N_M}{N_FN_M}\]</span></p>
<p>i.e. the rate of coalescence is the harmonic mean of the two sexes population sizes, equating this to <span class="math">\(\frac{1}{2N_e}\)</span> we find</p>
<p><span class="math">\[N_e = \frac{4N_FN_M}{N_F+N_M}\]</span></p>
<p>Thus if reproductive success is very skewed in one sex (e.g. <span class="math">\(N_M \ll
N/2\)</span>) our effective population size will be much reduced as a result.<br /></p>
<h2 id="neutral-diversity-and-population-structure">Neutral diversity and population structure</h2>
<p>Upto now we have assumed that our alleles that we have modelled in the coalescent setting are drawn from a randomly mating population such that any pair of lineages is equally likely to coalesce with each other. However, when there is population structure this assumption is violated.<br /></p>
<p>We have previously written the measure of population structure <span class="math">\({F_{\mathrm{ST}}}\)</span> as</p>
<p><span class="math">\[{F_{\mathrm{ST}}}= \frac{H_T-H_S}{H_T}\]</span></p>
<p>where <span class="math">\(H_S\)</span> is the probability that two alleles sampled at random from a subpopulation differ, and <span class="math">\(H_T\)</span> is the probability that two alleles sampled at random from the total population differ.</p>
<h4 id="a-simple-population-split-model">A simple population split model</h4>
<p>Imagine a population of constant size of <span class="math">\(N_e\)</span> diploid individuals that <span class="math">\(\tau\)</span> generations in the past split into two daughter populations (sub-populations) each of size <span class="math">\(N_e\)</span> individuals, who do not subsequently exchange migrants. In the current day we sample an equal number of alleles from both subpopulations.</p>
<p>Consider a pair of alleles sampled within one of our sub-populations, they have experienced a population of size <span class="math">\(N_e\)</span> and so the probability that they differ is <span class="math">\(H_S = \theta/(1+\theta)\)</span> (where <span class="math">\(\theta=4N_e\mu\)</span>). The heterozygosity in our total population is a little more tricky to calculate. Assuming that we equally sample both sub-populations, when we draw two alleles from our total sample, <span class="math">\(50\%\)</span> of the time they are drawn from the same subpopulation and <span class="math">\(50\%\)</span> of the time they are drawn from different subpopulations. Therefore, our total heterozygosity is given by</p>
<p><span class="math">\[H_T = {\tfrac{1}{2}}H_S + {\tfrac{1}{2}}H_B\]</span></p>
<p>where <span class="math">\(H_B\)</span> is the probability that a pair of alleles drawn from our two different sub-populations differ from each other. Our pair of alleles can not find a common ancestor with each other for at least <span class="math">\(\tau\)</span> generations into he past as they are in distinct populations (not connected by migration). The probability that one or other of them mutates in this time is <span class="math">\(1-(1-\mu)^{2T}\)</span>. With probability <span class="math">\((1-\mu)^{2T} \)</span> neither of our alleles mutate in the <span class="math">\(T\)</span> generations back in time before they find themselves back in the combined ancestral population. Conditional on failing to mutating before the combined ancestral population, the probability that they do manage to mutate before coalescing in that population of size <span class="math">\(N_e\)</span> is <span class="math">\(\theta/(\theta+1)\)</span>. Putting these components together</p>
<p><span class="math">\[H_B = \left( 1-(1-\mu)^{2T} \right) + (1-\mu)^{2T}
  \frac{\theta}{\theta+1}\]</span></p>
<p>We can plug this into our expression for <span class="math">\(H_T\)</span>, and then that in turn into <span class="math">\({F_{\mathrm{ST}}}\)</span>.</p>
<p>To understand this better we can make a simple approximation based on our mutation rate being very low, such that <span class="math">\(N_e \mu \ll 1\)</span> ao <span class="math">\(H_S \approx
4N_e\mu\)</span>, and that <span class="math">\(\mu \ll 1\)</span> and <span class="math">\(\mu T \ll 1\)</span>. Assuming this, then</p>
<p><span class="math">\[H_B \approx 2 \mu T + 4N_e\mu.\]</span></p>
<p>So that</p>
<p><span class="math">\[{F_{\mathrm{ST}}}\approx \frac{ \mu T}{\mu T +  4N_e\mu }\]</span></p>
<p>note that <span class="math">\(\mu\)</span> cancels out of this. In this simple toy model <span class="math">\({F_{\mathrm{ST}}}\)</span> is increasing because the amount of between population diversity increases with the divergence time of the two populations (initially linearly with <span class="math">\(T\)</span>). It does so at a rate give by <span class="math">\(T/(4N_e)\)</span> so that differentiation will be higher between populations seperated by long divergence times or with small effective population sizes.</p>
<h4 id="a-simple-model-of-migration-between-an-island-and-the-mainland.">A simple model of migration between an island and the mainland.</h4>
<p>We can also use the coalescent to think about patterns of differentiation under a simple model of migration drift equilibrium. Lets consider a small island population that is relatively isolated from a large mainland population, and that both of these populations are constant in size. We’ll assume that the expected heterozygosity for a pair of alleles sampled on the mainland is <span class="math">\(H_M\)</span>.</p>
<p>Our island has a population size <span class="math">\(N_{I}\)</span> that is very small compared to our mainland population. Each generation some low fraction <span class="math">\(m\)</span> of our individuals on the island have migrant parents from the mainland the generation before. Our island may also send migrants back to the mainland, but these are a drop in the ocean compared to the large population size on the mainland and their effect can be ignored.</p>
<p>If we sample an allele on the island back and trace its ancestral lineage backward in time, each generation our ancestral allele have a low probability <span class="math">\(m\)</span> of being descended from the mainland in the proceeding generation (if we go far enough the allele eventually has to be descended from an allele on the mainland). The probability that a pair of alleles sampled on the island are descended from a shared recent common ancestral allele on the island, is the probability that our pair of alleles coalesce before either lineage migrates. For example, the probability that our pair of alleles coalesce <span class="math">\(t+1\)</span> generations back is</p>
<p><span class="math">\[\frac{1}{2N_I}(1-m)^{2(t+1)} \left(1-\frac{1}{2N_I} \right)^{t} \approx
\frac{1}{2N_I} \exp\left( -t\left (\frac{1}{2N_I} + 2m\right) \right),\]</span></p>
<p>with the approximation following from assuming that <span class="math">\(m \ll 1\)</span> &amp; <span class="math">\(1/(2N_I)
\ll 1\)</span> (note that this is very similar to our derivation of heterozygosity above). The probability that our alleles coalescence before either one of them migrates off the island, irrespective of the time, is</p>
<p><span class="math">\[\int_0^{\infty} \frac{1}{2N_I} \exp\left( -t\left (\frac{1}{2N_I} +
    2m\right) \right) dt = \frac{1/(2N_I) }{1/(2N_I) +
    2m}.\]</span></p>
<p>Lets assume that the mutation rate is very low such as it is very unlikely that the pair of alleles mutate before they coalesce on the island. Therefore, the only way that the alleles can be different from each other is if one or other of them migrates to the mainland, which happens with probability</p>
<p><span class="math">\[1 - \frac{1/(2N_I) }{1/(2N_I) + 2m}\]</span></p>
<p>Conditional on one or other of our alleles migrating to the mainland, both of our alleles represent independent draws from the mainland and so differ from each other with probability <span class="math">\(H_M\)</span>. Therefore, the level of heterozygosity on the island is given by</p>
<p><span class="math">\[H_I = (1 - \frac{1/(2N_I) }{1/(2N_I) + 2m})H_M\]</span></p>
<p>So the reduction of heterozygosity on the island compared to the mainland is</p>
<p><span class="math">\[F_{IM} = 1- \frac{H_I}{H_M} = \frac{ 1/(2N_I) }{1/(2N_I) + 2m} = \frac{ 1 }{1 + 4N_Im}.\]</span></p>
<p>The level of inbreeding on the island compared to the mainland will be high in the migration rate is low and the effective population size of the island is low, as allele frequencies on the island are drifting and diversity is not being replenished on the island by migration. The key parameter here is the number individuals on the island replaced by immigrants from the mainland each generation (<span class="math">\(N_I m\)</span>).</p>
<p>We have framed this as being about the reduction in genetic diversity on the island compared to the mainland. However, if we consider collecting a individuals on the island and mainland in proportion to population sizes the total level of heterozygosity would be <span class="math">\(H_T=H_M\)</span>, as samples from our mainland would greatly outnumber those from our island. Therefore, considering our island our sub-population we have derived another simple model of <span class="math">\(F_{ST}\)</span> .</p>
<h2 id="other-approaches-to-population-structure">Other approaches to population structure</h2>
<p>There is a broad spectrum of methods to describe patterns of population structure in populaion genetic datasets. We’ll briefly discuss two broad-classes of methods, assigment methods and principal components analysis,that appear often in the literature.</p>
<h3 id="assignment-methods">Assignment Methods</h3>
<p>Here we’ll describe a simple probabilistic assignment to find the probability that an individual of unknown population comes from one of <span class="math">\(K\)</span> predefined populations. We’ll then briefly explain how to extend this to cluster individuals into <span class="math">\(K\)</span> initially unknown populations. This method is a simplified version of what Bayesian population genetics clustering algorithms such as STRUCTURE and ADMIXTURE do (Pritchard et al. Genetics 2000).</p>
<h4 id="a-simple-assignment-method">A simple assignment method</h4>
<p>We have genotype data from unlinked S bi-allelic loci for <span class="math">\(K\)</span> populations. The allele frequency of allele <span class="math">\(A_1\)</span> at locus <span class="math">\(l\)</span> in population <span class="math">\(k\)</span> is denoted by <span class="math">\(p_{k,l}\)</span>, so that the allele frequencies in population 1 are <span class="math">\(p_{1,1},\cdots p_{1,L}\)</span> and population 2 are <span class="math">\(p_{2,1},\cdots p_{2,L}\)</span> and so on.</p>
<p>You type a new individual from an unknown population at these <span class="math">\(L\)</span> loci. This individual’s genotype at locus <span class="math">\(l\)</span> is <span class="math">\(g_l\)</span>, where <span class="math">\(g_l\)</span> denotes the number of copies of allele <span class="math">\(A_1\)</span> this individual carries at this locus (<span class="math">\(g_l=0,1,2\)</span>).</p>
<p>The probability of this individual’s genotype at locus <span class="math">\(l\)</span> conditional on coming from population <span class="math">\(k\)</span> (i.e. their alleles being a random HW draw from population <span class="math">\(k\)</span>) is</p>
<p><span class="math">\[P(g_l | \textrm{pop k}) = I(g_l=0) (1-p_{k,l})^2 +  I(g_l=1) 2 p_{k,l} (1-p_{k,l}) + I(g_l=2) p_{k,l}^2\]</span></p>
<p>where <span class="math">\(I(g_l=0)\)</span> is an indicator function which is <span class="math">\(1\)</span> if <span class="math">\(g_l=0\)</span> and zero otherwise, and likewise for the other indicator functions. This follows simply from HWE.</p>
<p>Assuming that the loci are independent, the probability of individual’s genotypes conditional on them coming from population <span class="math">\(k\)</span> is</p>
<p><span class="math">\[P(\textrm{ind.} | \textrm{pop k})  = \prod_{l=1}^S P(g_l | \textrm{pop k}) \label{eqn_assignment}\]</span></p>
<p>We wish to know the probability that this new individual comes from population <span class="math">\(k\)</span>, i.e. <span class="math">\(P(\textrm{pop k} | \textrm{new ind.})\)</span>. We can obtain this through Bayes rule</p>
<p><span class="math">\[P(\textrm{pop k} | \textrm{ind.})  = \frac{P(\textrm{ind.} | \textrm{pop k}) P(\textrm{pop k})}{P(\textrm{ind.})}\]</span></p>
<p>where</p>
<p><span class="math">\[P(\textrm{ind.}) = \sum_{k=1}^K  P(\textrm{ind.} | \textrm{pop k}) P(\textrm{pop k})\]</span></p>
<p>is the normalizing constant. We interpret <span class="math">\(P(\textrm{pop k})\)</span> as the prior probability of the individual coming from population <span class="math">\(k\)</span>, unless we have some other prior knowledge we will assume that the new individual has a equal probability of coming from each population <span class="math">\(P(\textrm{pop k})=1/K\)</span>.</p>
<p>We intepret</p>
<p><span class="math">\[P(\textrm{pop k} | \textrm{ind.})\]</span></p>
<p>as the posterior probability that our new individual comes from each of our <span class="math">\(1,\cdots, K\)</span> populations.</p>
<p>More sophisticated versions of this are now used to allow for hybrids, e.g, we can have a proportion <span class="math">\(q_k\)</span> of our individual’s genome come from population <span class="math">\(k\)</span> and estimate the set of <span class="math">\(q_k\)</span>’s.</p>
<p><span><strong>Question.</strong></span> We have two populations where the frequency of allele <span class="math">\(A_1\)</span> at two SNPs (<span class="math">\(A_1/A_2\)</span>) is given by</p>
<table>
<thead>
<tr class="header">
<th style="text-align: center;">Population</th>
<th style="text-align: center;">locus 1</th>
<th style="text-align: center;">locus 2</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">A</td>
<td style="text-align: center;"><span class="math">\(0.1\)</span></td>
<td style="text-align: center;"><span class="math">\(0.85\)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;">B</td>
<td style="text-align: center;"><span class="math">\(0.95\)</span></td>
<td style="text-align: center;"><span class="math">\(0.2\)</span></td>
</tr>
</tbody>
</table>
<p>We sample an individual whose genotype is <span class="math">\(A_1A_1\)</span> at the first locus and <span class="math">\(A_2A_2\)</span> at the second. What is the probability that our indvidual comes from population 1 vs population 2? Lets assume that with probability <span class="math">\(q_1\)</span> our individual draws an allele from population <span class="math">\(1\)</span> and that with probability <span class="math">\(q_2=1-q_1\)</span> they draw an allele from population <span class="math">\(2\)</span>. What is the probability of our individual’s genotype given <span class="math">\(q_1\)</span>? Plot this probability as a function of <span class="math">\(q_1\)</span>. How does your plot change if our individual is heterozygote at both loci?</p>
<h4 id="clustering-based-on-assignment-methods">Clustering based on assignment methods</h4>
<p>While it is great to be able to assign our individuals to particular population, these ideas can be pushed to learn about how best to describe our genotype data in terms of discrete populations without assigning any of our individuals to populations <span><em>a priori</em></span>. We wish to cluster our individuals into <span class="math">\(K\)</span> unknown populations. We begin by assigning our individuals at random to these <span class="math">\(K\)</span> populations.</p>
<ul>
<li><p>Given these assignments we estimate the allele frequencies at all of our loci in each population.</p></li>
<li><p>Given these allele frequencies we chose to reassign each individual to a population <span class="math">\(k\)</span> with a probability given by eqn. (<span class="math">\(\ref{eqn_assignment}\)</span>).</p></li>
</ul>
<p>We iterate steps 1 and 2 for many iterations. If the data is sufficiently informative the assignments and allele frequencies will quickly converge.</p>
<p>To do this in a full bayesian scheme we need to place priors on the allele frequencies (e.g. a beta distribution).Technically we are using this is the joint posterior of our allele frequencies and assignments.</p>
<h3 id="principal-components-analysis">Principal components analysis</h3>
<p>The use of principal component analysis in population genetics was pioneered by Cavalli-Sforza. With large genotyping datasets PCA has made a come back. See Patterson et al 2006, PLoS Genetics and McVean, G. 2010 PLoS Genetics and for recent discussion.</p>
<p>Consider a dataset consisting of N individuals at S bi-allelic SNPs. The <span class="math">\(i^{th}\)</span> individual’s genotype data at locus <span class="math">\(\ell\)</span> takes value <span class="math">\(g_{i,\ell}\)</span>=0,1, or 2 (corresponding to the number of copies of allele <span class="math">\(A_1\)</span> an individual carrys at this SNP). We can think of this as a N x S matrix (where usually <span class="math">\(N \ll S\)</span>).</p>
<p>Denoting the sample mean allele freq at SNP <span class="math">\(\ell\)</span> by <span class="math">\(p_{\ell}\)</span> we usually standardize the genotype in the following way</p>
<p><span class="math">\[\frac{g_{i,\ell} - 2 p_{\ell}}{\sqrt{p_{\ell}(1-p_{\ell})}}\]</span></p>
<p>i.e. at each SNP we center the genotypes by minusing of the mean genotype (<span class="math">\(2\epsilon_{\ell}\)</span>) and divide through by the expected variance assuming that alleles are sampled binomially from the mean frequency (<span class="math">\(\sqrt{p_{\ell} (1-p_{\ell})}\)</span>). Doing this to all of our genotypes we form a data matrix (of dimension N x S). We can then perform principal components analysis of this data matrix to cover the major axes of genotype variance in our sample.</p>
<p>It is worth taking a moment to delve further into what we are doing here. There’s a number of equivalent ways to thinking about what PCA is doing, one of these is to think that when we do PCA we are building the individual by individual covariance matrix and performing eigen-value decomposition of this matrix (with the eigen-vectors giving the PC). This individual by individual covariance matrix has entries the <span class="math">\((i,~j)^{th}\)</span> entry given by</p>
<p><span class="math">\[\sum_{\ell=1}^S \frac{(g_{i,\ell} - 2p_{\ell})(g_{j,\ell} - 2p_{\ell})}{p_{\ell}(1-p_{\ell})}\]</span></p>
<p>note that this is the covariance, is very similar to those we encountered in discussing <span class="math">\(F\)</span>-statistics as correlations (equation ), expect now we are asking about the allelic covariance between two individuals above that expected if they were both drawn from the total sample at random (rather than the covariance of alleles within a single individual). So by performing PCA on the data we are learning about the major (orthogonal) axes of the kinship matrix.</p>
<h1 id="correlations-between-loci-linkage-disequilibrium-and-recombination">Correlations between loci, linkage disequilibrium, and recombination</h1>
<p>Up to now we have been interested in correlations between alleles at the same locus, e.g. correlations within individuals (inbreeding) or between individuals (relatedness). We have seen how relatedness between parents affects the extent to which their offspring is inbred. We now turn to correlations between alleles at different loci. To understand correlations between loci we need to understand recombination.<br /></p>
<h4 id="recombination">Recombination</h4>
<p>Lets consider an individual heterozygous for a <span class="math">\(AB\)</span> and <span class="math">\(ab\)</span> haplotype. If no recombination occurs between our two loci in this individual, then these two haplotypes will be transmitted intact to the next generation. While if a recombination (or more generally an odd number of recombinations) occurs between our two loci on the haplotype transmitted to the child then <span class="math">\(\tfrac{1}{2}\)</span> the time the child receives a <span class="math">\(Ab\)</span> haplotype and <span class="math">\(\tfrac{1}{2}\)</span> the time the child receives a <span class="math">\(aB\)</span> haplotype. So recombination is breaking up the association between loci. We’ll define the recombination fraction (<span class="math">\(r\)</span>) to be the probability of an odd number of recombinations between our loci. In practice we’ll often be interested in relatively short regions where recombination is relatively rare, and so we might think that <span class="math">\(r=r_{BP}L \ll 1\)</span>, where <span class="math">\(r_{BP}\)</span> is the average recombination rate per base pair (typically <span class="math">\(\sim 10^{-8}\)</span>) and L is the number of base pairs separating our two loci.<br /></p>
<h4 id="linkage-disequilibrium">Linkage disequilibrium</h4>
<p>The (horrible) phrase linkage disequilibrium (LD) refers to the statistical non-independence (i.e. a correlation) of alleles at different loci. Our two loci, which segregate alleles <span class="math">\(A/a\)</span> and <span class="math">\(B/b\)</span>, have allele frequencies of <span class="math">\(p_A\)</span> and <span class="math">\(p_B\)</span> respectively. The frequency of the two locus haplotype is <span class="math">\(p_{AB}\)</span>, and likewise for our other three combinations. If our loci were statistically independent then <span class="math">\(p_{AB} = p_Ap_B\)</span>, otherwise <span class="math">\(p_{AB} \neq p_Ap_B\)</span> We can define a covariance between the <span class="math">\(A\)</span> and <span class="math">\(B\)</span> alleles at our two loci as</p>
<p><span class="math">\[D_{AB} = p_{AB} - p_Ap_B\]</span></p>
<p>and likewise for our other combinations at our two loci (<span class="math">\(D_{Ab},~D_{aB},~D_{ab}\)</span>). These <span class="math">\(D\)</span> statistics are all closely related to each other as <span class="math">\(D_{AB} = - D_{Ab}\)</span> and so on. Thus we only need to specify one <span class="math">\(D_{AB}\)</span> to know them all, so we’ll drop the subscript and just refer to <span class="math">\(D\)</span>. Also a handy result is that we can rewrite our haplotype frequency <span class="math">\(p_{AB}\)</span> as</p>
<p><span class="math">\[p_{AB} = p_Ap_B+D. \label{eqn:ABviaD}\]</span></p>
<p>If <span class="math">\(D=0\)</span> we’ll say the two loci are in linkage equilibrium, while if <span class="math">\(D&gt;0\)</span> or <span class="math">\(D&lt;0\)</span> we’ll say that the loci are in linkage disequilibrium (we’ll perhaps want to test whether <span class="math">\(D\)</span> is statistically different from <span class="math">\(0\)</span> before making this choice). You should be careful to keep the concepts of linkage and linkage disequilibrium separate in your mind. Genetic linkage refers to the linkage of multiple loci due to the fact that they are transmitted through meiosis together (most often because the loci are on the same chromosome). Linkage disequilibrium merely refers to the correlation between the alleles at different loci, this may in part be due to the genetic linkage of these loci but does not necessarily imply this (e.g. genetically unlinked loci can be in LD due to population structure).<br /></p>
<p>Another common statistic for summarizing LD is <span class="math">\(r^2\)</span> which we write as</p>
<p><span class="math">\[r^2 = \frac{D^2}{p_A(1-p_A) p_B(1-p_B) }\]</span></p>
<p>as <span class="math">\(D\)</span> is a covariance, and <span class="math">\(p_A(1-p_A) \)</span> is the variance of an allele drawn at random from locus <span class="math">\(A\)</span>, <span class="math">\(r^2\)</span> is the squared correlation coefficient.<br /></p>
<p><span><strong>Question.</strong></span> You genotype 2 bi-allelic loci (A &amp; B) segregating in two mouse subspecies (1 &amp; 2) which mate randomly among themselves, but have not historically interbreed since they speciated. On the basis of previous work you estimate that the two loci are separated by a recombination fraction of 0.1. The frequencies of haplotypes in each population are:</p>
<table>
<thead>
<tr class="header">
<th style="text-align: center;">Pop</th>
<th style="text-align: center;"><span class="math">\(p_{AB}\)</span></th>
<th style="text-align: center;"><span class="math">\(p_{Ab}\)</span></th>
<th style="text-align: center;"><span class="math">\(p_{aB}\)</span></th>
<th style="text-align: center;"><span class="math">\(p_{ab}\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">1</td>
<td style="text-align: center;">.02</td>
<td style="text-align: center;">.18</td>
<td style="text-align: center;">.08</td>
<td style="text-align: center;">.72</td>
</tr>
<tr class="even">
<td style="text-align: center;">2</td>
<td style="text-align: center;">.72</td>
<td style="text-align: center;">.18</td>
<td style="text-align: center;">.08</td>
<td style="text-align: center;">.02</td>
</tr>
</tbody>
</table>
<p><span><strong>A)</strong></span> How much LD is there within populations, i.e. estimate D?<br /></p>
<p><span><strong>B)</strong></span> If we mixed the two populations together in equal proportions what value would D take before any mating has had the chance to occur?<br /></p>
<h4 id="the-decay-of-ld-due-to-recombination">The decay of LD due to recombination</h4>
<p>We will now examine what happens to LD over the generations if we only allow recombination to occur in a very large population (i.e. no genetic drift, i.e. the frequencies of our loci follow their expectations). To do so consider the frequency of our <span class="math">\(AB\)</span> haplotype in the next generation <span class="math">\(p_{AB}^{\prime}\)</span>. We lose a fraction <span class="math">\(r\)</span> of our <span class="math">\(AB\)</span> haplotypes to recombination ripping our alleles apart but gain a fraction <span class="math">\(rp_A p_B\)</span> per generation from other haplotypes recombining together to form <span class="math">\(AB\)</span> haplotypes. Thus in the next generation</p>
<p><span class="math">\[p_{AB}^{\prime} = (1-r)p_{AB} + rp_Ap_B\]</span></p>
<p>this last term here is <span class="math">\(r(p_{AB}+p_{Ab})(p_{AB}+p_{aB})\)</span>, which multiplying this out is the probability of recombination in the different diploid genotypes that could generate a <span class="math">\(p_{AB}\)</span> haplotype.<br /></p>
<p>We can then write the change in the frequency of the <span class="math">\(p_{AB}\)</span> haplotype as</p>
<p><span class="math">\[\Delta p_{AB} = p_{AB}^{\prime} -p_{AB} = -r p_{AB} + rp_Ap_B = - r D\]</span></p>
<p>so recombination will cause a decrease in the frequency of <span class="math">\(p_{AB}\)</span> if there is an excess of <span class="math">\(AB\)</span> haplotypes within the population (<span class="math">\(D&gt;0\)</span>), and an increase if there is a deficit of <span class="math">\(AB\)</span> haplotypes within the population (<span class="math">\(D&lt;0\)</span>). Our LD in the next generation is <span class="math">\(D^{\prime} =
p_{AB}^{\prime}\)</span>, so we can rewrite the above eqn. in terms of the <span class="math">\(D^{\prime} \)</span></p>
<p><span class="math">\[D^{\prime}= (1-r) D\]</span></p>
<p>so if the level of LD in generation <span class="math">\(0\)</span> is <span class="math">\(D_0\)</span> the level <span class="math">\(t\)</span> generations later (<span class="math">\(D_t\)</span>) is</p>
<p><span class="math">\[D_t=  (1-r)^t D_0\]</span></p>
<p>so recombination is acting to decrease LD, and it does so geometrically at a rate given by <span class="math">\((1-r)\)</span>. If <span class="math">\(r \ll 1\)</span> then we can approximate this by an exponential and say that</p>
<p><span class="math">\[D_t \approx  D_0 e^{-rt}\]</span></p>
<p><br /></p>
<p><span><strong>Q C)</strong></span> You find a hybrid population between the two mouse subspecies described in the question above, which appears to be comprised of equal proportions of ancestry from the two subspecies. You estimate LD between the two markers to be 0.0723. Assuming that this hybrid population is large and was formed by a single mixture event, can you estimate how long ago this population formed?<br /></p>
<h1 id="the-phenotypic-resemblance-between-relatives">The phenotypic resemblance between relatives</h1>
<p>We can use our understanding of the sharing of alleles between relatives to understand the phenotypic resemblance between relatives in quantitative phenotypes. We can then use this to understand the evolutionary change in quantitative phenotypes in response to selection.<br /></p>
<p>Let’s imagine that the genetic component of the variation in our trait is controlled by <span class="math">\(L\)</span> autosomal loci that act in an additive manner. The frequency of allele <span class="math">\(1\)</span> at locus <span class="math">\(l\)</span> is <span class="math">\(p_l\)</span>, with each copy of allele <span class="math">\(1\)</span> at this locus increasing your trait value by <span class="math">\(a_l\)</span> above the population mean. The phenotype of an individual, let’s call her <span class="math">\(i\)</span>, is <span class="math">\(X_i\)</span>. Her genotype at SNP <span class="math">\(l\)</span>, is <span class="math">\(G_{i,l}\)</span>. Here <span class="math">\(G_{i,l}=0,~1,\)</span> or <span class="math">\(2\)</span> represents the number of copies of allele <span class="math">\(1\)</span> she has at this SNP. Her expected phenotype, given her genotype, is then</p>
<p><span class="math">\[X_{A,i} = \mu + {\mathbb{E}}(X_i | G_{i,1},\cdots,G_{i,L}) = \mu+ \sum_{l=1}^L G_{i,l} a_{l}\]</span></p>
<p>where <span class="math">\(\mu\)</span> is the mean phenotype in our population. Now in reality the genetic phenotype is a function of the expression of those alleles in a particular environment. Therefore, we can think of this expected phenotype as being an average across a set of environments that occur in the population.<br /></p>
<p>When we measure our individual’s phenotype we see</p>
<p><span class="math">\[X_i =   \mu+X_{A,i} + X_{E,i} \label{pheno_geno_environ}\]</span></p>
<p>where <span class="math">\(X_E\)</span> is the deviation from the mean phenotype due to the environment. This <span class="math">\(X_E\)</span> included the systematic effects of the environment our individual finds herself in and all of the noise during development, growth, and the various random insults that life throws at our individual. If a reasonable number of loci contribute to variation in our trait then we can approximate the distribution of <span class="math">\(X_{A,i}\)</span> by a normal distribution due to the central limit theorem (see R exercise). Thus if we can approximate the distribution of the effect of environmental variation on our trait (<span class="math">\(X_{E,i}\)</span>) also by a normal distribution, which is reasonable as there are many small environmental effects, then the distribution of phenotypes within the population (<span class="math">\(X_i\)</span>) will be normally distributed (see Figure [fig:QT1]).<br /></p>
<p><embed src="Figs/QT1.eps" /></p>
<p>[fig:QT1]</p>
<p>Note that as this is an additive model we can decompose eqn. [pheno<sub>g</sub>eno<sub>e</sub>nviron] into the effects of the two alleles at each locus, in particular we can rewrite it as</p>
<p><span class="math">\[X_i = \mu + X_{iM}+X_{iP} +X_{iE}\]</span></p>
<p>where <span class="math">\(X_{iM}\)</span> and <span class="math">\(X_{iP}\)</span> are the contribution to the phenotype of the allele that our individual received from her mother (maternal alleles) and father (paternal alleles) respectively. This will come in handy in just a moment when we start thinking about the phenotype covariance of relatives.<br /></p>
<p>Now obviously this model seems silly at first sight as alleles don’t only act in an additive manner, as they interact with alleles at the same loci (dominance) and at different loci (epistasis). Later we’ll relax this assumption, however, we’ll find that if we are interested in evolutionary change over short time-scales it is actually only the “additive component” of genetic variation that will (usually) concern us. We will define this more formally later on, but for the moment we can offer the intuition that parents only get to pass on a single allele at each locus on to the next generation. As such, it is the effect of these transmitted alleles, averaged over possible matings, that is an individual’s average contribution to the next generation (i.e. the additive effect of the alleles that their genotype consists of).</p>
<h3 id="additive-genetic-variance-and-heritability">Additive genetic variance and heritability</h3>
<p>As we are talking about an additive genetic model we’ll talk about the additive genetic variance (<span class="math">\(V_A\)</span>), the variance due to the additive effects of segregating genetic variation. This is a subset of the total genetic variance if we allow for non-additive effects.<br /></p>
<p>The variance of our phenotype across individuals (<span class="math">\(V\)</span>) can write this as</p>
<p><span class="math">\[V = Var(X_A) + Var(X_E) = V_A+V_E\]</span></p>
<p>in writing this we are assuming that there is no covariance between <span class="math">\(X_{G,i}\)</span> and <span class="math">\(X_{E,i}\)</span> i.e. there is no covariance between genotype and environment.<br /></p>
<p>Our additive genetic variance can be written as</p>
<p><span class="math">\[V_A = \sum_{l=1}^L Var(G_{i,l} a_{l})\]</span></p>
<p>where <span class="math">\(Var(G_{i,l} a_{l})\)</span> is the contribution to the additive variance among individuals of the <span class="math">\(l\)</span> locus. Assuming random mating we can write our additive genetic variance as</p>
<p><span class="math">\[V_A = \sum_{l=1}^L a_{l}^2 2 p_l(1-p_l)\]</span></p>
<p>where the <span class="math">\( 2 p_l(1-p_l)\)</span> term follows the binomial sampling of two alleles per individual at each locus.<br /></p>
<h4 id="the-narrow-sense-heritability">The narrow sense heritability</h4>
<p>We would like a way to think about what proportion of the variation in our phenotype across individuals is due to genetic differences as opposed to environmental differences. Such a quantity will be key in helping us think about the evolution of phenotypes. For example, if variation in our phenotype had no genetic basis then no matter how much selection changes the mean phenotype within a generation the trait will not change over generations.<br /></p>
<p>We’ll call the proportion of the variance that is genetic the heritability, and denote it by <span class="math">\(h^2\)</span>. We can then write this as</p>
<p><span class="math">\[h^2 = \frac{Var(X_A)}{V} = \frac{V_A}{V}\]</span></p>
<p>remember that we thinking about a trait where all of the alleles act in a perfectly additive manner. In this case our heritability <span class="math">\(h^2\)</span> is referred to as the narrow sense heritability, the proportion of the variance explained by the additive effect of our loci. When we allow dominance and epistasis into our model we’ll also have to define the broad sense heritability (the total proportion of the phenotypic variance attributable to genetic variation).<br /></p>
<p>The narrow sense heritability of a trait is a useful quantity, indeed we’ll see shortly that it is exactly what we need to understand the evolutionary response to selection on a quantitative phenotype. We can calculate the narrow sense heritability by using the resemblance between relatives. For example, if our phenotype was totally environmental we should not expect relatives to resemble each other any more than random individuals drawn from the population. Now the obvious caveat here is that relatives also share an environment, so may resemble each other due to shared environmental effects.<br /></p>
<h3 id="the-covariance-between-relatives">The covariance between relatives</h3>
<p>So we’ll go ahead and calculate the covariance in phenotype between two individuals (<span class="math">\(1\)</span> and <span class="math">\(2\)</span>) who have a phenotype <span class="math">\(X_1\)</span> and <span class="math">\(X_2\)</span> respectively.</p>
<p><span class="math">\[Cov(X_1,X_2) =
Cov\left((X_{1M}+X_{1P}+X_{1E}),((X_{2M}+X_{2P}+X_{2E}) \right)\]</span></p>
<p>We can expand this out in terms of the covariance between the various components in these sums.<br /></p>
<p>To make our task easier we (and most analyses) will assume two things</p>
<ol>
<li><p>that we can ignore the covariance of the environments between individuals (i.e. <span class="math">\(Cov(X_{1E},X_{2E})=0\)</span>)</p></li>
<li><p>that we can ignore the covariance between the environment variation experience by an individual and the genetic variation in another individual (i.e. <span class="math">\(Cov(X_{1E},(X_{2M}+X_{2P}))=0\)</span>).</p></li>
</ol>
<p>The failure of these assumptions to hold can severely undermine our estimates of heritability, but we’ll return to that later. Moving forward with these assumptions, we can write our phenotypic covariance between our pair of individuals as</p>
<p><span class="math">\[Cov(X_1,X_2) =
Cov((X_{1M},X_{2M})+Cov(X_{1M},X_{2P})+Cov(X_{1P},X_{2M})
+Cov(X_{1P},X_{2P}) \label{cov_rels_1}\]</span></p>
<p>This is saying that under our simple additive model we can see the covariance in phenotypes between individuals as the covariance between the allelic effects in our individuals. We can use our results about the sharing of alleles between relatives to obtain these terms. But before we write down the general case lets quickly work through some examples.<br /></p>
<h4 id="the-covariance-between-identical-twins">The covariance between Identical Twins</h4>
<p>Lets first consider the case of a pair of identical twins from two unrelated parents. Our pair of twins share their maternal and paternal allele identical by descent (<span class="math">\(X_{1M}=X_{2M}\)</span> and <span class="math">\(X_{1P}=X_{2P}\)</span>). As their maternal and paternal alleles are not correlated draws from the population, i.e. have no probability of being <span class="math">\(IBD\)</span> as we’ve said the parents are unrelated, the covariance between their effects on the phenotype is zero (i.e. <span class="math">\(Cov(X_{1P},X_{2M})=Cov(X_{1M},X_{2P})=0\)</span>). In that case eqn. [cov<sub>r</sub>els<sub>1</sub>] is</p>
<p><span class="math">\[Cov(X_1,X_2) = Cov((X_{1M},X_{2M})+Cov(X_{1P},X_{2P}) = 2Var(X_{1M})
= V_A\]</span></p>
<p>Now in general identical twins are not going to be super helpful for us in estimating <span class="math">\(h^2\)</span> as under models with non-additive effects identical twins have higher covariance than we’d expect as they resemble each other also because of the dominance effects as they don’t just share alleles they share their entire genotype.<br /></p>
<h4 id="the-covariance-in-phenotype-between-mother-and-child">The covariance in phenotype between mother and child</h4>
<p>. If the mother and father are unrelated individuals (i.e. are two random draws from the population) then the mother and a child share one allele IBD at each locus (i.e. <span class="math">\(r_1=1\)</span> and <span class="math">\(r_0=r_2=0\)</span>). Half the time our mother transmits her paternal allele to the child, in which case <span class="math">\(X_{P1}=X_{M2}\)</span> and so <span class="math">\(Cov(X_{P1},X_{M2})=Var(X_{P1})\)</span> and all the other covariances in eqn. [cov<sub>r</sub>els<sub>1</sub>] zero, and half the time she transmits her maternal allele to the child <span class="math">\(Cov(X_{M1},X_{M2})=Var(X_{M1})\)</span> and all the other terms zero. By this argument <span class="math">\(Cov(X_1,X_2) = {\tfrac{1}{2}}Var(X_{M1}) + {\tfrac{1}{2}}Var(X_{P1}) = {\tfrac{1}{2}}V_A\)</span>.<br /></p>
<h4 id="the-covariance-between-general-pairs-of-relatives-under-an-additive-model">The covariance between general pairs of relatives under an additive model</h4>
<p>The two examples make clear that to understand the covariance between phenotypes of relatives we simply need to think about the alleles they share IBD. Consider a pair of relatives (<span class="math">\(1\)</span> and <span class="math">\(1\)</span>) with a probability <span class="math">\(r_0\)</span>, <span class="math">\(r_1\)</span>, and <span class="math">\(r_2\)</span> of sharing zero, one, or two alleles IBD respectively. When they share zero alleles <span class="math">\(Cov((X_{1M}+X_{1P}),(X_{2M}+X_{2P}))=0\)</span>, when they share one allele <span class="math">\(Cov((X_{1M}+X_{1P}),(X_{2M}+X_{2P}))=
Var(X_{1M})=\frac{1}{2}V_A\)</span>, and when they share two alleles <span class="math">\(Cov((X_{1M}+X_{1P}),(X_{2M}+X_{2P}))=
V_A\)</span>. Therefore, the general covariance between two relatives is</p>
<p><span class="math">\[Cov(X_1,X_2) = r_0 \times 0 + r_1 \frac{1}{2}V_A + r_2  V_A =
2 F_{1,2} V_A\]</span></p>
<p><br />So under a simple additive model of the genetic basis of a phenotype to measure the narrow sense heritability we need to measure the covariance between a set of pairs of relatives (assuming that we can remove the effect of shared environmental noise). From the covariance between relatives we can calculate <span class="math">\(V_A\)</span>, we can then divide this by the total phenotypic variance to get <span class="math">\(h^2\)</span>.<br /></p>
<p>Another way that we can estimate the narrow sense heritability is through the regression of child’s phenotype on the parental mid-point phenotype. The parental mid-point phenotype is simple the average of the mum and dad’s phenotype. Denoting the child’s phenotype by <span class="math">\(X_{kid}\)</span> and mid-point phenotype by <span class="math">\(X_{mid}\)</span> so that if we take the regression <span class="math">\(X_{kid} \sim X_{mid}\)</span> this regression has slope <span class="math">\(\beta = Cov(X_{kid},X_{mid})/Var(X_{mid})\)</span>. The covariance of <span class="math">\(Cov(X_{kid},X_{mid})={\tfrac{1}{2}}V_A\)</span>, and <span class="math">\(Var(X_{mid}) = {\tfrac{1}{2}}V\)</span> as by taking the average of the parents we have halved the variance, such that the slope of the regression is</p>
<p><span class="math">\[\beta= \frac{Cov(X_{kid},X_{mid})}{Var(X_{mid})} = \frac{V_A}{V} = h^2\]</span></p>
<p>i.e. the regression of the child’s phenotype on the parental midpoint phenotype is an estimate of the narrow sense heritability. This is a common way to estimate heritability, although it doesn’t bypass the need to control for environmental correlations between relatives.<br /></p>
<p>Our regression allows us to attempt to predict the phenotype of the child given the child; how well we can do this depends on the slope. If the slope is close to zero then the parental phenotypes hold no information about the phenotype of the child, while if the slope is close to one then the parental mid-point is a good guess at the child’s phenotype.<br /></p>
<p><embed src="Figs/QT2.eps" /></p>
<p>More formally the expected phenotype of the child given the parental phenotypes is</p>
<p><span class="math">\[{\mathbb{E}}(X_{kid} | X_{mum},X_{dad}) = \mu +
\beta(X_{mid} - \mu) =\mu + h^2(X_{mid} - \mu)  \label{predict_kid}\]</span></p>
<p>this follows from the definition of linear regression. So to find the child’s predicted phenotype we simply take the mean phenotype and add on the difference between our parental mid-point multiplied by our narrow sense heritability.<br /></p>
<h4 id="estimating-additive-genetic-variance-across-a-variety-of-different-relationships.">Estimating additive genetic variance across a variety of different relationships.</h4>
<p>In many natural populations we may have access to individuals of a range of different relationships to each other (through monitoring of the paternity of individuals), but relatively few individuals of a given relationship (e.g. sibs). We can try and use this information as fully as possible in a mixed model framework. Considering equation [pheno<sub>g</sub>eno<sub>e</sub>nviron] we can write an individual’s phenotype <span class="math">\(X_i\)</span> as</p>
<p><span class="math">\[X_i =  \mu  + X_{A,i} + e_i\]</span></p>
<p>where <span class="math">\(e_i \sim N(0,V_E)\)</span> and <span class="math">\(X_{A,i}\)</span> is normally distributed across individuals with covariance matrix <span class="math">\(V_A A\)</span> where the the entries for an pair of individuals i and i are <span class="math">\(A_{ij}= 2 F_{i,j}\)</span> and <span class="math">\(A_{ii} 1\)</span>. Given the matrix <span class="math">\(A\)</span> we can estimate <span class="math">\(V_A\)</span>. We can also add fixed effects into this model to account for generation effects, additional mixed effects could also be included to account for shared environments between particular individuals (e.g. a shared nest). This is sometimes called the “animal model”, and also goes by the name of variance components analysis.</p>
<h4 id="multiple-traits.">Multiple traits.</h4>
<p>Traits often covary with each other, due to both environmentally induced effects (e.g. due to the effects of diet on multiple traits) and due to the expression of underlying genetic covariance between traits. In turn this genetic covariance can reflect pleiotropy, a mechanistic effect of an allele on multiple traits (e.g. variants that effect skin pigmentation often effect hair color) or the genetic linkage of loci independently affecting multiple traits. If we are interested in evolution over short time-scales we can (often) ignore the genetic basis of this correlation.</p>
<p>Consider two traits <span class="math">\(X_{1,i}\)</span> and <span class="math">\(X_{2,i}\)</span> in an indivdual <span class="math">\(i\)</span>, these could be say the indivdual’s leg length and nose length. As before we can write these as</p>
<p><span class="math">\[\begin{aligned}
X_{1,i} &amp;= \mu_1+ X_{1,A,i} + X_{1,E,i}  \nonumber \\
X_{2,i} &amp;= \mu_2 +X_{2,A,i} + X_{2,E,i} \nonumber \\\end{aligned}\]</span></p>
<p>As before we can talk about the total phenotypic variance (<span class="math">\(V_1,V_2\)</span>), environmental variance (<span class="math">\(V_{1,E}\)</span> and <span class="math">\(V_{2,E}\)</span>), and the additive variance and in trait one and two and (<span class="math">\(V_{1,A}\)</span>, <span class="math">\(V_{2,A}\)</span>). But now we also have to consider the total covariance <span class="math">\(V_{1,2}=Cov(X_{1},X_{2})\)</span>, the environmentally induced covariance between the traits (<span class="math">\(V_{E,1,2}=Cov(X_{1,E}
,X_{2,E} )\)</span>) and the additive genetic covariance (<span class="math">\(V_{A,1,2}
=Cov(X_{1,A} ,X_{2,A} )\)</span>) between trait one and two.</p>
<p>We can store these values in a matrices</p>
<p><span class="math">\[\bf{V}= \left( \begin{array}{cc} 
V_{1} &amp; V_{1,2} \\
V_{1,2} &amp; V_{2} \\
\end{array} \right) \label{P_matrix}\]</span></p>
<p>and</p>
<p><span class="math">\[\bf{G}= \left( \begin{array}{cc} 
V_{1,A} &amp; V_{A,1,2} \\
V_{A,1,2} &amp; V_{2,A} \\
\end{array} \right)  \label{G_matrix}\]</span></p>
<p>we can generalize this to an abitrary number of traits.</p>
<p>We can estimate these quantities, in a similar way to before, by studying the covariance in different traits between relatives:</p>
<p><span class="math">\[Cov(X_{1,i},X_{2,j}) = 2 F_{i,j} V_{A,1,2}\]</span></p>
<h3 id="the-response-to-selection">The response to selection</h3>
<p>Evolution by natural selection requires:</p>
<ol>
<li><p>Variation in a phenotype</p></li>
<li><p>That survival is non-random with respect to this phenotypic variation.</p></li>
<li><p>That this variation is heritable.</p></li>
</ol>
<p>Points 1 and 2 encapsulate our idea of Natural Selection, but evolution by natural selection will only occur if the 3rd condition is met. It is the heritable nature of variation that couples change within a generation due to natural selection, to change across generations (evolutionary change).<br /></p>
<p>Lets start by thinking about the change within a generation due to directional selection, where selection acts to change the mean phenotype within a generation. For example, a decrease in mean height within a generation, due to taller organisms having a lower chance of surviving to reproduction than shorter organisms. Specifically, we’ll denote our mean phenotype at reproduction by <span class="math">\(\mu_S\)</span>, i.e. after selection has acted, and our mean phenotype before selection acts by <span class="math">\(\mu_{BS}\)</span>. This second quantity may be hard to measure, as obviously selection acts throughout the life-cycle, so it might be easier to think of this as the mean phenotype if selection hadn’t acted. So the mean phenotype changes within a generation is <span class="math">\(\mu_{S} - \mu_{BS}= S\)</span>.<br /></p>
<p>We are interested in predicting the distribution of phenotypes in next generation, in particular we are interested in the mean phenotype in the next generation to understand how directional selection has contributed to evolutionary change. We’ll denote the mean phenotype in offspring, i.e. the mean phenotype in the next generation before selection acts, as <span class="math">\(\mu_{NG}\)</span>. The change across generations we’ll call the response to selection <span class="math">\(R\)</span> and put this equal to <span class="math">\(\mu_{NG}- \mu_{BS}\)</span>.<br /></p>
<p>The mean phenotype in the next generation is</p>
<p><span class="math">\[\mu_{NG} = {\mathbb{E}}\left( {\mathbb{E}}(X_{kid} | X_{mum},X_{dad}) \right)\]</span></p>
<p>where the outer expectation is over the randomly mating of individuals who survive to reproduce. We can use eqn. [predict<sub>k</sub>id] to obtain an expression for this</p>
<p><span class="math">\[\mu_{NG} = \mu_{BS} +
\beta ( {\mathbb{E}}(X_{mid}) - \mu_{BS})\]</span></p>
<p>so to obtain <span class="math">\(\mu_{NG}\)</span> we need to compute <span class="math">\({\mathbb{E}}(X_{mid})\)</span> the expected mid-point phenotype of pairs of individuals who survive to reproduce. Well this is just the expected phenotype in the individuals who survived to reproduce (<span class="math">\(\mu_{S}\)</span>), so</p>
<p><span class="math">\[\mu_{NG} = \mu_{BS} +
h^2 (\mu_S - \mu_{BS})\]</span></p>
<p>So we can write our response to selection as</p>
<p><span class="math">\[R = \mu_{NG} -\mu_{BS}  =
h^2 (\mu_S - \mu_{BS}) = h^2 S \label{breeders_eqn}\]</span></p>
<p>So our response to selection is proportional to our selection differential, and the constant of proportionality is the narrow sense heritability. This equation is sometimes termed the Breeders equation. It is a statement that the evolutionary change across generations (<span class="math">\(R\)</span>) is proportional to the change caused by directional selection within a generation, and the strength of this relationship is determined by the narrow sense heritability.<br /></p>
<p>Using the fact that <span class="math">\(h^2=V_A/V\)</span> we can rewrite this in a different form as</p>
<p><span class="math">\[R= V_A \frac{S}{V}\]</span></p>
<p>i.e. our response to selection is the additive genetic variance of our trait (<span class="math">\(V_A\)</span>) multiplied by the change within a generation as a fraction of the total phenotypic variance (<span class="math">\(S/V\)</span>, sometimes called the the selection gradient <span class="math">\(\beta\)</span>).<br /></p>
<p><embed src="Figs/QT3.eps" /></p>
<p>A change in mean phenotype within a generation occurs because of the differential fitness of our organisms. To think more carefully about this change within a generation lets think about a simple fitness model where our phenotype affects the viability of our organisms (i.e. the probability they survive to reproduce). The probability that an individual has a phenotype <span class="math">\(X\)</span> before selection is <span class="math">\(p(X)\)</span>, so that the mean phenotype before selection is</p>
<p><span class="math">\[\mu_{BS} = {\mathbb{E}}[X] =  \int_{-\infty}^{\infty} x p(x) dx\]</span></p>
<p>The probability that an organism with a phenotype <span class="math">\(X\)</span> survives to reproduce is <span class="math">\(w(X)\)</span>, and we’ll think about this as the fitness of our organism. The probability distribution of phenotypes in those who do reproduce is</p>
<p><span class="math">\[{\mathbb{P}}(X | \textrm{survive}) =  \frac{p(x) w(x)}{
\int_{-\infty}^{\infty} p(x) w(x) dx}.\]</span></p>
<p>where the denominator is a normalization constant which ensures that our phenotypic distribution integrates to one. The denominator also has the interpretation of being the mean fitness of the population, which we’ll call <span class="math">\({\overline{w}}\)</span>, i.e.</p>
<p><span class="math">\[{\overline{w}}=  
\int_{-\infty}^{\infty} p(x) w(x) dx.\]</span></p>
<p>Therefore, we can write the mean phenotype in those who survive to reproduce as</p>
<p><span class="math">\[\mu_S = \frac{1}{{\overline{w}}}\int_{-\infty}^{\infty} x p(x) w(x) dx\]</span></p>
<p>If we mean center our population, i.e. set the phenotype before selection to zero, then</p>
<p><span class="math">\[S= \frac{1}{{\overline{w}}}\int_{-\infty}^{\infty} x p(x) w(x) dx\]</span></p>
<p>if <span class="math">\(\mu_S=0\)</span>. Inspecting this more closely we can see that <span class="math">\(S\)</span> has the form of a covariance between our phenotype <span class="math">\(X\)</span> and our fitness <span class="math">\(w(X)\)</span> (<span class="math">\(Cov(X,w(X))\)</span>). Thus our change in mean phenotype is directly a measure of the covariance of our phenotype and our fitness. Rewriting our breeder’s equation using this observation we see</p>
<p><span class="math">\[R = \frac{V_A}{V}  Cov(X,w(X))\]</span></p>
<p>we see that the response to selection is due to the fact that our fitness (viability) of our organisms/parents covaries with our phenotype, and that our child’s phenotype is correlated with the parent phenotype.</p>
<h4 id="the-response-of-multiple-traits-to-selection-the-multivariate-breeders-equation.">The response of multiple traits to selection, the multivariate breeder’s equation.</h4>
<p>We can generalize these results for multiple traits, to ask how selection on multiple phenotypes plays out over short time intervals. We’ll write our change in the mean our multiple phenotypes within a generation as the vector <span class="math">\(\bf{S}\)</span> and our response across multiple generations as the vector <span class="math">\(\bf{R}\)</span>. These two quantities are related by</p>
<p><span class="math">\[\bf{R} = \bf{G} \bf{V}^{-1} \bf{S} = \bf{G} \bf{\beta}\]</span></p>
<p>where <span class="math">\(\bf{V}\)</span> and <span class="math">\(\bf{G}\)</span> are our matrices of the variance-covariance of phenotypes and additive genetic values (eqn. ) and <span class="math">\(\bf{\beta}\)</span> is a vector of selection gradients (i.e. the change within a generation as a fraction of the total phenotypic variance). To make this a bit more intuitive, consider two traits we are writing</p>
<p><span class="math">\[\begin{aligned}
R_1 &amp; = V_{A,1} \beta_1 + V_{A,1,2} \beta_2 \nonumber \\
R_2 &amp; = V_{A,2} \beta_2 + V_{A,1,2} \beta_1  \nonumber \\\end{aligned}\]</span></p>
<p>where the <span class="math">\(1\)</span> and <span class="math">\(2\)</span> index our two different traits. This is a statement that our response in any one phenotype is modified by selection on other traits that covary with that trait. This offers a good way to think about how genetic trade offs play out over evolution over short time-scales.</p>
<h1 id="one-locus-models-of-selection">One-locus models of selection</h1>
<h2 id="fitness">Fitness</h2>
<p>As we have seen, natural selection occurs when there are differences between individuals in fitness. We may define fitness in various ways. Most commonly, it is defined with respect to the contribution of a phenotype or genotype to the next generation. Differences in fitness can arise at any point during the life cycle. For instance, different genotypes or phenotypes may have different survival probabilities from one stage in their life to the stage of reproduction (viability), or they may differ in the number of offspring produced (fertility), or both. Here, we define the absolute fitness of a genotype as the expected number of offspring of an individual of that genotype.<br /></p>
<h2 id="haploid-selection-model">Haploid selection model</h2>
<p>We start out by modelling selection in a haploid model, as this is mathematically relatively simple. Let the number of individuals carrying alleles <span class="math">\(A_1\)</span> and <span class="math">\(A_2\)</span> in generation <span class="math">\(t\)</span> be <span class="math">\(P_t\)</span> and <span class="math">\(Q_t\)</span>. Then, the relative frequencies at time <span class="math">\(t\)</span> of alleles <span class="math">\(A_1\)</span> and <span class="math">\(A_2\)</span> are <span class="math">\(p_t = P_t / (P_t + Q_t)\)</span> and <span class="math">\(q_t = Q_t / (P_t + Q_t) = 1 - p_t\)</span>. Further, assume that individuals of type <span class="math">\(A_1\)</span> and <span class="math">\(A_2\)</span> on average produce <span class="math">\(W_1\)</span> and <span class="math">\(W_2\)</span> offspring individuals, respectively. We call <span class="math">\(W_i\)</span> the absolute fitness.<br /></p>
<p>Therefore, in the next generation, the absolute number of carriers of <span class="math">\(A_1\)</span> and <span class="math">\(A_2\)</span> are <span class="math">\(P_{t+1} = W_1 P_t\)</span> and <span class="math">\(Q_{t+1} = W_2 Q_t\)</span>, respectively. The mean absolute fitness of the population at time <span class="math">\(t\)</span> is</p>
<p><span class="math">\[\label{eq:meanAbsFit}
    {\overline{W}}_t = W_1 \frac{P_t}{P_t + Q_t} + W_2 \frac{Q_t}{P_t + Q_t} = W_1 p_t + W_2 q_t,\]</span></p>
<p>i.e. the sum of the fitness of the two types weighted by their relative frequencies. Note that the mean fitness depends on time, as it is a function of the allele frequencies, which are themselves time dependent.<br /></p>
<p>The frequency of allele <span class="math">\(A_1\)</span> in the next generation is then given by</p>
<p><span class="math">\[\label{eq:eq:recHaplMod1}
    p_{t+1} = \frac{P_{t+1}}{P_{t+1} + Q_{t+1}} = \frac{W_1 P_t}{W_1 P_t + W_2 Q_t}
        = \frac{W_1 p_t}{W_1 p_t + W_2 q_t} = \frac{W_1}{{\overline{W}}_t} p_t.\]</span></p>
<p>Importantly, eqn. ([eq:eq:recHaplMod1]) tells us that the change in <span class="math">\(p\)</span> only depends on a ratio of fitnesses. Therefore, we need to specify fitness only up to an arbitrary constant. As long as we multiply all fitnesses by the same value, that constant will cancel out and eqn. ([eq:eq:recHaplMod1]) will hold. Based on this argument, it is very common to scale absolute fitnesses by the absolute fitness of one of the genotypes, e.g. the most or the least fit genotype, to obtain relative fitnesses. Here, we will use <span class="math">\(w_i\)</span> for the relative fitness of genotype <span class="math">\(i\)</span>. If we choose to scale by the absolute fitness of genotype <span class="math">\(A_1\)</span>, we obtain the relative fitnesses <span class="math">\(w_1 = W_1/W_1 = 1\)</span> and <span class="math">\(w_2 = W_2/W_1\)</span>.<br />Without loss of generality, we can therefore rewrite eqn. ([eq:eq:recHaplMod1]) as</p>
<p><span class="math">\[\label{eq:recHaplMod2}
    p_{t+1} = \frac{w_1}{{\overline{w}}} p_t,\]</span></p>
<p>dropping the dependence of the mean fitness on time in our notation, but remembering it. The change in frequency from one generation to the next is then given by</p>
<p><span class="math">\[\Delta p_t = p_{t+1} - p_t= \frac{ w_1 p_t}{ {\overline{w}}} - p_t = \frac{w_1 p_t - {\overline{w}}p_t}{{\overline{w}}} = \frac{w_1 p_t - (w_1 p_t + w_2 q_t) p_t}{{\overline{w}}} = \frac{w_1 - w_2}{{\overline{w}}} p_t q_t,
\label{eq:deltap_haploid}\]</span></p>
<p>recalling that <span class="math">\(q_t = 1 - p_t\)</span>.<br /></p>
<p>Assuming that the fitnesses of the two alleles are constant over time, the number of the two allelic types <span class="math">\(\tau\)</span> generations after time <span class="math">\(t\)</span> are <span class="math">\(P_{t+\tau} = (W_1)^{\tau} P_t\)</span> and <span class="math">\(Q_{t+\tau}=  (W_2)^{\tau} Q_t\)</span>, respectively. Therefore, the relative frequency of allele <span class="math">\(A_1\)</span> after <span class="math">\(\tau\)</span> generations past <span class="math">\(t\)</span> is</p>
<p><span class="math">\[p_{t+\tau} = \frac{ (W_1)^{\tau} P_t}{ (W_1)^{\tau} P_t+(W_2)^{\tau} Q_t} = \frac{ (w_1)^{\tau} P_t}{ (w_1)^{\tau} P_t+(w_2)^{\tau} Q_t} = \frac{p_t}{p_t + (w_2/w_1)^{\tau} q_t},
    \label{eq:haploid_tau_gen}\]</span></p>
<p>where the last step includes dividing the whole term by <span class="math">\(w_1\)</span> and switching from absolute to relative allele frequencies.<br /></p>
<p>Rearranging eqn.  and setting <span class="math">\(t = 0\)</span>, we can work out the time <span class="math">\(\tau\)</span> for the frequency of <span class="math">\(A_1\)</span> to change from <span class="math">\(p_0\)</span> to <span class="math">\(p_{\tau}\)</span>. First, we write</p>
<p><span class="math">\[p_{\tau} = \frac{p_0}{p_0 + (w_2/w_1)^{\tau} q_0}\]</span></p>
<p>and rearrange this to obtain</p>
<p><span class="math">\[\label{eq:estTau}
    \frac{p_{\tau}}{q_{\tau}} = \frac{p_0}{q_0} \left(\frac{w_1}{w_2}\right)^{\tau}.\]</span></p>
<p>Solving this for <span class="math">\(\tau\)</span> yields</p>
<p><span class="math">\[\label{eq:solTau}
    \tau = \log \left(\frac{p_{\tau} q_0}{q_{\tau} p_0}\right) /  \log\left(  \frac{w_1}{w_2} \right).\]</span></p>
<p><br /></p>
<p>In practice, it is often helpful to parametrize the relative fitnesses <span class="math">\(w_i\)</span> in a specific way. For example, we may set <span class="math">\(w_1 = 1\)</span> and <span class="math">\(w_2 = 1 - s\)</span>, where <span class="math">\(s\)</span> is called the selection coefficient. Using this parametrization, <span class="math">\(s\)</span> is simply the difference in relative fitnesses between the two alleles. Equation becomes</p>
<p><span class="math">\[\label{eq:haploid_tau_gen_expl}
    p_{t+\tau} = \frac{p_{t}}{p_{t} + q_{t} (1 - s)^{\tau}},\]</span></p>
<p>as <span class="math">\(w_2 / w_1 = 1 - s\)</span>. Then, if <span class="math">\(s \ll 1\)</span>, we can approximate <span class="math">\((1-s)^{\tau}\)</span> in the denominator by <span class="math">\(\exp(-s\tau)\)</span> to obtain</p>
<p><span class="math">\[\label{eq:haploid_logistic growth}
    p_{t+\tau} \approx \frac{p_t}{p_t + q_t e^{-s\tau}}.\]</span></p>
<p>This equation takes the form of a logistic function. That is because we are looking at the relative frequencies of two ‘populations’ (of alleles <span class="math">\(A_1\)</span> and <span class="math">\(A_2\)</span>) that are growing (or declining) exponentially, under the constraint that <span class="math">\(p\)</span> and <span class="math">\(q\)</span> always sum to 1.<br /></p>
<p>Moreover, eqn.  for the time <span class="math">\(\tau\)</span> it takes for a certain change in frequency to occur becomes</p>
<p><span class="math">\[\label{eq:estTauExpl}
    \tau = - \log \left(\frac{p_{\tau} q_0}{q_{\tau} p_0}\right) /  \log\left(1-s\right).\]</span></p>
<p>Assuming again that <span class="math">\(s \ll 1\)</span>, this simplifies to</p>
<p><span class="math">\[\label{eq:estTauExplSimpl}
    \tau \approx \frac{1}{s} \log \left(\frac{p_{\tau} q_0}{q_{\tau} p_0}\right).\]</span></p>
<p>One particular case of interest is the time it takes to go from an absolute frequency of 1 to near fixation in a population of size <span class="math">\(N\)</span>. In this case, we have <span class="math">\(p_0 = 1/N\)</span>, and we may set <span class="math">\(p_{\tau} = 1 - 1/N\)</span>, which is very close to fixation. Of course, we then have <span class="math">\(q_0 = 1 - 1/N\)</span> and <span class="math">\(q_{\tau} = 1 - 1/N\)</span>. If <span class="math">\(N\)</span> is sufficiently large, we may for mathematical convenience approximate <span class="math">\(q_0\)</span> by <span class="math">\(q_0 = 1\)</span>. Plugging these values into eqn. , we obtain</p>
<p><span class="math">\[\label{eq:fixTimeSimpl}
    \tau \approx \frac{1}{s} \log\left( \frac{1 - 1/N}{(1 - 1/N)\ 1/N}  \right) = \frac{1}{s} \log(N)\]</span></p>
<p>as an approximation for the time to fixation.</p>
<h4 id="haploid-model-with-fluctuating-selection">Haploid model with fluctuating selection</h4>
<p>We can now consider the case where the fitnesses depend on time, and say that <span class="math">\(w_{1,t}\)</span> and <span class="math">\(w_{2,t}\)</span> are the fitnesses of the two types in generation <span class="math">\(t\)</span>. The frequency of allele <span class="math">\(A_1\)</span> in generation <span class="math">\(t+1\)</span> is</p>
<p><span class="math">\[p_{t+1} = \frac{w_{1,t}}{{\overline{w}}_t} p_t,\]</span></p>
<p>which simply follows from eqn. . The ratio of the frequency of allele <span class="math">\(A_1\)</span> to that of allele <span class="math">\(A_2\)</span> in generation <span class="math">\(t+1\)</span> is</p>
<p><span class="math">\[\frac{p_{t+1}}{q_{t+1}} = \frac{w_{1,t}}{w_{2,t}}  \frac{p_{t}}{q_{t}}.\]</span></p>
<p>Therefore, if we think of the two alleles starting in generation <span class="math">\(t\)</span> at frequencies <span class="math">\(p_t\)</span> and <span class="math">\(q_t\)</span>, then <span class="math">\(\tau\)</span> generations later,</p>
<p><span class="math">\[\frac{p_{t+\tau}}{q_{t+\tau}} = \left(\prod_{i=t}^{\tau-1} \frac{w_{1,i}}{w_{2,i}}  \right) \frac{p_{t}}{q_{t}}.\]</span></p>
<p><br /></p>
<p>The question of which allele is increasing or decreasing in frequency comes down to whether <span class="math">\(\left(\prod_{i=t}^{\tau-1} \frac{w_{1,i}}{w_{2,i}}  \right)\)</span> is <span class="math">\(&gt;1\)</span> or <span class="math">\(&lt;1\)</span>. As it is a little hard to think about this ratio, we can instead take the <span class="math">\(\tau^{\mathrm{th}}\)</span> root of it and consider</p>
<p><span class="math">\[\sqrt[\tau]{\left(\prod_{i=t}^{\tau-1} \frac{w_{1,i}}{w_{2,i}}  \right)} = \frac{\sqrt[\tau]{\prod_{i=t}^{\tau-1}w_{1,i}}}{\sqrt[\tau]{\prod_{i=t}^{\tau-1}w_{2,i}}}.\]</span></p>
<p>The term <span class="math">\(\sqrt[\tau]{\prod_{i=t}^{\tau-1}w_{1,i}}\)</span> is the geometric mean fitness of allele <span class="math">\(A_1\)</span> over the <span class="math">\(\tau\)</span> generations past generation <span class="math">\(t\)</span>. Therefore, allele <span class="math">\(A_1\)</span> will only increase in frequency if it has a higher geometric mean fitness than allele <span class="math">\(A_2\)</span> (at least in our simple deterministic model).<br /></p>
<h2 id="diploid-model">Diploid model</h2>
<p>We will now move on to a diploid model of a single locus with two segregating alleles. We will assume that the difference in fitness between the three genotypes comes from differences in viability, i.e. differential survival of individuals from the formation of zygotes to reproduction. We denote the absolute fitnesses of genotypes <span class="math">\(A_1A_1\)</span>, <span class="math">\(A_1A_2\)</span>, and <span class="math">\(A_2A_2\)</span> by <span class="math">\(W_{11}\)</span>, <span class="math">\(W_{12}\)</span>, and <span class="math">\(W_{22}\)</span>. Specifically, <span class="math">\(W_{ij}\)</span> is the probability that a zygote of genotype <span class="math">\(A_iA_j\)</span> survives to reproduction. Assuming that individuals mate at random, the number of zygotes that are of the three genotypes and form generation <span class="math">\(t\)</span> are</p>
<p><span class="math">\[Np_t^2, ~~~  N2p_tq_t, ~~~ Nq_t^2.\]</span></p>
<p>The mean fitness of the population of zygotes is then</p>
<p><span class="math">\[{\overline{W}}_t = W_{11} p_t^2+W_{12} 2p_tq_t  +  W_{22} q_t^2.\]</span></p>
<p>Again, this is simply the weighted mean of the genotypic fitnesses.<br /></p>
<p>How many zygotes of each of the three genotypes survive to reproduce? An individual of genotype <span class="math">\(A_1A_1\)</span> has a probability of <span class="math">\(W_{11}\)</span> of surviving to reproduce, and similarly for other genotypes. Therefore, the expected number of <span class="math">\(A_1A_1\)</span>, <span class="math">\(A_1A_2\)</span>, and <span class="math">\(A_2A_2\)</span> individuals who survive to reproduce is</p>
<p><span class="math">\[NW_{11} p_t^2, ~~~ NW_{12} 2p_tq_t , ~~~ N W_{22} q_t^2.\]</span></p>
<p>It then follows that the total number of individuals who survive to reproduce is</p>
<p><span class="math">\[N \left(W_{11} p_t^2+W_{12} 2p_tq_t  +  W_{22} q_t^2 \right).\]</span></p>
<p>This is simply the mean fitness of the population multiplied by the population size (i.e. <span class="math">\(N {\overline{w}}\)</span>).<br /></p>
<p>The relative frequency of <span class="math">\(A_1A_1\)</span> individuals at reproduction is simply the number of <span class="math">\(A_1A_1\)</span> genotype individuals at reproduction (<span class="math">\(NW_{11} p_t^2\)</span>) divided by the total number of individuals who survive to reproduce (<span class="math">\(N {\overline{W}}\)</span>), and likewise for the other two genotypes. Therefore, the relative frequency of individuals with the three different genotypes at reproduction is</p>
<p><span class="math">\[\frac{NW_{11} p_t^2}{N{\overline{W}}}, ~~~ \frac{NW_{12} 2p_tq_t}{N{\overline{W}}} , ~~~ \frac{N W_{22} q_t^2}{N{\overline{W}}}\]</span></p>
<p>(see Table [dip<sub>f</sub>itness<sub>t</sub>able]).<br /></p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: center;"><span class="math">\(A_1A_1\)</span></th>
<th style="text-align: center;"><span class="math">\(A_1A_2\)</span></th>
<th style="text-align: center;"><span class="math">\(A_2A_2\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Absolute no. at birth</td>
<td style="text-align: center;"><span class="math">\(Np_t^2\)</span></td>
<td style="text-align: center;"><span class="math">\(N2p_tq_t\)</span></td>
<td style="text-align: center;"><span class="math">\(Nq_t^2\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">Fitnesses</td>
<td style="text-align: center;"><span class="math">\(W_{11}\)</span></td>
<td style="text-align: center;"><span class="math">\(W_{12}\)</span></td>
<td style="text-align: center;"><span class="math">\(W_{22}\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;">Absolute no. at reproduction</td>
<td style="text-align: center;"><span class="math">\(NW_{11} p_t^2\)</span></td>
<td style="text-align: center;"><span class="math">\(NW_{12} 2p_tq_t\)</span></td>
<td style="text-align: center;"><span class="math">\(N W_{22} q_t^2\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">Relative freq. at reproduction</td>
<td style="text-align: center;"><span class="math">\(\frac{NW_{11} p_t^2}{N {\overline{W}}} = \frac{W_{11}}{{\overline{W}}} p_{t}^2\)</span></td>
<td style="text-align: center;"><span class="math">\(\frac{NW_{12} 2p_tq_t}{N
{\overline{W}}} = \frac{W_{12}}{{\overline{W}}} 2 p_{t} q_{t}\)</span></td>
<td style="text-align: center;"><span class="math">\(\frac{N W_{22} q_t^2}{N{\overline{W}}} = \frac{W_{22}}{{\overline{W}}} q_{t}^2\)</span></td>
</tr>
</tbody>
</table>
<p>[dip<sub>f</sub>itness<sub>t</sub>able]</p>
<p>As there is no difference in the fecundity of the three genotypes, the allele frequencies in the zygotes forming the next generation are simply the allele frequency among the reproducing individuals of the previous generation. Hence, the frequency of <span class="math">\(A_1\)</span> in generation <span class="math">\(t+1\)</span> is</p>
<p><span class="math">\[p_{t+1} = \frac{W_{11} p_t^2 + W_{12} p_tq_t}{{\overline{W}}}
    \label{pgen_dip}.\]</span></p>
<p>Note that, again, the absolute value of the fitnesses is irrelevant to the frequency of the allele. Therefore, we can just as easily replace the absolute fitnesses with the relative fitnesses. That is, we may replace <span class="math">\(W_{ij}\)</span> by <span class="math">\(w_{ij} = W_{ij}/W_{11}\)</span>, for instance.<br /></p>
<p>The change in frequency from generation <span class="math">\(t\)</span> to <span class="math">\(t+1\)</span> is</p>
<p><span class="math">\[\Delta p_t = p_{t+1} -p_{t}= \frac{w_{11} p_t^2 + w_{12} p_tq_t}{{\overline{w}}} - p_t. \label{deltap_dip1}\]</span></p>
<p>To simplify this equation, we will first define two variables <span class="math">\({\overline{w}}_1\)</span> and <span class="math">\({\overline{w}}_2\)</span> as</p>
<p><span class="math">\[\begin{aligned}
    {\overline{w}}_1 &amp; = w_{11} p_t + w_{12} q_t, \\
    {\overline{w}}_2 &amp; =  w_{12} p_t+ w_{22} q_t.\end{aligned}\]</span></p>
<p>These are called the marginal fitnesses of allele <span class="math">\(A_1\)</span> and <span class="math">\(A_2\)</span>, respectively. They are so called as <span class="math">\({\overline{w}}_1\)</span> is the average fitness of an allele <span class="math">\(A_1\)</span>, i.e. the fitness of <span class="math">\(A_1\)</span> in a homozygote weighted by the probability it is in a homozygote (<span class="math">\(p_t\)</span>) plus the fitness of <span class="math">\(A_1\)</span> in a heterozygote weighted by the probability it is in a heterozygote (<span class="math">\(q_t\)</span>). We further note that the mean relative fitness can be expressed in terms of the marginal fitnesses as</p>
<p><span class="math">\[\label{eq:meanFitInTermsOfMargFit}
    {\overline{w}}= {\overline{w}}_1 p_t + {\overline{w}}_2 q_t,\]</span></p>
<p>where, for notational simplicity, we have omitted the dependence of mean and marginal fitnesses on time.<br /></p>
<p>We can then rewrite eqn.  using <span class="math">\({\overline{w}}_1\)</span> and <span class="math">\({\overline{w}}_2\)</span> as</p>
<p><span class="math">\[\Delta p_t = \frac{ ({\overline{w}}_1-{\overline{w}}_2)}{{\overline{w}}} p_t q_t.
    \label{deltap_dip2}\]</span></p>
<p>The sign of <span class="math">\(\Delta p_t\)</span>, i.e. whether allele <span class="math">\(A_1\)</span> increases of decreases in frequency, depends only on the sign of <span class="math">\(({\overline{w}}_1-{\overline{w}}_2)\)</span>. The frequency of <span class="math">\(A_1\)</span> will keep increasing over the generations so long as its marginal fitness is higher than that of <span class="math">\(A_2\)</span>, i.e. <span class="math">\({\overline{w}}_1 &gt; {\overline{w}}_2\)</span>, while if <span class="math">\({\overline{w}}_1 &lt; {\overline{w}}_2\)</span>, the frequency of <span class="math">\(A_1\)</span> will decrease. Note the similarity between eqn.  and the respective expression for the haploid model in eqn. . (We will return to the special case where <span class="math">\({\overline{w}}_1 = {\overline{w}}_2\)</span> shortly).<br /></p>
<p>We can also rewrite as the demonstration of this we leave to the reader. <span><span> This form shows that the frequency of <span class="math">\(A_1\)</span> will increase (<span class="math">\(\Delta p_t &gt; 0\)</span>) if the mean fitness is an increasing function of the frequency of <span class="math">\(A_1\)</span> (i.e. if <span class="math">\(\frac{d {\overline{w}}}{dp}&gt;0\)</span>). On the other hand, the frequency of <span class="math">\(A_1\)</span> will decrease (<span class="math">\(\Delta p_t &lt; 0\)</span>) if the mean fitness is a decreasing function of the frequency of <span class="math">\(A_1\)</span> (i.e. if <span class="math">\(\frac{d {\overline{w}}}{dp}&lt;0\)</span>).</span></span> Thus, although selection acts on individuals, under this simple model, selection is acting to increase the mean fitness of the population. <span><span> The rate of this increase is proportional to the variance in allele frequencies within the population (<span class="math">\(p_tq_t\)</span>).</span></span><br /></p>
<p><span><strong>Question</strong></span> Show that eqns.  and are equivalent.<br /></p>
<p><span><span> So far, our treatment of the diploid model of selection has been in terms of generic fitnesses <span class="math">\(w_{ij}\)</span>. In the following, we will use particular parametrizations to gain insight about two specific modes of selection: directional selection and heterozygote advantage.</span></span></p>
<h3 id="diploid-directional-selection">Diploid directional selection</h3>
<p><span><span> Directional selection means that</span></span> one of the two alleles always has higher marginal fitness than the other one. Let us assume that <span class="math">\(A_1\)</span> is the fitter allele, so that <span class="math">\(w_{11} \geq w_{12} \geq w_{22}\)</span>, <span><span> and hence <span class="math">\({\overline{w}}_1 &gt; {\overline{w}}_2\)</span></span></span>. As we are interested in changes in allele frequencies, we <span><span> may use</span></span> relative fitnesses. We parameterize the reduction in relative fitness in terms of a selection coefficient, similar to the one we met in the haploid selection section, as follows:<br /></p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">genotype</td>
<td style="text-align: center;"><span class="math">\(A_1A_1\)</span></td>
<td style="text-align: center;"><span class="math">\(A_1A_2\)</span></td>
<td style="text-align: center;"><span class="math">\(A_2A_2\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">absolute fitness</td>
<td style="text-align: center;"><span class="math">\(W_{11}\)</span></td>
<td style="text-align: center;"><span class="math">\( \geq W_{12} \geq\)</span></td>
<td style="text-align: center;"><span class="math">\(W_{22}\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;">relative fitness (generic)</td>
<td style="text-align: center;"><span class="math">\(w_{11} = W_{11}/W_{11}\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{12} = W_{12}/W_{11}\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{22} = W_{22}/W_{11}\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">relative fitness (specific)</td>
<td style="text-align: center;"><span class="math">\(1\)</span></td>
<td style="text-align: center;"><span class="math">\(1-sh\)</span></td>
<td style="text-align: center;"><span class="math">\(1-s\)</span>.</td>
</tr>
</tbody>
</table>
<p><br /></p>
<p>Here, the selection coefficient <span class="math">\(s\)</span> is the difference in relative fitness between the two homozygotes, and <span class="math">\(h\)</span> is the dominance coefficient. <span><span> For selection to be directional, we require that <span class="math">\(0 \leq h \leq 1\)</span> holds. The dominance coefficient allows us to move between two extremes. One is when <span class="math">\(h = 0\)</span>, such that allele <span class="math">\(A_1\)</span> is fully dominant and <span class="math">\(A_2\)</span> fully recessive. In this case, the heterozygote <span class="math">\(A_1A_2\)</span> is as fit as the <span class="math">\(A_1A_1\)</span> homozogote genotype. The inverse holds when <span class="math">\(h = 1\)</span>, such that allele <span class="math">\(A_1\)</span> is fully recessive and <span class="math">\(A_2\)</span> fully dominant.</span></span><br /></p>
<p>We can then rewrite eqn.  as</p>
<p><span class="math">\[\Delta p_t = \frac{p_ths + q_t s(1-h)}{{\overline{w}}}p_tq_t ,
\label{deltap_direct}\]</span></p>
<p>where</p>
<p><span class="math">\[{\overline{w}}_t = 1-2p_tq_t sh-q_t^2s.\]</span></p>
<p><br /></p>
<p><span><span> A special case is when <span class="math">\(h = 0.5\)</span>. This case is the case of no dominance, as the interaction among alleles with respect to fitness is strictly additive. Then, eqn.  simplifies to</span></span></p>
<p><span class="math">\[\Delta p_t = \frac{1}{2}\frac{s}{{\overline{w}}}p_tq_t .
    \label{deltap_add}\]</span></p>
<p>If selection is very weak, i.e. <span class="math">\(s \ll 1\)</span>, the denominator (<span class="math">\({\overline{w}}\)</span>) is close to <span class="math">\(1\)</span> and <span><span> we have</span></span></p>
<p><span class="math">\[\Delta p_t = \frac{1}{2} s p_t q_t .
    \label{deltap_add_simpl}\]</span></p>
<p><span class="math">\[\tau \approx \frac{2}{s} \log(2N)\]</span></p>
<p>generations for the favourable allele (<span class="math">\(A_1\)</span>) to transit from its entry into the population (<span class="math">\(p_0 =1/(2N)\)</span>) to close to fixation (<span class="math">\(p_{\tau} =1-1/(2N)\)</span>). <span><span> Note again the difference by a factor of 2 due to the choice of parametrization. Also, the total number of alleles is <span class="math">\(2N\)</span> in the diploid model, rather than <span class="math">\(N\)</span>, which explains another factor of 2 in the argument of the logarithm.</span></span><br /></p>
<p><embed src="Figs/simple_diploid_trajs.eps" /></p>
<p>[fig:diploid<sub>t</sub>raj]</p>
<p><span><span> <span><strong>Question</strong></span> Comparing the red (<span class="math">\(h=0\)</span>) and black (<span class="math">\(h=0.5\)</span>) trajectories in Figure [fig:diploid<sub>t</sub>raj], provide an explanation for why <span class="math">\(A_1\)</span> increases faster initially if <span class="math">\(h=0\)</span>, but then approaches fixation more slowly compared to the case of <span class="math">\(h=0.5\)</span>.</span></span><br /></p>
<h3 id="heterozygote-advantage">Heterozygote advantage</h3>
<p>What if the heterozygotes are fitter than either of the homozygotes? In this case, it is useful to parameterize the relative fitnesses as follows:<br /></p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">genotype</td>
<td style="text-align: center;"><span class="math">\(A_1A_1\)</span></td>
<td style="text-align: center;"><span class="math">\(A_1A_2\)</span></td>
<td style="text-align: center;"><span class="math">\(A_2A_2\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">absolute fitness</td>
<td style="text-align: center;"><span class="math">\(w_{11}\)</span></td>
<td style="text-align: center;"><span class="math">\(&lt;w_{12}&gt;\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{22}\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;">relative fitness (generic)</td>
<td style="text-align: center;"><span class="math">\(w_{11}=W_{11}/W_{12}\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{12} = W_{12}/W_{12}\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{22} = W_{22}/W_{12}\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">relative fitness (specific)</td>
<td style="text-align: center;"><span class="math">\(1-s_1\)</span></td>
<td style="text-align: center;"><span class="math">\(1\)</span></td>
<td style="text-align: center;"><span class="math">\(1-s_2\)</span></td>
</tr>
</tbody>
</table>
<p><br /></p>
<p>Here, <span class="math">\(s_1\)</span> and <span class="math">\(s_2\)</span> are the differences between the relative fitnesses of the two homozygotes and the heterozygote. Note that to obtain relative fitnesses we have divided absolute fitness by the heterozygote fitness. We could use the same parameterization as in the model of directional selection, but the reparameterization we have choisen here makes the math prettier.<br /></p>
<p>In this case, when allele <span class="math">\(A_1\)</span> is rare, it is often found in a heterozygous state and so it increases in frequency. However, when allele <span class="math">\(A_1\)</span> is common, it is often found in the homozygote state, while the allele <span class="math">\(A_2\)</span> is often found in the heterozygote state; it is now <span class="math">\(A_2\)</span> that increases in frequency at the expense of allele <span class="math">\(1\)</span>. Thus, at least in the deterministic model, neither allele can reach fixation and both alleles will be maintained as a balanced polymorphism in the population at an equilibrium frequency.<br /></p>
<p>We can solve for this equilibrium frequency by setting <span class="math">\(\Delta p_t = 0\)</span> in eqn. , i.e. <span class="math">\(p_tq_t ({\overline{w}}_1-{\overline{w}}_2)=0\)</span>. Doing so, we find that there are three equilibria, all of which are stable. Two of them are not very interesting (<span class="math">\(p=0\)</span> or <span class="math">\(q=0\)</span>), but the third one is the polymorphic equilibrium, where <span class="math">\({\overline{w}}_1-{\overline{w}}_2\)</span> holds. Using the parametrization above, we see that the marginal fitnesses of the two alleles are formally equivalent. Insertion of the selection coefficients <span class="math">\(s_1\)</span> and <span class="math">\(s_2\)</span> yields</p>
<p><span class="math">\[p_e = \frac{s_2}{s_1+s_2}\]</span></p>
<p>for the equilibrium frequency of interest. This is also the frequency of <span class="math">\(A_1\)</span> at which the mean fitness of the population is maximised.<br /></p>
<h4 id="underdominance.">Underdominance.</h4>
<p>Another case that is of potential interest is the case of fitness underdominance, where the heterozygote is less fit than either of the homozygotes. This can be parametrized as follows:<br /></p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">genotype</td>
<td style="text-align: center;"><span class="math">\(A_1A_1\)</span></td>
<td style="text-align: center;"><span class="math">\(A_1A_2\)</span></td>
<td style="text-align: center;"><span class="math">\(A_2A_2\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">absolute fitness</td>
<td style="text-align: center;"><span class="math">\(w_{11}\)</span></td>
<td style="text-align: center;"><span class="math">\(&lt;w_{12}&gt;\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{22}\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;">relative fitness (generic)</td>
<td style="text-align: center;"><span class="math">\(w_{11}=W_{11}/W_{12}\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{12} = W_{12}/W_{12}\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{22} = W_{22}/W_{12}\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">relative fitness (specific)</td>
<td style="text-align: center;"><span class="math">\(1+s_1\)</span></td>
<td style="text-align: center;"><span class="math">\(1\)</span></td>
<td style="text-align: center;"><span class="math">\(1+s_2\)</span></td>
</tr>
</tbody>
</table>
<p><br /></p>
<p>This case also permits three equilibria, <span class="math">\(p=0\)</span>, <span class="math">\(p=1\)</span>, and a polymorphic equilibrium <span class="math">\(p=p_U\)</span>. However, now only the first two equilibria are stable, while the polymorphic equilibrium is unstable. If <span class="math">\(p&lt;p_U\)</span> then <span class="math">\(\Delta p_t \)</span> is negative and allele <span class="math">\(A_1\)</span> will be lost, while if <span class="math">\(p&gt;p_U\)</span>, allele <span class="math">\(A_1\)</span> will become fixed.<br /></p>
<p>While such alleles might not spread within populations (if <span class="math">\(p_U \gg
0\)</span> and selection is reasonably strong), they are of interest in the study of speciation and hybrid zones. That is because alleles <span class="math">\(A_1\)</span> and <span class="math">\(A_2\)</span> may have arisen in a stepwise fashion, i.e. not by a single mutation, in separate subpopulations. Now, heterozygote disadvantage will play a potential role in species maintenance, <span><span> if isolation of the subpopulations is not complete</span></span>.<br /></p>
<h4 id="diploid-fluctuating-fitness">Diploid fluctuating fitness</h4>
<p>We would like to think about the case where the diploid absolute fitnesses are time-dependent. The three genotypes then have fitnesses <span class="math">\(w_{11,t}\)</span>, <span class="math">\(w_{12,t}\)</span>, and <span class="math">\(w_{22,t}\)</span> in generation <span class="math">\(t\)</span>. However, this case is much less tractable than the haploid case, as segregation makes it tricky to keep track of the genotype frequencies. We can make some progress and gain some intuition by thinking about how the frequency of allele <span class="math">\(A_1\)</span> changes when it is rare.<br /></p>
<p>When <span class="math">\(A_1\)</span> is rare, i.e. <span class="math">\(p_t \ll 1\)</span>, its frequency in the next generation can be approximated as</p>
<p><span class="math">\[p_{t+1} \approx \frac{w_{12}}{{\overline{w}}} p_t.\]</span></p>
<p>To obtain this, we have ignored the <span class="math">\(p_{t}^2\)</span> term and assumed that <span class="math">\(q_t \approx 1\)</span> in the numerator. Following a similar argument to approximate <span class="math">\(q_{t+1}\)</span>, we can write</p>
<p><span class="math">\[\frac{p_{t+1}}{q_{t+1}} = \frac{w_{12,t}}{w_{22,t}}  \frac{p_{t}}{q_{t}}.\]</span></p>
<p>Then, starting from out from <span class="math">\(p_0\)</span> and <span class="math">\(q_0\)</span> in generation <span class="math">\(0\)</span>, <span class="math">\(t+1\)</span> generations later, we have</p>
<p><span class="math">\[\frac{p_{t+1}}{q_{t+1}} = \left( \prod_{i=0}^{t-1} \frac{w_{12,i}}{w_{22,i}}  \right) \frac{p_{0}}{q_{0}}.\]</span></p>
<p>From this, we can see, following our haploid argument from above, that the frequency of allele <span class="math">\(A_1\)</span> will increase when rare only if</p>
<p><span class="math">\[\frac{\sqrt[t]{\prod_{i=0}^{t-1}w_{12,i}}}{\sqrt[t]{\prod_{i=0}^{t-1}w_{22,i}}}&gt;1 \label{geometric_1wins},\]</span></p>
<p>i.e. if the heterozygote has higher geometric mean fitness than the <span class="math">\(A_2A_2\)</span> homozygote.<br /></p>
<p>The question now is, whether allele <span class="math">\(A_1\)</span> will approach fixation in the population, or whether there are cases in which we can obtain a balanced polymorphism. To investigate that, we can simply repeat our analysis for <span class="math">\(q \ll 1\)</span>, and see that in that case</p>
<p><span class="math">\[\frac{p_{t+1}}{q_{t+1}} = \left( \prod_{i=0}^{t-1} \frac{w_{11,i}}{w_{12,i}}  \right) \frac{p_{0}}{q_{0}}.\]</span></p>
<p>Now, for allele <span class="math">\(A_1\)</span> to carry on increasing in frequency and to approach fixation, the <span class="math">\(A_1A_1\)</span> genotype has to be out-compeeting the heterozygotes. For allele <span class="math">\(A_1\)</span> to approach fixation, we need the geometric mean of <span class="math">\(w_{11,i}\)</span> to be greater than the geometric mean fitness of heterozygotes (<span class="math">\(w_{12,i}\)</span>). At the same time, if heterozygotes have higher geometric mean fitness than the <span class="math">\(A_1A_1\)</span> homozygotes, then the <span class="math">\(A_2\)</span> allele will increase in frequency when it is rare. Therefore, a balanced polymorphism can result when the heterozygote has higher geometric fitness than either of the homozygotes.<br /></p>
<p>Intriguingly, we can have a balanced polymorphism even if the heterozygote is never the fittest genotype in any generation. To see this, consider the simple example, where there are two environments alternate from generation to generation:<br /></p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">genotype</td>
<td style="text-align: center;"><span class="math">\(A_1A_1\)</span></td>
<td style="text-align: center;"><span class="math">\(A_1A_2\)</span></td>
<td style="text-align: center;"><span class="math">\(A_2A_2\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">relative fitness in environment A</td>
<td style="text-align: center;"><span class="math">\(w_{11,A}\)</span></td>
<td style="text-align: center;"><span class="math">\(&gt;w_{12,A}&gt;\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{22,A}\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;">relative fitness in environment B</td>
<td style="text-align: center;"><span class="math">\(w_{11,B}\)</span></td>
<td style="text-align: center;"><span class="math">\(&lt;w_{12,B}&lt;\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{22,B}\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">Geometric mean fitness</td>
<td style="text-align: center;"><span class="math">\(w_{11,B}\)</span></td>
<td style="text-align: center;"><span class="math">\(&lt;w_{12,B}&gt;\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{22}\)</span></td>
</tr>
</tbody>
</table>
<p><br /></p>
<p>In this case, the polymorphism will remain balanced in the population, despite the fact that the heterozygote is never the fitest genotype.</p>
<h2 id="mutationselection-balance">Mutation–selection balance</h2>
<p>Mutation is constantly introducing new alleles into the population. Therefore, variation can be maintained within a population <span><span> not only if selection is balancing (e.g. through heterozygote advantage or fluctuating selection over time, as we have seen in the previous section), but also due to</span></span> a balance between mutation <span><span> and selection. A case of particular interest is when mutation introduces deleterious</span></span> alleles and selection acts against these alleles. To study this balance, we return to the model of directional selection, where allele <span class="math">\(A_1\)</span> is advantageous, i.e.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">genotype</td>
<td style="text-align: center;"><span class="math">\(A_1A_1\)</span></td>
<td style="text-align: center;"><span class="math">\(A_1A_2\)</span></td>
<td style="text-align: center;"><span class="math">\(A_2A_2\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">absolute fitness</td>
<td style="text-align: center;"><span class="math">\(W_{11}\)</span></td>
<td style="text-align: center;"><span class="math">\( \geq W_{12} \geq\)</span></td>
<td style="text-align: center;"><span class="math">\(W_{22}\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;">relative fitness</td>
<td style="text-align: center;"><span class="math">\(w_{11}=1\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{12}=1-sh\)</span></td>
<td style="text-align: center;"><span class="math">\(w_{22}=1-s\)</span>.</td>
</tr>
</tbody>
</table>
<p><br /></p>
<p><span><span> For a start, we consider the case where allele <span class="math">\(A_2\)</span> is not completely recessive (<span class="math">\(h&gt;0\)</span>), so that the heterozygotes suffer at least some disadvantage. We denote by <span class="math">\(\mu = \mu_{1\rightarrow2}\)</span> the mutation rate per generation from <span class="math">\(A_1\)</span> to the deleterious allele <span class="math">\(A_2\)</span>, and assume that there is no reverse mutation (<span class="math">\(\mu_{2\rightarrow1} = 0\)</span>). Let us assume that selection against <span class="math">\(A_2\)</span> is relatively strong compared to the mutation rate, so that it is justified to assume that <span class="math">\(A_2\)</span> is always rare, i.e. <span class="math">\(q_t = 1-p_t \ll 1\)</span>. Compared to previous sections, for mathematical clarity, we also switch from following the frequency <span class="math">\(p_t\)</span> of <span class="math">\(A_1\)</span> to following the frequency <span class="math">\(q_t\)</span> of <span class="math">\(A_2\)</span>. Of course, this is without loss of generality. The change in frequency of <span class="math">\(A_2\)</span> due to selection can be written as</span></span></p>
<p><span class="math">\[\Delta_S q_t = \frac{{\overline{w}}_2 - {\overline{w}}_1}{{\overline{w}}} p_t q_t  \approx  -hs q_t.
    \label{eq:dirSelApprox}\]</span></p>
<p>This approximation can be found by assuming that <span class="math">\(q^2 \approx 0\)</span>, <span class="math">\(p \approx 1\)</span>, and that <span class="math">\({\overline{w}}\approx w_1\)</span>.<br /></p>
<h3 id="inbreeding-depression">Inbreeding depression</h3>
<p>All else being equal, eqn.  suggests that mutations that have a smaller effect in the heterozygote can segregate at higher frequency under mutation–selection balance. As a consequence, alleles that have strongly deleterious effects in the homozygous state can segregate at low frequencies in the population, as long as they do not have a strong effect in heterozygotes. Thus, outbred populations may have many alleles with recessive deleterious effects segregating within them.</p>
<p>On consequence of this is that inbred individuals from usually outbred populations may have dramatically lower fitnesses than outbred individuals. This is a consequence of being homozygous at many loci for alleles with recessive deleterious effects. Indeed, this seems to be a common observation, dating back to systematic surveys by Darwin. In typically outbred populations, the mean fitness of individuals decreases with the inbreeding coefficient, i.e. this so-called inbreeding depression is a common observation.</p>
<h4 id="purging-the-inbreeding-load.">Purging the inbreeding load.</h4>
<p>That said, populations that regularly inbreed over sustained periods of time are expected to partially purge this load of deleterious alleles. This is because such populations have exposed many of these alleles in a homozygous state, and so selection can more readily remove these alleles from the population.</p>
<h2 id="migrationselection-balance">Migration–selection balance</h2>
<p>Another reason for the persistence of deleterious alleles in a population is that there is a constant influx of maladaptive alleles from other populations where these alleles are locally adapted. This seems unlikely to be as broad an explanation for the persistence of deleterious alleles genome-wide as mutation-selection balance. However, a brief discussion of such alleles is worthwhile as it helps to inform our ideas about local adaptation.<br /></p>
<p>As a first pass at this lets consider a haploid two allele model with two different populations, where the relative fitnesses of our alleles are as follows</p>
<table>
<thead>
<tr class="header">
<th style="text-align: center;">allele</th>
<th style="text-align: center;"><span class="math">\(1\)</span></th>
<th style="text-align: center;"><span class="math">\(2\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">population 1</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">1-s</td>
</tr>
<tr class="even">
<td style="text-align: center;">population 2</td>
<td style="text-align: center;">1-s</td>
<td style="text-align: center;">1</td>
</tr>
</tbody>
</table>
<p>As a simple model of migration lets suppose within a population a fraction of <span class="math">\(m\)</span> individuals are migrants from the other population, and <span class="math">\(1-m\)</span> individuals are from the same deme.<br /></p>
<p>To quickly sketch a solution to this well set up a situation analogous to our mutation-selection balance model. to do this lets assume that selection is strong compared to migration (<span class="math">\(s \gg m\)</span>) then allele <span class="math">\(1\)</span> will be almost fixed in population <span class="math">\(1\)</span> and allele <span class="math">\(2\)</span> will be almost fixed in population <span class="math">\(2\)</span>. If that is the case, migration changes the frequency of allele <span class="math">\(2\)</span> in population <span class="math">\(1\)</span> (<span class="math">\(q_1\)</span>) by</p>
<p><span class="math">\[\Delta_{Mig.} q_1 \approx m\]</span></p>
<p>while as noted above <span class="math">\(\Delta_{S} q_1= -sq_1\)</span>, so that migration and selection are at an equilibrium when <span class="math">\(\Delta_{S} q_1+
\Delta_{Mig.}q_1\)</span>, i.e. an equilibrium frequency of allele <span class="math">\(2\)</span> in population <span class="math">\(1\)</span> of</p>
<p><span class="math">\[q_{e,1} = \frac{m}{s}\]</span></p>
<p>so that migration is playing to role of mutation and so migration-selection balance (at least under strong selection) is analogous to mutation selection balance.<br /></p>
<h3 id="some-theory-of-the-spatial-distribution-of-allele-frequencies-under-deterministic-models-of-selection">Some theory of the spatial distribution of allele frequencies under deterministic models of selection</h3>
<p>Imagine a continuous haploid population spread out along a line. individual dispersals a random distance <span class="math">\(\Delta x\)</span> from its birthplace to the location where it reproduces, where <span class="math">\(\Delta x\)</span> is drawn from the probability density <span class="math">\(g(~)\)</span>. To make life simple we will assume that <span class="math">\(g(\Delta x)\)</span> is normally distributed with mean zero and standard deviation <span class="math">\(\sigma\)</span>, i.e. migration is unbiased an individuals migrate an average distance of <span class="math">\(\sigma\)</span>.<br /></p>
<p>Our frequency of allele <span class="math">\(2\)</span> at time <span class="math">\(t\)</span> in the population at spatial location <span class="math">\(x\)</span> is <span class="math">\(q(x,t)\)</span>. Assuming that only dispersal occurs, how does our allele frequency change in the next generation. Our allele frequency in the next generation at location <span class="math">\(x\)</span> reflects the migration from different locations in the proceeding generation. Our population at location <span class="math">\(x\)</span> receives a contribution <span class="math">\(g(\Delta x)q(x+\Delta x,t)\)</span> of allele <span class="math">\(2\)</span> from the population at location <span class="math">\(x+\Delta x\)</span>, such that the frequency of our allele at <span class="math">\(x\)</span> in the next generation is</p>
<p><span class="math">\[q(x,t+1) = \int_{-\infty}^{\infty} g(\Delta x)q(x+\Delta x,t) d \Delta x.\]</span></p>
<p>To obtain <span class="math">\(q(x+\Delta x,t)\)</span> lets take a taylor series expansion of <span class="math">\(q(x,t)\)</span></p>
<p><span class="math">\[q(x+\Delta x,t) = q(x,t) + \Delta x \frac{dq(x,t)}{dx}+ \tfrac{1}{2}(\Delta x)^2 \frac{d^2q(x,t)}{dx^2}+\cdots\]</span></p>
<p>then</p>
<p><span class="math">\[q(x,t+1) = q(x,t) +\left( \int_{-\infty}^{\infty} \Delta x g(\Delta x) 
  d \Delta x \right) \frac{dq(x,t)}{dx} + \tfrac{1}{2}\left( \int_{-\infty}^{\infty}(\Delta x)^2 g(\Delta x) 
  d \Delta x \right)  \frac{d^2q(x,t)}{dx^2}+\cdots\]</span></p>
<p><span class="math">\(g(~)\)</span> has a mean of zero so <span class="math">\( \int_{-\infty}^{\infty} \Delta x g(\Delta x) 
  d \Delta x =0\)</span> and has variance <span class="math">\(\sigma^2\)</span> so <span class="math">\(\int_{-\infty}^{\infty}(\Delta x)^2 g(\Delta x) 
  d \Delta x = \sigma^2\)</span> and all higher terms are zero (as all high moments of the normal are zero). Looking at the change in frequency <span class="math">\(\Delta
  q(x,t) = q(x,t+1)-q(x,t)\)</span> then</p>
<p><span class="math">\[\Delta q(x,t) = \frac{\sigma^2}{2} \frac{d^2q(x,t)}{dx^2}\]</span></p>
<p>this is a diffusion equation, so that migration is acting to smooth out allele frequency differences with a diffusion constant of <span class="math">\(\tfrac{\sigma^2}{2}\)</span>. This is exactly analogous to the equation describing how a gas diffuses out to equal density, as both particles in a gas and our individuals of type <span class="math">\(2\)</span> are performing brownian motion (blurring our eyes and seeing time as continuous).<br /></p>
<p>We will now introduce fitness differences into our model and set the relative fitnesses of allele <span class="math">\(1\)</span> and <span class="math">\(2\)</span> at location <span class="math">\(x \)</span> to be <span class="math">\(1\)</span> and <span class="math">\(1+s\gamma(x)\)</span>. To make progress in this model we’ll have to assume that selection isn’t too strong i.e. <span class="math">\(s \gamma(x) \ll 1\)</span> for all <span class="math">\(x\)</span>. The change in frequency of allele <span class="math">\(2\)</span> obtained within a generation due to selection is</p>
<p><span class="math">\[q^{\prime}(x,t) - q(x,t) \approx s\gamma(x) q(x,t) \big( 1 - q(x,t) \big)\]</span></p>
<p>i.e. logistic growth of our favoured allele at location <span class="math">\(x\)</span>. Putting our selection and migration terms together we find</p>
<p><span class="math">\[q(x,t+1) - q(x,t) = s\gamma(x) q(x,t) \big( 1 - q(x,t)
\big)+\frac{\sigma^2}{2} \frac{d^2q(x,t)}{dx^2} \label{eqn:fisherKPP}\]</span></p>
<p>in deriving this we have essentially assumed that migration acted upon our original frequencies before selection and in doing so have ignored terms of the order of <span class="math">\(\sigma s\)</span>.</p>
<p><embed src="Figs/equilib_cline.eps" /></p>
<p>[fig:cline]</p>
<p>To make progress lets consider a simple model of location adaptation where the environment abruptly changes. Specifically we assume that <span class="math">\(\gamma(x)= 1\)</span> for <span class="math">\(x&lt;0\)</span> and <span class="math">\(\gamma(x)= -1\)</span> for <span class="math">\(x \geq 0\)</span>, i.e. our allele <span class="math">\(2\)</span> has a selective advantage at locations to the left of zero, while this allele is at a disadvantage to the right of zero. In this case we can get an equilibrium distribution of our two alleles were to the left of <span class="math">\(zero\)</span> our allele <span class="math">\(2\)</span> is at higher frequency, while to the right of zero allele <span class="math">\(1\)</span> predominates. As we cross from the left to the right side of our range the frequency of our allele <span class="math">\(2\)</span> decreases in a smooth cline.<br /></p>
<p>Our equilibrium spatial distribution of allele frequencies can be found by setting the LHS of eqn. to zero to arrive at</p>
<p><span class="math">\[s\gamma(x) q(x) \left( 1 - q(x) \right) = - \frac{\sigma^2}{2} \frac{d^2q(x)}{dx^2}\]</span></p>
<p>We then could solve this differential equation with appropriate boundary conditions (<span class="math">\(q(-\infty)=1\)</span> and <span class="math">\(q(\infty) = 0\)</span>) to arrive at the appropriate functional form to our cline. While we won’t go into the solution of this equation here, we can note that by dividing our distance <span class="math">\(x\)</span> by <span class="math">\(\ell=\sigma/\sqrt{s}\)</span> we can remove the effect of our parameters from the above equation. This compound parameter <span class="math">\(\ell\)</span> is the characteristic length of our cline, and it is this parameter which determines over what geographic scale we change from allele <span class="math">\(2\)</span> predominating to allele <span class="math">\(1\)</span> predominating as we move across our environmental shift.<br /></p>
<p>The width of our cline, i.e. over what distance do we make this shift from allele <span class="math">\(2\)</span> predominating to allele <span class="math">\(1\)</span>, can be defined in a number of different ways. One simple way to define the cline width, which is easy to define but perhaps hard to measure accurately, is the slope (i.e. the tangent) of <span class="math">\(q(x)\)</span> at <span class="math">\(x=0\)</span>. Under this definition the cline width is approximately <span class="math">\(0.6
\sigma/\sqrt{s}\)</span>.<br /></p>
<h1 id="stochasticity-and-genetic-drift-in-allele-frequencies">Stochasticity and Genetic Drift in allele frequencies</h1>
<h2 id="stochastic-loss-of-strongly-selected-alleles">Stochastic loss of strongly selected alleles</h2>
<p>Even strongly selected alleles can be lost from the population when they are sufficiently rare. This is because the number of offspring left by individuals to the next generation is fundamentally stochastic. A selection coefficient of s=<span class="math">\(1\%\)</span> is a strong selection coefficient, which can drive an allele through the population in a few hundred generations once the allele is established. However, if individuals have on average a small number of offspring per generation the first individual to carry our allele who has on average <span class="math">\(1\%\)</span> more children could easily have zero offspring, leading to the loss of our allele before it ever get a chance to spread.<br /></p>
<p>To take a first stab at this problem lets think of a very large haploid population, and in order for this population to stay constant in size we’ll assume that individuals without the selected mutation have on average one offspring per generation. While individuals with our selected allele have on average <span class="math">\(1+s\)</span> offspring per generation. We’ll assume that the distribution of offspring number of an individual is Poisson distributed with this mean, i.e. the probability that an individual with the selected allele has <span class="math">\(i\)</span> children is</p>
<p><span class="math">\[P_i= \frac{(1+s)^i e^{-(1+s)}}{i!}\]</span></p>
<p>Consider starting from a single individual with the selected allele, and ask about the probability of eventual loss of our selected allele starting from this single copy (<span class="math">\(p_L\)</span>). To derive this we’ll make use of a simple argument (derived from branching processes). Our selected allele will be eventually lost from the population if every individual with the allele fails to leave descendents.</p>
<ol>
<li><p>In our first generation with probability <span class="math">\(P_0\)</span> our individual leaves no copies of itself to the next generation, in which case our allele is lost.</p></li>
<li><p>Alternatively it could leave one copy of itself to the next generation (with probability <span class="math">\(P_1\)</span>), in which case with probability <span class="math">\(p_L\)</span> this copy eventually goes extinct.</p></li>
<li><p>It could leave two copies of itself to the next generation (with probability <span class="math">\(P_2\)</span>), in which case with probability <span class="math">\(p_L^2\)</span> both of these copies eventually goes extinct.</p></li>
<li><p>More generally it could leave could leave <span class="math">\(k\)</span> copies (<span class="math">\(k&gt;0\)</span>) of itself to the next generation (with probability <span class="math">\(P_k\)</span>), in which case with probability <span class="math">\(p_L^k\)</span> all of these copies eventually go extinct.</p></li>
</ol>
<p>summing over this probabilities we see that</p>
<p><span class="math">\[\begin{aligned}
p_L &amp;= \sum_{k=0}^{\infty} P_k p_L^{k}  \nonumber \\
&amp;=  \sum_{k=0}^{\infty} \frac{(1+s)^ke^{-(1+s)}}{k!} p_L^{k} \nonumber
\\
&amp;= e^{-(1+s)} \left( \sum_{k=0}^{\infty} \frac{\left(p_L(1+s) \right)^k}{k!}  \right)\end{aligned}\]</span></p>
<p>well the term in the brackets is itself an exponential expansion, so we can rewrite this as</p>
<p><span class="math">\[p_L = e^{(1+s)(p_L-1)} \label{prob_loss}\]</span></p>
<p>solving this would give us our probability of loss for any selection coefficient. Lets rewrite this in terms of the the probability of escaping loss <span class="math">\(p_F = 1-p_L\)</span>. We can rewrite eqn as</p>
<p><span class="math">\[1-p_F = e^{-p_F(1+s)}\]</span></p>
<p>to gain an approximation to this lets consider a small selection coefficient <span class="math">\(s \ll 1\)</span> such that <span class="math">\(p_F \ll 1\)</span> and then expanded out the exponential on the right hand side (ignoring terms of higher order than <span class="math">\(s^2\)</span> and <span class="math">\(p_F^2\)</span>) then</p>
<p><span class="math">\[1-p_F \approx 1-p_F(1+s)+p_F^2(1+s)^2/2\]</span></p>
<p>solving this we find that</p>
<p><span class="math">\[p_F = 2s.\]</span></p>
<p>Thus even an allele with a <span class="math">\(1\%\)</span> selection coefficient has a <span class="math">\(98\%\)</span> probability of being lost when it is first introduced into the population by mutation.<br /></p>
<p>We can also adapt this result to a diploid setting. Assuming that heterozygotes for the <span class="math">\(1\)</span> allele have <span class="math">\(1+(1-h)s\)</span> children, the probability of allele <span class="math">\(1\)</span> is not lost, starting from a single copy in the population, is</p>
<p><span class="math">\[p_F = 2 (1-h)s \label{eqn:diploid_escape}\]</span></p>
<p>for <span class="math">\(h&gt;0\)</span>.<br /></p>
<h2 id="the-interaction-between-genetic-drift-and-weak-selection.">The interaction between genetic drift and weak selection.</h2>
<p>For strongly selected alleles, once the allele has escaped initial loss at low frequencies, their path will be determined deterministically by their selection coefficients. However, if selection is weak the stochasticity of reproduction can play a role in the trajectory an allele takes even when it is common in the population.<br /></p>
<p>To see this lets think of our simple Wright-Fisher model (see R exercise). Each generation we allow a deterministic change in our allele frequency, and then binomially sample two alleles for each of our offspring to construct our next generation.<br /></p>
<p>So the expected change in our allele frequency within a generation is given just by our deterministic formula. To make things easy on our self lets assume an additive model, i.e. <span class="math">\(h=1/2\)</span>, and that <span class="math">\(s \ll 1\)</span> so that <span class="math">\({\overline{w}}\approx 1\)</span>. This gives us</p>
<p><span class="math">\[{\mathbb{E}}(\Delta p ) = \frac{s}{2} p(1-p) \label{eqn:WF_mean}\]</span></p>
<p>our variance in our allele frequency change is given by</p>
<p><span class="math">\[Var(p^{\prime} - p) = Var(p^{\prime}) = \frac{p^{\prime}(1-p^{\prime})}{2N}\]</span></p>
<p>this variance in our allele frequency follows from the fact that we are binomially sampling <span class="math">\(2N\)</span> new alleles in the next generation from a frequency <span class="math">\(p^{\prime}\)</span>. Denoting our count of allele <span class="math">\(1\)</span> by <span class="math">\(i\)</span> our</p>
<p><span class="math">\[Var(p^{\prime} - p) = Var(\frac{i}{2N} - p) =  Var(\frac{i}{2N} ) =\frac{Var(i)}{(2N)^2}\]</span></p>
<p>and from binomial sampling <span class="math">\(Var(i) = 2N p^{\prime}(1-p^{\prime})\)</span> and so we arrive at our answer. Assuming that <span class="math">\(s \ll 1\)</span>, <span class="math">\(p^{\prime}
\approx p\)</span>, then in practice we can use</p>
<p><span class="math">\[Var(\Delta p)  =Var(p^{\prime} - p) \approx \frac{p(1-p)}{2N}. \label{eqn:WF_var}\]</span></p>
<p>To get our first look at the relative effects of selection vs drift we can simply look at when our change in allele frequency caused selection within a generate is reasonably faithfully passed across the generations. In particular if our expected change in frequency is much great than the variance around this change, genetic drift will play little role in the fate of our selected allele (once the allele is not too rare within the population). When does selected dominant genetic drift? This will happen if <span class="math">\({\mathbb{E}}(\Delta p) \gg Var(\Delta p)\)</span> when <span class="math">\(Ns \gg 1\)</span>. Conversely any hope of our selected allele following its deterministic path will be quickly undone if our change in allele frequencies due to selection is much less than the variance induced by drift. So if <span class="math">\(Ns \ll 1\)</span> then drift will dominate the fate of our allele.<br /></p>
<p>To make further progress on understanding the fate of alleles with selection coefficients of the order <span class="math">\(1/N\)</span> requires more careful modeling. However, we can obtain the probability that under our diploid model, with an additive selection coefficient <span class="math">\(s\)</span>, the probability of allele <span class="math">\(1\)</span> fixing within the population starting from a frequency <span class="math">\(p\)</span> is given by</p>
<p><span class="math">\[\pi(p) = \frac{1-e^{-2Ns p }}{1-e^{-2Ns}} \label{eqn:prob_fixed}\]</span></p>
<p>the proof of this is sketched out below (see Section [Section:fixation<sub>w</sub>eakly<sub>s</sub>el]). A new allele will arrive in the population at frequency <span class="math">\(p=1/(2N)\)</span>, then its probability of reaching fixation is</p>
<p><span class="math">\[\pi \left(\frac{1}{2N} \right) = \frac{1-e^{-s }}{1-e^{-2Ns}}\]</span></p>
<p>if <span class="math">\(s \ll1\)</span> but <span class="math">\(Ns \gg 1\)</span> then <span class="math">\(\pi(\frac{1}{2N}) \approx s\)</span>, which nicely gives us back our result that we obtained above (eqn. ).<br /></p>
<p>In the case where <span class="math">\(Ns\)</span> close to <span class="math">\(1\)</span> then</p>
<p><span class="math">\[\pi(\frac{1}{2N}) \approx \frac{s}{1-e^{-2Ns}} \label{eqn:escape_from_intro}\]</span></p>
<p>this is greater than <span class="math">\(s\)</span>, increasingly so for smaller <span class="math">\(N\)</span>. Why is this? Well in small populations selected alleles spend a somewhat shorter time segregating (especially at low frequencies), and so are slightly less susceptible to genetic drift.<br /></p>
<h3 id="the-fixation-of-slightly-deleterious-alleles.">The fixation of slightly deleterious alleles.</h3>
<p>We can also use eqn. to understand how likely it is that deleterious alleles accidently reach fixation by genetic drift, assuming a diploid model with additive selection (with a selection coefficient of <span class="math">\(-s\)</span> against our allele <span class="math">\(2\)</span>). If <span class="math">\(N s \gg 1\)</span> then our deleterious allele (allele <span class="math">\(2\)</span>) can not possibly reach fixation. However, if <span class="math">\(Ns\)</span> is not large then</p>
<p><span class="math">\[\pi(\frac{1}{2N}) = \frac{s}{e^{2Ns}-1} \label{eqn:fix_deleterious}\]</span></p>
<p>for our deleterious allele. So deleterious alleles can fix within populations (albeit at a low rate) if <span class="math">\(Ns\)</span> is not too large.<br /></p>
<h3 id="Section:fixation_weakly_sel">A Sketch Proof of the probability of fixation of weakly selected alleles</h3>
<p>We’ll let <span class="math">\(P(\Delta p)\)</span> be the probability that our allele frequency shifts by <span class="math">\(\Delta p\)</span> in the next generation. Using this we can write our probability <span class="math">\(\pi(p)\)</span> in terms of the probability of achieving fixation averaged over the frequency in the next generation</p>
<p><span class="math">\[\pi(p)  = \int \pi(p+\Delta p) P(\Delta p) d(\Delta p) \label{eqn:prob_fix_diff_step1}\]</span></p>
<p>This is very similar to the technique that we used deriving our probability of escaping loss in a very large population above.<br /></p>
<p>So we need an expression for <span class="math">\(\pi(p+\Delta p)\)</span>. To obtain this we’ll do a Taylor series expansion of <span class="math">\(\pi(p)\)</span> assuming that <span class="math">\(\Delta p \)</span> is small</p>
<p><span class="math">\[\pi(p+\Delta p) \approx \pi(p) + \Delta p \frac{d\pi(p)}{dp} + (\Delta p)^2
\frac{d^2\pi(p)}{dp^2} (p)\]</span></p>
<p>ignoring higher order terms.<br /></p>
<p>Taking the expectation over <span class="math">\(\Delta p \)</span> on both sides, as in eqn. [eqn:prob<sub>f</sub>ix<sub>d</sub>iff<sub>s</sub>tep1], we obtain</p>
<p><span class="math">\[\pi(p) = \pi(p) + {\mathbb{E}}(\Delta p) \frac{d\pi (p)}{dp} + {\mathbb{E}}((\Delta p)^2)
\frac{d^2\pi(p)}{dp^2}\]</span></p>
<p>Well <span class="math">\({\mathbb{E}}(\Delta p) = \frac{s}{2}p(1-p)\)</span> and <span class="math">\(Var(\Delta p)= {\mathbb{E}}((\Delta
p)^2)-{\mathbb{E}}^2(\Delta p)\)</span>, so if <span class="math">\(s \ll 1\)</span> then <span class="math">\({\mathbb{E}}^2(\Delta p) \approx
0\)</span>, and <span class="math">\({\mathbb{E}}(\Delta p)^2 = \frac{p(1-p)}{2N}\)</span>. This leaves us with</p>
<p><span class="math">\[0= \frac{s}{2}p(1-p)\frac{d\pi (p) }{dp} + \frac{p(1-p)}{2N}
\frac{d^2\pi (p) }{dp^2}\]</span></p>
<p>and we can specify the boundary conditions to be <span class="math">\(\pi(1)=1\)</span> and <span class="math">\(\pi(0)=0\)</span>. Solving this differential equation is somewhat involved process but in doing so we find that</p>
<p><span class="math">\[\pi(p) = \frac{1-e^{-2Ns p }}{1-e^{-2Ns}}\]</span></p>
<h1 id="genetic-drift-and-neutral-alleles">Genetic drift and Neutral alleles</h1>
<h2 id="the-fixation-of-neutral-alleles">The fixation of neutral alleles</h2>
<p>It is very unlikely that a rare neutral allele accidentally drifts up to fixation, it is much more likely that such an allele is eventually lost from the population. However, there is a large and constant influx of rare alleles into the population due to mutation, so even if it is very unlikely that an individual allele fixes within the population, some neutral alleles will fix.<br /></p>
<h4 id="probability-of-the-eventual-fixation-of-a-neutral-allele.">Probability of the eventual fixation of a neutral allele.</h4>
<p>An allele which reaches fixation within a population, is an ancestor to the entire population. In a particular generation there can be only single allele that all other alleles at the locus in later generation can claim as an ancestor. As at a neutral locus all of our alleles are exchangeable, as they have no effect on the number of descendents an individual leaves, so any allele is equally likely to be the ancestor of the entire population. In a diploid population size of size <span class="math">\(N\)</span>, there are <span class="math">\(2N\)</span> alleles all of which are equally likely to be the ancestor of the entire population at some later time point. So if our allele is present in a single copy, the chance that is the ancestor to the entire population in some future generation is <span class="math">\(1/(2N)\)</span>, i.e. the chance our neutral allele is eventually fixed is <span class="math">\(1/(2N)\)</span>.<br /></p>
<p>More generally if our neutral allele is present in <span class="math">\(i\)</span> copies in the population, of <span class="math">\(2N\)</span> alleles, the probability that this allele is fixed is <span class="math">\(i/(2N)\)</span>. I.e. the probability that a neutral allele is eventually fixed is simply given by its frequency (<span class="math">\(p\)</span>) in the population. We can also derive this result by letting <span class="math">\(Ns \rightarrow
0\)</span> in eqn. .</p>
<h4 id="rate-of-substitution-of-neutral-alleles.">Rate of substitution of neutral alleles.</h4>
<p>A substitution between populations that do not exchange gene flow is simply a fixation event within one population. The rate of substitution is therefore the rate at which new alleles fix in the population, so that the long-term substitution rate is the rate at which mutations arise that will eventually become fixed within our population.<br /></p>
<p>Assume that there are two classes of mutational changes that can occur with a region, highly deleterious mutations and neutral mutations. A fraction <span class="math">\(C\)</span> of all mutational changes are highly deleterious, and can not possibly contribute to substitution nor polymorphism (i.e. <span class="math">\(Ns \gg 1\)</span>). While a fraction <span class="math">\(1-C\)</span> are neutral. If our mutation rate is <span class="math">\(\mu\)</span> per transmitted allele per generation, then a total of <span class="math">\(2N \mu (1-C)\)</span> neutral mutations enter our population each generation.<br /></p>
<p>Each of these neutral mutations has a <span class="math">\(1/(2N)\)</span> probability chance of eventually becoming fixed in the population. Therefore, the rate at which neutral mutations arise that eventually become fixed within our population is</p>
<p><span class="math">\[2N\mu(1-C)\frac{1}{2N} = \mu(1-C)\]</span></p>
<p>thus the rate of substitution under a model where newly arising alleles are either highly deleterious or neutral, is simply given by the mutation rate towards neutral alleles, i.e. <span class="math">\(\mu(1-C)\)</span>.<br /></p>
<p>Consider a pair of species have diverged for <span class="math">\(T\)</span> generations, i.e. orthologous sequences shared between the species last shared a common ancestor <span class="math">\(T\)</span> generations ago. If they have maintained a constant <span class="math">\(\mu\)</span> over that time, will have accumulated an average of</p>
<p><span class="math">\[2\mu(1-C)T\]</span></p>
<p>neutral substitutions. This assumes that <span class="math">\(T\)</span> is a lot longer than the time it takes to fix a neutral allele, such that the total number of alleles introduced into the population that will eventually fix is the total number of substitutions. We’ll see below that a neutral allele takes on average <span class="math">\(4N\)</span> generations to fix from its introduction into the population.<br /></p>
<p>This is a really pretty result as the population size has completely canceled out of the neutral substitution rate. However, there is another way to see this in a more straightward way. If I look at a sequence in me compared to say a particular chimp, I’m looking at the mutations that have occurred in both of our germlines since they parted ways <span class="math">\(T\)</span> generations ago. Since neutral alleles do not alter the probability of their transmission to the next generation, we are simply looking at the mutations that have occurred in <span class="math">\(2T\)</span> generations worth of transmissions. Thus the average number of neutral mutational differences separating our pair of species is simply <span class="math">\(2\mu (1-C) T\)</span>.<br /></p>
<h2 id="the-coalescent-and-patterns-of-neutral-diversity">The Coalescent and patterns of neutral diversity</h2>
<h4 id="pairwise-coalescent-time-distribution-and-the-number-of-pairwise-differences.">Pairwise Coalescent time distribution and the number of pairwise differences.</h4>
<p>Thinking back to our calculations we made about the loss of neutral heterozygosity and equilibrium levels of diversity (in Sections [LossofHet] and [DriftMutationBalance]), you’ll note that we could first specify what generation a pair of sequences coalesce in, and then calculate some properties of heterozygosity based on that. That’s because neutral mutations do not affect the probability that an individual transmits that allele, so don’t affect the way in which we can trace ancestral lineages back.<br /></p>
<p>As such it will often be helpful to consider the time to the common ancestor of a pair of sequences, and then think of the impact of that on patterns of diversity. See Figure [fig:Coalescent<sub>s</sub>imulation] for an example of this. The probability that a pair of alleles have failed to coalesce in <span class="math">\(t\)</span> generations and then coalesce in the <span class="math">\(t+1\)</span> generation back is</p>
<p><span class="math">\[\frac{1}{2N} (1- \frac{1}{2N} )^{t} \approx \frac{1}{2N} e^{-t/(2N)} \label{eqn:coal_time_dist}\]</span></p>
<p>thus the coalescent time of a pair of sequences (<span class="math">\(T_2\)</span>) is approximately exponentially distributed with a rate <span class="math">\(1/(2N)\)</span>. We’ll denote that by saying that <span class="math">\(T_2 \sim Exp \left( 1/(2N) \right)\)</span>. The mean coalescent time of a pair of a pair of alleles is <span class="math">\(2N\)</span> generations<br /></p>
<p><img src="Figs/Coalescent.png" alt="image" /></p>
<p>[fig:Coalescent<sub>s</sub>imulation]</p>
<p>Conditional on a pair of alleles coalescing <span class="math">\(t\)</span> generations ago there are <span class="math">\(2t\)</span> generations in which a mutation could occur. Thus the probability of our pair of alleles are separated by <span class="math">\(j\)</span> mutations since they last shared a common ancestor is</p>
<p><span class="math">\[P(j | T_2 = t ) = {2t \choose j} \mu^{j} (1-\mu)^{2t-j}\]</span></p>
<p>i.e. mutations happen in <span class="math">\(j\)</span> generations, and do not happen in <span class="math">\(2t-j\)</span> generations (with <span class="math">\({2t \choose j}\)</span> ways this can possibly happen). Assuming that <span class="math">\(\mu \ll 1\)</span>, and that <span class="math">\(2t-j \approx 2t\)</span> then we can approximate the probability that we have <span class="math">\(j\)</span> mutations as a Poisson distribution</p>
<p><span class="math">\[P(j | T_2 = t ) = \frac{ (2 \mu t )^{j} e^{-2\mu t}}{j!}\]</span></p>
<p>i.e. a Poisson with mean <span class="math">\(2\mu t \)</span>.<br /></p>
<p>As our expected coalescent time is <span class="math">\(2N\)</span> generations, the expected number of mutations separating two alleles drawn at random from the population is</p>
<p><span class="math">\[{\mathbb{E}}(j) = 4N\mu = \theta\]</span></p>
<p>We’ll assume that mutations never happen at the same site twice, i.e. no multiple hits, such that we get to see all of the mutation events that separate our pair of sequences (we’ll call this the infinitely-many-sites assumption, which should be fine if <span class="math">\(N\mu_{BP} \ll 1\)</span>). Thus the number of mutations between a pair of sites is the observed number of differences between a pair of sequences.<br /></p>
<p>We’ll denote the observed number of pairwise differences at putatively neutral sites separating a pair of sequences as <span class="math">\(\pi\)</span> (we usually average this over a number of pairs of sequences for a region). So we can estimate of <span class="math">\(\theta\)</span> from <span class="math">\(\pi\)</span>, <span class="math">\(\widehat{\theta}_{\pi}\)</span> by setting <span class="math">\(\widehat{\theta}_{\pi}=\pi\)</span>. If we have an independent estimate of <span class="math">\(\mu\)</span>, then from setting <span class="math">\(\pi = \widehat{\theta}_{\pi} = 4N\mu\)</span> we can obtain an estimate of the population size <span class="math">\(N\)</span> that is consistent with our levels of neutral polymorphism.</p>
<h2 id="the-coalescent-process-of-a-sample-of-alleles.">The coalescent process of a sample of alleles.</h2>
<p>Usually we are not just interested pairs of alleles, or the average pairwise diversity, we are interested in the properties of diversity in samples of a number of alleles drawn from the population. To allow for this instead of just following a pair of lineages back until they coalesce, we can follow the history of a sample of alleles back through the population.</p>
<p>Consider first sampling three alleles at random from the population. The probability that all three alleles choose exactly the same ancestral allele one generation back is <span class="math">\(1/(2N)^2\)</span>. If <span class="math">\(N\)</span> is reasonably large then this is a very small probability. As such it is very unlikely that our three alleles coalesce at once, a in a moment we’ll see that it is safe to ingnore such unlikely events.<br /></p>
<p>The probability that a specific pair of alleles find a common ancestor in the preceding generation is still <span class="math">\(1/(2N)\)</span>. There are three possible pairs of alleles so the probability that no pair finds a common ancestor is</p>
<p><span class="math">\[\left(1-\frac{1}{(2N)} \right)^3 \approx \left( 1- \frac{3}{2N} \right)\]</span></p>
<p>in making this approximation we are multiplying out the right hand-side and ignoring terms of <span class="math">\(1/N^2\)</span> and higher. See Figure [fig:Coalescent<sub>s</sub>imulation<sub>3</sub>] for a random realization of this process.<br /></p>
<p><img src="Figs/Coalescent_3" alt="image" /></p>
<p>[fig:Coalescent<sub>s</sub>imulation<sub>3</sub>]</p>
<p>More generally when we sample <span class="math">\(i\)</span> alleles there are <span class="math">\({i \choose
 2}\)</span> pairs, i.e. <span class="math">\(i(i-1)/2\)</span> pairs, thus the probability that no pair of alleles coalesces in the preceding generation is</p>
<p><span class="math">\[\left(1-\frac{1}{(2N)} \right)^{{i \choose
 2}} \approx \left( 1- \frac{{i \choose
 2}}{2N}\right)\]</span></p>
<p>while the probability of any pair coalescing is <span class="math">\(\approx \frac{{i \choose
 2}}{2N}\)</span>.<br /></p>
<p>We can ignore the possibility of more than pairs of alleles (e.g. tripletons) simultaneously coalescing at once as terms of <span class="math">\(1/N^2\)</span> and higher can be ignored as they are vanishingly rare. Obviously there are in reasonable sample sizes there are many more triples (<span class="math">\({i \choose 3}\)</span>), and higher order combinations, than pairs (<span class="math">\({i \choose 2}\)</span>) but if <span class="math">\(i \ll N\)</span> then we are safe to ignore these terms.</p>
<p>When there are <span class="math">\(i\)</span> alleles the probability that we wait until the <span class="math">\(t+1\)</span> generation before any pair of alleles coalesce is</p>
<p><span class="math">\[\frac{{i \choose
 2}}{2N}\left( 1- \frac{{i \choose
 2}}{2N}\right)^{t} \approx  \frac{{i \choose
 2}}{2N} \exp \left( - \frac{{i \choose
 2}}{2N} t \right)\]</span></p>
<p>thus the waiting time <span class="math">\(T_i\)</span> to the first coalescent event in a sample of <span class="math">\(i\)</span> alleles is exponentially distributed with rate <span class="math">\(\frac{{i \choose
 2}}{2N}\)</span>, i.e. <span class="math">\(T_i \sim Exp\left(\frac{{i \choose
 2}}{2N} \right)\)</span>. The mean waiting time till any of pair within our sample to coalesce is <span class="math">\(2N/{i \choose
 2}\)</span>.<br /></p>
<p>When a pair of alleles first find a common ancestral allele some number of generations back further into the past we only have to keep track of that common ancestral allele for the pair. Thus when a pair of alleles in our sample of <span class="math">\(i\)</span> alleles coalesce, we then switch to having to follow <span class="math">\(i-1\)</span> alleles back. Then when a pair of these <span class="math">\(i-1\)</span> alleles coalesce, we then have to follow <span class="math">\(i-2\)</span> alleles back. This process continues until we coalesce back to a sample of two, and from there to a single most recent common ancestor (MRCA).<br /></p>
<p>To simulate a coalescent genealogy at a locus for a sample of <span class="math">\(n\)</span> alleles we therefore simply follow this algorithm</p>
<ol>
<li><p>set <span class="math">\(i=n\)</span>.</p></li>
<li><p>We simulate a random variable to be the time <span class="math">\(t_i\)</span> to the next coalescent event from <span class="math">\(t_i \sim
 Exp\left(\frac{{i \choose
 2}}{2N} \right)\)</span></p></li>
<li><p>choose a pair of alleles to coalesce at random from all possible pairs.</p></li>
<li><p>set <span class="math">\(i=i-1\)</span></p></li>
<li><p>continue looping of steps 1-3 until <span class="math">\(i=1\)</span> i.e. the most recent common ancestor of the sample is found.</p></li>
</ol>
<p>by following this algorithm we are generating realizations of the genealogy of our sample.<br /></p>
<p>We will first consider the time to the most recent common ancestor of the entire sample (<span class="math">\(T_{MRCA}\)</span>). This is</p>
<p><span class="math">\[T_{MRCA} = \sum_{i=n}^2 T_i\]</span></p>
<p>generations back. As our coalescent times for different <span class="math">\(i\)</span> are independent, the expected time to the most recent common ancestor is</p>
<p><span class="math">\[{\mathbb{E}}(T_{MRCA}) = \sum_{i=n}^2 {\mathbb{E}}(T_i) = \sum_{i=n}^2  2N/{i \choose
 2}\]</span></p>
<p>using the fact that <span class="math">\(\frac{1}{i(i-1)}=\frac{1}{i-1} - \frac{1}{i}\)</span> with a bit of rearrangement we can rewrite this is</p>
<p><span class="math">\[{\mathbb{E}}(T_{MRCA}) = 4N\left(1- \frac{1}{n} \right)\]</span></p>
<p>so the average <span class="math">\(T_{MRCA}\)</span> scales linearly with population size. Interestingly, as we move to larger and larger samples (i.e. <span class="math">\(n \gg 1\)</span>) the average time to the most recent common ancestor is converging on <span class="math">\(4N\)</span>. What’s happening here is that in large samples our lineages typically coalesce rapidly at the start and very soon coalesce down to a much smaller number of lineages.<br /></p>
<p>Above we argued that a mutation is only becomes a fixed difference if it is lucky enough to be the ancestor of the entire population. As we saw above this occurs with probability <span class="math">\(1/(2N)\)</span>. How long does is take on average for such an allele to fix within our population. We’ve just seen that it takes <span class="math">\(4N\)</span> generations for a large sample of alleles to all trace their ancestry back to a single most recent common ancestor. Thus it must take roughly <span class="math">\(4N\)</span> generations for a neutral allele present in a single copy within the the population to the ancestor of all alleles within our population. This argument can be made more precise, but in general we would still find that it takes <span class="math">\(\approx 4N\)</span> generations for a neutral allele to go from its introduction to fixation with the population.<br /></p>
<p>The total amount of time in the genealogy (<span class="math">\(T_{tot}\)</span>)</p>
<p><span class="math">\[T_{tot} = \sum_{i=n}^2 iT_i\]</span></p>
<p>as when there are <span class="math">\(i\)</span> lineages each contributes a time <span class="math">\(T_i\)</span> to the total time. Taking the expectation of the total time in the genealogy</p>
<p><span class="math">\[{\mathbb{E}}(T_{tot}) = \sum_{i=n}^2 i \frac{2N}{{i \choose
 2} } = \sum_{i=n}^2 \frac{4N}{i -1} =\sum_{i=n-1}^1 \frac{4N}{i}\]</span></p>
<p>so our expected total amount of time in the genealogy scales linearly with our population size. Our expected total amount of time is also increasing with sample size but is doing so very slowly. To see this more carefully we can see that for large <span class="math">\(n\)</span></p>
<p><span class="math">\[{\mathbb{E}}(T_{tot}) = \sum_{i=n-1}^1 \frac{4N}{i} \approx 4N \int_1^n \frac{1}{i} di
= 4N \log(n-1)\]</span></p>
<p>here we are approximating our sum by an integral, which will work for large <span class="math">\(n\)</span>. So our expected total amount of time in the genealogy is growing with <span class="math">\(n\)</span> but it is doing so very slowly. This again follows from the fact that in large samples the initial coalescence usually happens very rapidly, so that extra samples adds little to the total amount of time in the tree.<br /></p>
<p>We saw above that the number of mutational differences between a pair of alleles that coalescence <span class="math">\(T_2\)</span> generations ago was Poisson with a mean of <span class="math">\(2 \mu T_2\)</span>. A mutation that occurs on any branch of our genealogy will cause a segregating polymorphism in the sample (making our infinitely-many-sites assumption). Thus if the total time in the genealogy is <span class="math">\(T_{tot}\)</span> there is <span class="math">\(T_{tot}\)</span> generations for mutations. So the total number of mutations segregating in our sample (<span class="math">\(S\)</span>) is Poisson with mean <span class="math">\(\mu T_{tot}\)</span>. Thus the expected number of segregting in history a sample of size <span class="math">\(n\)</span> is</p>
<p><span class="math">\[{\mathbb{E}}(S) = \mu {\mathbb{E}}(T_{tot}) = \sum_{i=n-1}^1 \frac{4N\mu }{i} = \theta
\sum_{i=n-1}^1 \frac{1}{i}\]</span></p>
<p>Thus we can use this formula to derive another estimate of the population scaled mutation rate, by setting our observed number of segregating sites in a sample (<span class="math">\(S\)</span>) equal to this expectation. We’ll call this estimator <span class="math">\(\widehat{\theta}_W\)</span></p>
<p><span class="math">\[\widehat{\theta}_W =\frac{ S}{\sum_{i=n-1}^1 \frac{1}{i}}\]</span></p>
<p>this estimator was devised by Watterson, hence the <span class="math">\(W\)</span>.</p>
<h2 id="comparing-polymorphism-and-divergence">Comparing polymorphism and divergence</h2>
<h2 id="deviations-from-the-constant-population-model.">Deviations from the constant population model.</h2>
<p>We’ve seen previously that changes in our population size can be captured by an effective population size. However, this will only be a useful measure if population sizes vary rapidly enough, that the harmonic mean effective population size over short time periods (<span class="math">\(\ll
N_e\)</span> generations) is representative of the effective population size averaged over longer time periods. If this is not the case there is no one effective population size, as we can not approximate our rate of drift by a single constant population. Furthermore, we’ve ignored the effect of population structure and selection which will violate our modeling assumptions.<br /></p>
<p>We can hope to detect violations from our constant population size neutral model, by comparing aspects of our dataset to their expectations and distributions under our neutral model.<br /></p>
<p>For example we have devised two estimates of <span class="math">\(\theta\)</span>, <span class="math">\(\widehat{\theta_{\pi}}\)</span> and <span class="math">\(\widehat{\theta_{W}}\)</span>, using expectations of different aspects of our data (pairwise diversity and number of segregating sites respectively). Under our constant neutral model if we have sufficient data those two estimates should be equal to each other on average. But if there’s some violation of our model they might not be. So one test statistic might be to take</p>
<p><span class="math">\[D = \widehat{\theta_{\pi}} - \widehat{\theta_{W}}\]</span></p>
<p>which will be zero in expectation if our data was generated by a neutral constant population model.</p>
<h1 id="the-effect-of-linked-selection-on-patterns-of-neutral-diversity">The effect of linked selection on patterns of neutral diversity</h1>
<p>A newly derived allele with an additive selection coefficient <span class="math">\(s\)</span> will take a time <span class="math">\(\tau \approx 2\log(2N)/s\)</span> generations to reach to fixation within our population. This short time window offers very little time for recombination between the the selected site and linked neutral sites.<br /></p>
<p>First lets imagine examining variation at a locus fully linked to our selected locus, just after our sweep reached fixation. A pair of neutral alleles sampled at this locus must both trace their ancestral lineages back through to the neutral allele on whose background the selected allele initially arose. As that neutral allele, which existed <span class="math">\(\tau\)</span> generations ago is the ancestor of the entire population at this locus. Our individuals who carry the beneficial allele are, from the perspective of these two alleles, exactly like a rapidly expanding population. Therefore, our pair of neutral alleles sampled at our locus will be forced to coalesce <span class="math">\(\approx \tau\)</span> generations ago. This is a very short-time scale compared to the average neutral coalescent tie of <span class="math">\(2N\)</span> generations of a pair of alleles.<br /></p>
<p>If we now allow recombination into our model we can think about a pair of alleles sampled at a neutral locus a recombination distance <span class="math">\(r\)</span> away from our selected site. Our pair of alleles will be forced to coalesce <span class="math">\(\approx \tau\)</span> generations if neither of them reside on haplotypes that the selected allele recombined onto during the sweep. This is equivalent to saying that neither of our neutral alleles recombine off of the beneficial allele’s background moving backward in time.<br /></p>
<p>The probability that our lineage fail recombines off our beneficial allele’s background and onto the ancestral background in the <span class="math">\(j^{th}\)</span> generation back is</p>
<p><span class="math">\[r (1-X(j))\]</span></p>
<p>so the probability (<span class="math">\(p_{NR}\)</span>) that our lineage fails to recombine off in the <span class="math">\(\tau\)</span> generations it takes our selected allele to move through the population is</p>
<p><span class="math">\[p_{NR}=\prod_{j=1}^{\tau} \big(1- r(1-X(j))\big)\]</span></p>
<p>assuming that <span class="math">\(r\)</span> is small then <span class="math">\( \left(1- r(1-X(j))\right) \approx
e^{-r(1-X(j))}\)</span>, such that</p>
<p><span class="math">\[p_{NR}=\prod_{j=1}^{\tau} \left(1- r(1-X(j))\right) \approx \exp
\left( -r\sum_{j=1}^{\tau}
1- X(j) \right) =\exp
\left( -r \tau (1-\widehat{X}) \right)\]</span></p>
<p>where <span class="math">\(\widehat{X}\)</span> is the average frequency of the derived allele across the trajectory <span class="math">\(\widehat{X} = \frac{1}{\tau}  \sum_{j=1}^{\tau}
 X(j)\)</span>. As our allele is additive its trajectory for frequencies <span class="math">\(&lt;0.5\)</span> is the mirror image of its trajectory for frequency <span class="math">\(&gt;0.5\)</span>, therefore it average frequency <span class="math">\(\widehat{X} =0.5\)</span>. So</p>
<p><span class="math">\[p_{NR} = e^{-r \tau/2 }.\]</span></p>
<p>The probability that both of our lineages fail to recombine off the sweep and hence are forced to coalesce is <span class="math">\(p_{NR}^2\)</span>, assuming that they coalesce at a time close to <span class="math">\(\tau\)</span> so that they recombine independently of each other for times <span class="math">\(&lt; \tau\)</span>.<br /></p>
<p>If one or other of our lineages recombine off the sweep it will take them on average <span class="math">\(\approx 2N\)</span> generations to find a common ancestor as we are back our neutral coalescent. Thus the expected time till our pair of lineages find a common ancestor is</p>
<p><span class="math">\[{\mathbb{E}}(T_2)  = \tau \times p_{NR}^2 +(1-p_{NR}^2) (\tau +2N) \approx
\left(1-p_{NR}^2 \right) 2N\]</span></p>
<p>where this last approximation assumes that <span class="math">\(\tau \ll 2N\)</span>. So the expected pairwise diversity for neutral alleles at a recombination distance <span class="math">\(r\)</span> away from the selected sweep (<span class="math">\(\pi_r\)</span>) is</p>
<p><span class="math">\[{\mathbb{E}}(\pi_r) = 2\mu {\mathbb{E}}(T_2)  \approx \theta \left(1-e^{-r\tau} \right) \label{eqn:pi_HH}\]</span></p>
<p>so diversity increases as we move away from the selected site, slowly exponentially plauteuing to its neutral expectation <span class="math">\(\theta=4N\mu\)</span>.<br /></p>
<p>To get a sense of the physical scale over which diversity is reduced consider a region where recombination occurs at a rate <span class="math">\(r_{BP}\)</span> per base pair per generation, and our locus is <span class="math">\( \ell \)</span> base pairs away from the selected site <span class="math">\(r=r_{BP } \ell \)</span> (where <span class="math">\(r_{BP}  \ell  \ll 1\)</span> so we don’t need to worry about more than one recombination event occurring per generation). Typical recombination rates are on the order of <span class="math">\(r_{BP} = 10^{-8}\)</span>, in Figure [fig:hitchhiking<sub>r</sub>eduction] we show the reduction in diversity, given by eqn. , for two different selection coefficients.<br /></p>
<p>For our expected diversity levels to recover to <span class="math">\(50\%\)</span> of its neutral expectation <span class="math">\({\mathbb{E}}(\pi_r)/\theta=0.5\)</span>, requires a physical distance <span class="math">\(\ell^{*}\)</span> such that <span class="math">\(\log(0.5) = -r_{BP} \ell ^*\tau\)</span> as using our expression for <span class="math">\(\tau\)</span> then <span class="math">\( \ell^* = \frac{s}{r_{BP} \log (2N) }\)</span>. The width of our trough of reduced diversity depends on <span class="math">\(s/r_{BP}\)</span>, so else being equal we expect stronger sweeps or sweeps in regions of low recombination to have a larger hitchhiking effect. So that a selection coefficient of <span class="math">\(s=0.1\%\)</span> would reduce diversity over 10’s of kb, while a sweep of <span class="math">\(s=1\%\)</span> would affect <span class="math">\(\sim\)</span>100kb.<br /></p>
<p><embed src="Figs/hitchhiking_reduction.eps" /></p>
<p>[fig:hitchhiking<sub>r</sub>eduction]</p>
<h2 id="a-simple-recurrent-model-of-selective-sweeps">A simple recurrent model of selective sweeps</h2>
<p>We sample a pair of neutral alleles at a locus a genetic distance <span class="math">\(r\)</span> away from a locus where sweeps are initiated within the population at some very low rate <span class="math">\(\nu\)</span> per generation. The waiting time between sweeps at our locus is exponential <span class="math">\(\sim Exp(\nu)\)</span>. Each sweep rapidly transits through the population in <span class="math">\(\tau\)</span> generations, such that each sweep is finished long before the next sweep (<span class="math">\(\tau \ll 1/\nu\)</span>).<br /></p>
<p>As before our chance that our neutral lineage fails to recombine off the sweep is <span class="math">\(p_{NR}\)</span>, such that the probability that our pair of lineages are forced to coalesce by a sweep <span class="math">\(e^{-r \tau}\)</span>. Our lineages therefore have a very low probability</p>
<p><span class="math">\[\nu e^{-r \tau}\]</span></p>
<p>of being forced to coalesce by a sweep per generation. In addition of lineages can coalesce at a neutral rate of <span class="math">\(1/(2N)\)</span>. Thus the average waiting time till a coalescent event between our neutral pair of lineages due to either a sweep or a neutral coalescent event is</p>
<p><span class="math">\[{\mathbb{E}}(T_2) = \frac{1}{\nu e^{-r \tau} + 1/(2N)}\]</span></p>
<p>Now imagine that the sweeps don’t occur at a fixed location with respect to our locus of interest, but now occur uniformly at random across our sequence. The sweeps are initiated at a very low rate of <span class="math">\(\nu_{BP}\)</span> per basepair per generation. The rate of coalescent due to sweeps at a locus <span class="math">\(\ell\)</span> basepairs away from our neutral loci is <span class="math">\(\nu_{BP} e^{-r_{BP} \ell \tau}\)</span>. If our neutral locus is in the middle of a chromosome that stretches <span class="math">\(L\)</span> basepairs in either direction the total rate of sweeps per generation that force our pair of lineages to coalesce is</p>
<p><span class="math">\[2\int_0^{L} \nu_{BP} e^{-r_{BP} \ell \tau} d \ell =
\frac{2\nu_{BP}}{r_{BP} \tau} \left(1-e^{-r_{BP} \tau L} \right)\]</span></p>
<p>so that if <span class="math">\(L\)</span> is very large (<span class="math">\(r_{BP} \tau L \gg 1\)</span>) the rate of coalesce per generation due to sweeps is <span class="math">\(\frac{2\nu_{BP}}{r_{BP} \tau}\)</span>. The total rate of coalescence for a pair of lineages per generation is then</p>
<p><span class="math">\[\frac{2\nu_{BP}}{r_{BP} \tau}+\frac{1}{2N}\]</span></p>
<p>So our average time till a pair of lineages coalesce is</p>
<p><span class="math">\[{\mathbb{E}}(T_2) = \frac{1}{\frac{2\nu_{BP}}{r_{BP} \tau}+\frac{1}{2N}} = \frac{r_{BP}2N}{\frac{4N\nu_{BP}}{ \tau}+r_{BP}}\]</span></p>
<p>such that our expected pairwise diversity (<span class="math">\(\pi=2\mu{\mathbb{E}}(T_2)\)</span>) in a region of recombination rate <span class="math">\(r_{BP}\)</span> that experiences sweeps at rate <span class="math">\(\nu_{BP}\)</span> is</p>
<p><span class="math">\[{\mathbb{E}}(\pi) = \theta \frac{r_{BP}}{\frac{4N\nu_{BP}}{ \tau}+r_{BP}} \label{eqn:pi_GW_HH}\]</span></p>
<p><embed src="Figs/Genomewide_HH.eps" /></p>
<p>[fig:GW<sub>h</sub>itchhiking<sub>r</sub>eduction]</p>
</body>
</html>
